
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>2011 COMBINE - Posters | COMBINE</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/sites/combine/files/Symbol-COMBINE_logo.ico" type="image/x-icon" />
  <link type="text/css" rel="stylesheet" media="all" href="/modules/node/node.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/poll/poll.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/defaults.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system-menus.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/user/user.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/ctools/css/ctools.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/panels/css/panels.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/views/css/views.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/html-reset.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/wireframes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/tabs.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/messages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/pages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/block-editing.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/blocks.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/navigation.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/nodes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/comments.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/forms.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/cti-flex2.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/combine-custom.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/layout-fixed.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/colors/combine.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/local.css?H" />
<link type="text/css" rel="stylesheet" media="print" href="/sites/all/themes/cti_flex2/css/print.css?H" />
<!--[if IE]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie.css?H" />
<![endif]-->
<!--[if lte IE 6]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie6.css?H" />
<![endif]-->
  <script type="text/javascript" src="/misc/jquery.js?H"></script>
<script type="text/javascript" src="/misc/drupal.js?H"></script>
<script type="text/javascript" src="/sites/all/modules/panels/js/panels.js?H"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/" });
//--><!]]>
</script>
  <style type="text/css">
body {font-family: Tahoma, Verdana, Arial, Helvetica, "Bitstream Vera Sans", sans-serif;}
</style>  <style type="text/css">
</style>
  <!--[if lte IE 6]>
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/suckerfish.js"></script>
  <![endif]-->
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/combine.js"></script>
</head>

<body class="page not-front not-logged-in node-type-page page-events-combine-2011-posters section-events one-sidebar sidebar-first page-panels ">

  <div id="page-wrapper"><div id="page">

    <div id="header"><div class="section clearfix">

      
              <div id="logo-title">

                      <div id="logo"><a href="/" title="Home" rel="home"><img src="/sites/combine/files/COMBINE_website-banner.png" alt="Home" id="logo-image" /></a></div>
          
      
        </div> <!-- /#logo-title -->
      
      
    </div></div> <!-- /.section, /#header -->

    <div id="main-wrapper"><div id="main" class="clearfix  with-navigation">

      <div id="content" class="column"><div class="section">

        
        
        <div class="breadcrumb"><a href="/">Home</a> › 2011 COMBINE - Posters</div>                  <h1 class="title">2011 COMBINE - Posters</h1>
                                
        
        <div id="content-area">
          <div id="node-39" class="node node-type-page build-mode-full clearfix">

   
  
  
  
  
  <div class="content">
    <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #ccc"> <tr> <th align="left" style="background-color:#ccc" width="200px"> Title </th> <th align="left" style="background-color:#ccc" width="150px"> Presenter </th> <th style="background-color:#ccc"> Abstract </th></tr> <tr> <td> Java software support <br />for SED-ML </td> <td> Richard Adams </td> <td> </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Leveraging Modeling <br />Approaches: Reaction <br />Networks and Rules </td> <td> Michael Blinov </td> <td> We have witnessed an explosive growth in research involving mathematical models and computer simulations of intracellular molecular interactions, ranging from metabolic pathways to signaling and gene regulatory networks. Many software tools have been developed to aid in the study of such biological systems, some of which have a wealth of features for model building and visualization, and powerful capabilities for simulation and data analysis. Novel high resolution and/or high throughput experimental techniques have led to an abundance of qualitative and quantitative data related to the spatiotemporal distribution of molecules and complexes, their interactions kinetics, and functional modifications. Based on this information, computational biology researchers are attempting to build larger and more detailed models. However, this has proved to be a major challenge. Traditionally, modeling tools require the explicit specification of all molecular species and interactions in a model, which can quickly become a major limitation in the case of complex networks – the number of ways biomolecules can combine to form multimolecular complexes can be combinatorially large. Recently, a new breed of software tools has been created to address the problems faced when building models marked by combinatorial complexity. These have a different approach for model specification, using reaction rules and species patterns. Here we compare the traditional modeling approach with the new rule-based methods. We make a case for combining the capabilities of conventional simulation software with the unique features and flexibility of a rule-based approach in a single software platform for building models of molecular interaction networks. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> The Liver Knowledgebase: <br />an integrated environment <br />for liver molecular network <br />representation and analysis </td> <td> Christian Bölling </td> <td> Liver functions are integrated from a vast array of individual biochemical activities and pathways in a seamless molecular interaction network across different cells and cell types. To study how those functions are integrated on the system’s level and how they feed back to individual molecular entities mathematical models can be used. The establishment and analysis of such models relies on a suitable representation of the liver molecular interaction network and all relevant meta-data, in particular evidence for the relations between network entities along with an integrated workflow and a toolbox for assembly, validation and evaluation of large metabolic network models. The Liver Knowledgebase (LKB) as part of the Virtual Liver Network aims at providing these resources. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> A Modular Semantic <br />Annotation Framework: <br />CellML Metadata <br />Specifications 2.0 </td> <td> Mike Cooling </td> <td> In the last decade or so, model encoding efforts such as CellML and SBML have greatly facilitated model availability. But, as the complexity of models increases, the utility of these models can vary. The addition of semantic information is crucial to transforming mathematical models from esoteric to informative resources. We have developed a metadata specification framework to better enable the annotation of CellML models with metadata. The framework consists of a core specification describing, in general terms, how annotations should be attached using RDF/XML, and satellite specifications covering several domains of immediate interest, using elements from the Dublin Core, FOAF (Friend-Of-A-Friend), BIBO (Bibliographic Ontology), MIRIAM URNs and Biomodels Qualifiers. We also describe what we see as several emerging challenges in the field, uncovered during the application of this annotation scheme to mathematical models. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Reactome: <br />a knowledgebase <br />of curated biological <br />pathways </td> <td> David Croft </td> <td> REACTOME is an open-source, open access, manually curated and peer-reviewed pathway database. Pathway annotations are authored by expert biologists, in collaboration with Reactome editorial staff and cross-referenced to many bioinformatics databases. The rationale behind Reactome is to convey the rich information in the visual representations of biological pathways familiar from textbooks and articles in a detailed, computationally accessible format. Examples of biological pathways in Reactome include signaling, innate and acquired immune function, transcriptional regulation, translation, apoptosis and classical intermediary metabolism. Reactome provides an intuitive website to navigate pathway knowledge and a suite of data analysis tools to support the pathway-based analysis of complex experimental and computational data sets. Visualisation of Reactome data is facilitated by the Pathway Browser, aninterface, that supports zooming, scrolling and event highlighting. Pathway and Expression Analysis tools analyze user-supplied datasets permitting ID mapping, pathway assignment and over-representation analysis. The curated human pathway data are used to infer orthologous events in 20 non-human species. Species Comparison tool allows users to compare predicted pathways with those of human to find reactions and pathways common to a selected species and human. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Novel developments<br /> in SBGN-ED <br />and applications </td> <td> Tobias Czauderna </td> <td> Systems Biology Graphical Notation (SBGN, <a rel="nofollow" class="external free" href="http://sbgn.org">http://sbgn.org</a>) [1] is an emerging standard for graphical representations of biochemical and cellular processes studied in systems biology. Three different views (Process Description, Entity Relationship, and Activity Flow) cover several aspects of the represented processes in different levels of detail. SBGN helps to communicate biological knowledge more efficient and accurate between different research communities in the life sciences. However, to support SBGN, methods and tools for editing, validating, and translating of SBGN maps are necessary. We present methods for these tasks and novel developments in SBGN-ED (www.sbgn-ed.org) [2], a tool which allows to create all three types of SBGN maps from scratch, to validate these maps for syntactical and semantical correctness, to translate maps from the KEGG database into SBGN, and to export SBGN maps into several file and image formats. SBGN-ED is based on VANTED (Visualization and Analysis of NeTworks containing Experimental Data, <a rel="nofollow" class="external free" href="http://www.vanted.org">http://www.vanted.org</a>) [3]. As applications of SBGN and SBGN-ED we present furthermore MetaCrop (<a rel="nofollow" class="external free" href="http://metacrop.ipk-gatersleben.de">http://metacrop.ipk-gatersleben.de</a>) [4], a database that summarizes diverse information about metabolic pathways in crop plants, and RIMAS (Regulatory Interaction Maps of Arabidopsis Seed Development, <a rel="nofollow" class="external free" href="http://rimas.ipk-gatersleben.de">http://rimas.ipk-gatersleben.de</a>) [5], an information portal that provides a comprehensive overview of regulatory pathways and genetic interactions during Arabidopsis embryo and seed development. [1] Le Novère, N. et al. (2009) The Systems Biology Graphical Notation. Nature Biotechnology, 27, 735-741. [2] Czauderna, T., Klukas, C., Schreiber, F. (2010) Editing, validating, and translating of SBGN maps. Bioinformatics, 26 (18), 2340-2341. [3] Junker, B.H., Klukas, C., Schreiber, F. (2006) VANTED: A system for advanced data analysis and visualization in the context of biological networks. BMC Bioinformatics, 7, 109+. [4] Grafahrend-Belau, E., Weise, S., Koschützki, D., Scholz, U., Junker, B.H., Schreiber, F. (2008) MetaCrop - A detailed database of crop plant metabolism. Nucleic Acids Research, 36, D954-D958. [5] Junker, A., Hartmann, A., Schreiber, F., Bäumlein, H. (2010) An engineer's view on regulation of seed development. Trends in Plant Science, 15(6), 303-307. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> SBRML- A Markup Language <br />for Encoding Systems <br />Biology Results </td> <td> Joseph Olufemi Dada </td> <td> Background: The wide acceptance of Systems Biology Markup Language (SBML) as standard format for models of biochemical systems makes it possible to build, share, evaluate and develop biochemical models cooperatively. However, there is currently no standard format for encoding the results of computational analyses of systems biology models. To address this issue, we present here the Systems Biology Results Markup Language (SBRML), an XML-based format for representing systems biology results. SBRML provides a means of representing computational simulation results as well as encoding experimental data in the context of a particular model. Methods &amp; Results: SBRML is based on XML and is specified through the XML Schema language. SBRML Object Model (SBRML-OM) was first developed using the Universal Modelling Language (UML), and a Model-Driven Architecture (MDA) approach was then used to derive the corresponding XML Schema semi-automatically with the help of mapping rules for classes and associations. SBRML can be used to describe any type of systems biology results, the model that is used to generate the results, the ontology terms that link all the terms/vocabularies used to an external ontology sources, the operation (e.g. time course, steady state etc.) that is performed on the model and a flexible means for encoding the results of the operation. SBRML associates a model with several data sets. Each data set consists of a series of values associated with model variables, and their corresponding parameter values. It provides a flexible way of indexing both simulation results and experimental data to model parameter values which supports both spreadsheet-like data or multidimensional data cubes. Some of its major uses are: associating experimental results with models for passing to analysis tools, sharing and archiving of model simulations and recording the results of analysis for validation, archiving or comparison. Conclusions: SBRML is a software-neutral language intended to allow any type of systems biology results to be represented. Standardization of systems biology results format will bring obvious benefits in terms of storage and retrieval, but will also benefit any program that needs to read data in the context of a biochemical model. Application areas are enzyme kinetics data, microarray gene expression data, and various types of simulation results. We will soon release a library (in various programming languages) for reading and writing SBRML documents. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> JSBML: a flexible <br />Java library for <br />working with SBML </td> <td> Andreas Dräger </td> <td> The specifications of the Systems Biology Markup Language (SBML) define standards for storing and exchanging computer models of biological processes in text files. In order to perform model simulations, graphical visualizations, and other software manipulations, an in-memory representation of SBML is required. We developed JSBML for this purpose. In contrast to prior implementations of SBML APIs, JSBML has been designed from the ground up for the Java(TM) programming language, and can therefore be used on all platforms supported by a Java Runtime Environment. This offers important benefits for Java users, including the ability to distribute software as Java Web Start applications. JSBML supports all SBML Levels and Versions through Level 3 Version 1, and we have strived to maintain the highest possible degree of compatibility with the popular library libSBML. JSBML also supports modules that can facilitate the development of plugins for end-user applications, as well as ease migration from a libSBML-based backend. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Normalization and <br />Matching of Chemical <br />Compound Names </td> <td> Martin Golebiewski </td> <td> We have developed ChemHits (<a rel="nofollow" class="external free" href="http://sabio.h-its.org/chemHits/">http://sabio.h-its.org/chemHits/</a>), an application which matches synonymic names of chemical compounds and thereby facilitates the bundling of data referring to the same compound. A chemical compound can have many different names - trivial, as well as systematic names. Hence, the identification of a chemical compound solely based on its name requires comprehensive chemical knowledge and often extensive searches in chemical databases. However, this identification is crucial for the integration of biochemical data, as many publications exclusively describe a compound by its name. The tool that we have developed is based on natural language processing (NLP) methods and applies rules to systematically normalize chemical compound names. Subsequently, matching of synonymous names is achieved by comparison of the normalized name forms. The tool is capable of normalizing a given name of a chemical compound and matching it against names in (bio-)chemical databases, like SABIO-RK, PubChem, ChEBI or KEGG, even when there is no exact name-to-name-match. Also complete lists of compound names can be processed which makes it useful for the automatic annotation of chemical data. The normalization and matching of various synonyms of a chemical compound as presented here constitute a platform for the unambiguous identification of compounds described in the literature or in databases. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Towards a web-based<br />simulation experiment<br />description repository </td> <td> Michael A. Guravage </td> <td> We have instantiated the SED-ML schema in a web-based content management system. With our system, you can create Simulation and Experiment Descriptions, enrich them with experimental data and annotate them with domain meta-information to facilitate classification, searching and cross referencing - all with the goal of reusing your models and reproducing your experimental results. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Managing Bio-Models: <br />Model Storage, Retrieval, <br />Ranking and Versioning </td> <td> Ron Henkel </td> <td> Models evolve, they are expanded, refined and adapted to new questions. With the large increase in the number of models published in systems biology, strategies for the re-use of models and the reproducibility of model-based results, or simulation experiments, become increasingly important. Building upon our experience with SBML, SED-ML and MIASE, we develop concepts for model management and specifically focus on: the use of meta-information in model storage, the use of information retrieval techniques to search for models in existing databases, the tracking of model evolution and the association of simulation experiments with models. Meta-information,for example, the further description of modeled entities, improves the search of models in databases, allows a comparison of models and also provides a basis for the merging of models. If appropriately stored, it enables a better understanding of the biological context in which the model is developed and how it is to be interpreted. Meta-information dramatically improves the search and comparison of models, helping users to re-use models and to reproduce model-based results in scientific publications. Our presentation will furthermore cover demands and solutions to track the model evolution using version control. The mean to store the whole evolution of a model enables researchers to follow its developments and to keep them aware of model updates. We will outline specifics of model version control and propose a conceptual solution. To date, the process of running a model to reproduce the desired results, e.g. as given in the publication, is mostly performed manually. Our presentation will discuss a format for the standardized encoding of simulation experiments and show how the integration of such information in model repositories will further support the modeling life cycle on systems biology. Our work demonstrates how the application of database and information retrieval techniques can support the field of systems biology, contributing to the exchange of models in research collaboration, improving the access of models in research and education and the reproducibility of scientific results that involve models. Although we have developed our tools and techniques with SBML in mind, they are applicable to other XML-based model encoding formats. All approaches have been implemented as prototypes. The code is freely available on Sourceforge from <a rel="nofollow" class="external free" href="http://bives.sourceforge.net/">http://bives.sourceforge.net/</a> and <a rel="nofollow" class="external free" href="http://sombi.sourceforge.net/">http://sombi.sourceforge.net/</a>. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> MIRIAM Registry and Identifiers.org </td> <td> Nick Juty </td> <td> We describe our recent work to provide the community with <p>directly resolvable URIs. As with the currently available MIRIAM URNs, which are well established and used, the identifiers.org URLs are based on the information stored in the MIRIAM Registry. Identifiers.org offers an annotation and cross-referencing framework which provides perennial, unambiguous and resolvable identifiers. </p> </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Implementation of <br />spatial model simulator <br />and its SBML support </td> <td> Tatsuhiro Matsui </td> <td> Our goal is to extend CellDesigner’s feature so that it can support spatial model simulation, and therefore supporting SBML spatial extension plays an important role in this project. In this research, we have made a spatial model simulator which imports an SBML model with spatial model extension (proposed in 2010.10.8) and simulate advection-diffusion-reaction equations. This simulator currently defines the shape of geometries including cells and nucleuses only with numerical formulae. We are planning to implement a feature that the shape and the geometries of the model will be imported from images, and to support up-to-date SBML </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Challenges in Modeling <br />and Curation of an <br />Influenza Viral Life Cycle <br />and Host Response Map </td> <td> Yukiko Matsuoka </td> <td> A systematic understanding of viral infection will facilitate the identification of inflammatory immune response mechanism and possible drug targets, which may selectively alter viral replication capacity. To understand the mechanisms of influenza viral replication and the host responses, we take the literature-based manual curation approach to construct a comprehensive influenza virus-host response map. Based on the Reactome (<a rel="nofollow" class="external free" href="http://www.reactome.org">http://www.reactome.org</a>) and KEGG (<a rel="nofollow" class="external free" href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>) pathways, the map is reconstructed with the pathway editor CellDesigner (<a rel="nofollow" class="external free" href="http://celldesigner.org">http://celldesigner.org</a>), stored in SBML Level 2 Version 4 format, enriched with literature-based information as well as inputs from the expert virologists and immunologists of the field. Literature references and curation comments are stored in the model file and can be retrieved via CellDesigner interface or via a web-based SBML model community curation platform called Payao (<a rel="nofollow" class="external free" href="http://www.payaologue.org">http://www.payaologue.org</a>). During the course of map building and curation, we have encountered various issues on, such as validating interpretation of the literature into the graphical representation, handling multi-level of granularities in notation, and merging parts of the map and consolidating the related annotations in both SBML notes as well as MIRIAM format. In this poster, we present the detail process to construct and curate a comprehensive map, elucidate issues and challenges we face during its process, and present further improvements in modeling and curation exercises. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Pathways and Semantics </td> <td> Ismael Navas-Delgado </td> <td> Biopax community is producing sets of data in RDF files, but most of them are not available through query interfaces. The publication of SPARQL endpoints is feasible with current sets of data, but the use of reasoning in these interfaces is unfeasible in many cases. The use of large scale reasoners is a need to take advantage of these data sets. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Rule-based and spatial <br />modeling on E-Cell <br />System Version 4 </td> <td> Kozo Nishida </td> <td> Rule-based and spatial modeling on E-Cell System Version 4 Kozo Nishida, Moriyoshi Koizumi, Satya Arjunan, Kazunari Kaizu, Koichi Takahashi The E-Cell System is a simulation platform for modeling and simulating intracellular biochemical reaction networks in silico. In the current version 3, an arbitrary number of sub-models running different algorithms at different time scales can coexist in a single model. However, it only partially supports representations of spatial aspects of the biochemical systems such as localization, diffusion and molecular crowding. Moreover, it lacks support for rule-based modeling of complex reaction networks. In the version 4, which is still under development, we aim to (i) enable modeling and simulation of cellular phenomena at the molecular resolution with particle-based simulation methods, (ii) fully support rule-based reaction networks, and (iii) include the interplay between biochemistry and biomechanics. New simulation algorithms already implemented on this platform, in addition to conventional ODE and CME-based methods, include the enhanced Green’s Function Reaction Dynamics (eGFRD), a highly accelerated reaction Brownian Dynamics method, and Spatiocyte, a very flexible and high-performance method based on microscopic lattice automata. In this poster, we mainly introduce our new rule-based modeling frontend on E-Cell. Rule-based modeling involves the representation of molecules as structured objects, and molecular interactions as rules for transforming the attributes of these objects. Not only is this approach nearly essential for easily and systematically incorporating site-specific details on protein-protein interactions into models for signal-transduction systems, but it is also useful for following fates of individual isotopic atoms in metabolic networks. We carefully designed our new rule-based infrastructure to be as compatible as possible to the de-facto BioNetGen language, but with some notable differences. Firstly, to allow seamless interactions with other components of the system and to avoid additional language requirements, the primary input format is in pure Python, our frontend scripting language. Secondly, since some of our new simulation methods are particle-based, we developed (and continuing to improve) a rule-based reaction network engine which can generate a part of the reaction network on-the-fly, in addition to generating the entire network at the pre-simulation stage. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> NineML: The Network <br />Interchange for Neuroscience <br />Modeling Language </td> <td> Ivan Raikov </td> <td> The growing number of large-scale neuronal network models has created a need for standards and guidelines to ease model sharing and facilitate the replication of results across different simulators. To foster community efforts towards such standards, the International Neuroinformatics Coordinating Facility (INCF) has formed its Multiscale Modeling program, and has assembled a task force of simulator developers to propose a declarative computer language for descriptions of large-scale neuronal networks. The name of the proposed language is "Network Interchange for Neuroscience Modeling Language" (NineML) and its initial focus is restricted to point neuron models. The INCF Multiscale Modeling task force has identified the key concepts of network modeling to be 1) spiking neurons 2) synapses 3) populations of neurons and 4) connectivity patterns across populations of neurons. Accordingly, the definition of NineML includes a set of mathematical abstractions to represent these concepts. NineML aims to provide tool support for explicit declarative definition of spiking neuronal network models both conceptually and mathematically in a simulator independent manner. In addition, NineML is designed to be self-consistent and highly flexible, allowing addition of new models and mathematical descriptions without modification of the previous structure and organization of the language. To achieve these goals, the language is being iteratively designed using several representative models with various levels of complexity as test cases. The design of NineML is divided in two semantic layers: the Abstraction Layer, which consists of core mathematical concepts necessary to express neuronal and synaptic dynamics and network connectivity patterns, and the User Layer, which provides constructs to specify the instantiation of a network model in terms that are familiar to computational neuroscience modelers. As part of the Abstraction Layer, NineML includes a flexible block diagram notation for describing spiking dynamics. The notation represents continuous and discrete variables, their evolution according to a set of rules such as a system of ordinary differential equations, and the conditions that induce a regime change, such as the transition from subthreshold mode to spiking and refractory modes. The User Layer provides syntax for specifying the structure of the elements of a spiking neuronal network. This includes parameters for each of the individual elements (cells, synapses, inputs) and the grouping of these entities into networks. In addition, the user layer defines the syntax for supplying parameter values to abstract connectivity patterns. The NineML specification is defined as an implementation-neutral object model representing all the concepts in the User and Abstraction Layers. Libraries for creating, manipulating, querying and serializing the NineML object model to a standard XML representation will be delivered for a variety of languages. The first priority of the task force is to deliver a publicly available Python implementation to support the wide range of simulators which provide a Python user interface (NEURON, NEST, Brian, MOOSE, GENESIS-3, PCSIM, PyNN, etc.). These libraries will allow simulator developers to quickly add support for NineML, and will thus catalyze the emergence of a broad software ecosystem supporting model definition interoperability around NineML. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> The System Biology <br />Format Converter </td> <td> Nicolas Rodriguez </td> <td> The System Biology Format Converter (SBFC) aims is to provide a generic framework that potentially allows any conversion between two formats. Interoperability between formats is a recurring issue in Systems Biology. Although there are various tools available to convert models from one format to another, most of them have been independently developed and cannot easily be combined, specially to provide support for more formats. The framework is written in Java and can be used as a standalone executable. This is a collaborative project and we hope that developers will provide support for more formats by creating new modules.SBFC allows anyone to easily add new converters and to integrate existing converters with a minimum of changes. We will also allow to combine several existing converters. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Proposal for an SBML <br />Level 3 Spatial Extension </td> <td> James Schaff </td> <td> This SBML spatial extension defines a common representation for cellular geometry, spatial mappings of species and reactions, and explicit species transport, and is equally applicable to deterministic or stochastic modeling. Geometry is specified as a coordinate system, a list of domains described topologically, and one or more alternate geometric definitions describing the shapes of each domain (e.g analytic functions, sampled fields, constructive solid geometry, and parametric shapes). We have developed a libSBML plugin to allow reading, writing, and manipulating SBML spatial models which is used within VCell 5.0. A detailed description of the current SBML L3 spatial extension can be found at <a rel="nofollow" class="external free" href="http://sbml.org/Community/Wiki/SBML_Level_3_Proposals/Spatial_Geometries_and_Spatial_Processes">http://sbml.org/Community/Wiki/SBML_Level_3_Proposals/Spatial_Geometries_and_Spatial_Processes</a>. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Curation and annotation <br />for BioModels Database, <br />a resource of published <br />quantitative models <br />in biology </td> <td> Michael Schubert </td> <td> BioModels Database (<a rel="nofollow" class="external free" href="http://www.ebi.ac.uk/biomodels">http://www.ebi.ac.uk/biomodels</a>) is a free resource for storing, viewing and retrieving quantitative, kinetic models described in peer reviewed publications. It uses the XML based Systems Biology Markup Language (SBML) for model storage and internal representation, but allows submission and export of models in various other commonly used formats. While models can be submitted publicly, submitted models are not directly included in the database, but checked and annotated both automatically and manually by the BioModels Database curation team. The database consists of two branches one for manually curated and annotated models and one for noncurated ones. On this poster we present the general buildup and working principles of the BioModels database, as well as use cases of curation on particular biological problems. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Conceptual model <br />of transcription machinery <br />in E. coli </td> <td> Anatoly Sorokin </td> <td> We have developed conceptual model of whole cell E. coli TRN with SBGN ER, SBGN PD and SBGN AF languages and going to discuss interconnectino between diagrams in different languages. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> An interactive <br />equation-based <br />biological model builder </td> <td> Akito Tabira </td> <td> Constructing mathematical model is becoming a standard approach to understand biological phenomena. Although many software tools have been developed to support this approach and are useful for analyzing or modifying existing models (from the databases or the papers), still they are not suitable for constructing models from scratch. To facilitate this from-scratch approach, we have developed a modeling tool. This tool takes differential equations as an input, and then allows researchers to simulate the model with various parameters and initial conditions. Also, this tool generates and visualizes biological networks automatically based on the inputs of the equations. This equation-driven model construction allows us to concentrate on the dynamics without considering the presence of the biological elements (e.g. molecules) beforehand. The simulating and analyzing functions of this tool support our understandings for the dynamics, and the automatic network generation makes us enable to ascertain the structure of the networks implying the equations. In order to be compatible with other software tools, we are planning to support import/export SBML files. Also, we are planning to improve the network’s visualization and implement the analyzing methods. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> LibSBGN: Electronic <br />Processing of SBGN maps </td> <td> Martijn van Iersel </td> <td> Introduction Graphical representations of participants and their relationships are essential for exchanging knowledge about complex biological processes. To convey this information clearly and unambiguously, it is necessary to assign standard meanings to symbols and their connectivity. For this purpose, the System Biology Graphical Notation (SBGN) has been developed. As SBGN is becoming more widely adopted, and used in various software tools, there is an increasing need for a standard file format able to capture the SBGN maps. Exchange using graphics-only file formats (such as SVG) is insufficient, because the biological meaning of elements is lost. There is a need for a toolset that enables diagram exchange while preserving biological meaning and relations. Results To meet this need, we are developing an Extensible Markup Language (XML) schema definition. In addition we are developing a software library called LibSBGN. Besides reading and writing files, this library will also be used to validate SBGN maps against the specifications, and convert to and from related systems biology standards, such as SBML and BioPAX. LibSBGN is still under development, but is already being adopted by several tools (See the LibSBGN project page for an up-to-date list of client software). The early adoption of LibSBGN by those tools helps to ensure that LibSBGN is independent of a specific software application, and does not contain artifacts for specific tools. LibSBGN is currently implemented in Java, a parallel C++ version is planned for the future. A test-suite of dozens of LibSBGN documents, covering every possible feature of SBGN maps, was created. This test-suite can be used by developers to check for compliance with the standard, and compare rendering capabilities with other tools. LibSBGN is a community effort, involving people from institutes all around the world, representing a wide selection of pathway tools. The community is organized around a sourceforge project site (<a rel="nofollow" class="external free" href="http://libsbgn.sourceforge.net">http://libsbgn.sourceforge.net</a>), a mailing list and monthly online meetings. A first release of LibSBGN, covering only the Process Description (PD) language, has been released in January 2011. Support for all three languages of SBGN is planned for a later release. </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> The Simulation Experiment <br />Description Markup Language </td> <td> Dagmar Waltemath </td> <td> </td></tr> <tr> <td colspan="3"> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> Kinetic Simulation <br />Algorithm Ontology </td> <td> Anna Zhukova </td> <td> I will present this poster remotely (via EVO or similar tool). To enable the accurate and repeatable execution of a computational simulation task, it is important to identify both the algorithm used and the initial setup. These minimum information requirements are described by the MIASE guidelines. Since the details of some algorithms are not always publicly available, and many are implemented only in a limited number of simulation tools, it is crucial to identify alternative algorithms with similar characteristics that may be used, in lieu, to provide comparable results in an equivalent simulation experiment. The Kinetic Simulation Algorithm Ontology (KiSAO) was developed to address this issue by describing existing algorithms and their inter-relationships through their characteristics and parameters. The use of KiSAO in conjunction with simulation descriptions, such as SED-ML, will allow simulation software to automatically choose the best algorithm available to perform a simulation. The availability of algorithm parameters, together with their type may permit the automatic generation of user-interfaces to configure simulators. </td></tr> </table> <!-- NewPP limit report Cached time: 20220802200424 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.060 seconds Real time usage: 0.061 seconds Preprocessor visited node count: 25/1000000 Preprocessor generated node count: 52/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  <div class="node-links">
      </div>

  
</div> <!-- /.node -->




          </div>

        
        
      </div></div> <!-- /.section, /#content -->

      <div id="navigation"><div class="section clearfix">

	           <div id="suckerfishmenu">
            <ul class="menu"><li class="leaf first"><a href="https://combine-org.github.io/events/" title="The annual COMBINE meeting will be held Oct 6-7, 2022, at the Humboldt University of Berlin, Germany.">COMBINE 2022</a></li>
<li class="leaf"><a href="/standards" title="Standards covered by or related to COMBINE activities">Standards</a></li>
<li class="leaf"><a href="/events" title="Events">Events</a></li>
<li class="collapsed"><a href="/documents" title="Documents">Documents</a></li>
<li class="leaf"><a href="/comm" title="Mailing lists and forums of discussion">Forums</a></li>
<li class="leaf last"><a href="/about" title="About">About</a></li>
</ul>          </div>
        
                  <div id="search-box">
            <form action="/events/COMBINE_2011/posters"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-nsTB7kh-FHx7lXPTX9EYMnOifrKB1jScaRbfKGZ9pJ4" value="form-nsTB7kh-FHx7lXPTX9EYMnOifrKB1jScaRbfKGZ9pJ4"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
          </div> <!-- /#search-box -->
        
        
      </div></div> <!-- /.section, /#navigation -->

      <div class="region region-sidebar-first column sidebar"><div class="section">
  
<div id="block-menu-menu-combine-left-menu" class="block block-menu first last region-odd odd region-count-1 count-1  block-1"><div class="block-inner">

      <h2 class="title">COMBINE</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first"><a href="/home" title="The home page of COMBINE">Home</a></li>
<li class="leaf"><a href="/help" title="">Help</a></li>
<li class="leaf last"><a href="/login" title="">Sign-in</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
</div></div><!-- /.section, /.region -->

      
    </div></div> <!-- /#main, /#main-wrapper -->

          <div id="footer"><div class="section">

        
        <div class="region region-footer">
  
<div id="block-block-1" class="block block-block first last region-odd even region-count-1 count-2  block-1"><div class="block-inner">

      <h2 class="title">COmputational Modeling in BIology NEtwork</h2>
  
  <div class="content">
    <p>COmputational Modeling in BIology NEtwork </p> <!-- NewPP limit report Cached time: 20220802130129 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.005 seconds Real time usage: 0.006 seconds Preprocessor visited node count: 1/1000000 Preprocessor generated node count: 4/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  
</div></div> <!-- /block-inner, /block -->
</div><!-- /.region -->
<div class="element-hidden" id="visible">
<a href="http://www.merrellpublishers.com/buy_meds.htm">cheap medicaments usa</a> 
<a href="http://www.drleanawen.com/styled-2/styled-9/commentary.html">generic sildenafil</a>
</div>
      </div></div> <!-- /.section, /#footer -->
    
  </div></div> <!-- /#page, /#page-wrapper -->

  
  
</body>
</html>
