
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>COMBINE 2016 Abstracts | COMBINE</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/sites/combine/files/Symbol-COMBINE_logo.ico" type="image/x-icon" />
  <link type="text/css" rel="stylesheet" media="all" href="/modules/node/node.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/poll/poll.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/defaults.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system-menus.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/user/user.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/ctools/css/ctools.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/panels/css/panels.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/views/css/views.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/html-reset.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/wireframes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/tabs.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/messages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/pages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/block-editing.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/blocks.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/navigation.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/nodes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/comments.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/forms.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/cti-flex2.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/combine-custom.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/layout-fixed.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/colors/combine.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/local.css?H" />
<link type="text/css" rel="stylesheet" media="print" href="/sites/all/themes/cti_flex2/css/print.css?H" />
<!--[if IE]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie.css?H" />
<![endif]-->
<!--[if lte IE 6]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie6.css?H" />
<![endif]-->
  <script type="text/javascript" src="/misc/jquery.js?H"></script>
<script type="text/javascript" src="/misc/drupal.js?H"></script>
<script type="text/javascript" src="/sites/all/modules/panels/js/panels.js?H"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/" });
//--><!]]>
</script>
  <style type="text/css">
body {font-family: Tahoma, Verdana, Arial, Helvetica, "Bitstream Vera Sans", sans-serif;}
</style>  <style type="text/css">
</style>
  <!--[if lte IE 6]>
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/suckerfish.js"></script>
  <![endif]-->
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/combine.js"></script>
</head>

<body class="page not-front not-logged-in node-type-hpage page-events-combine-2016-abstracts section-events one-sidebar sidebar-first page-panels ">

  <div id="page-wrapper"><div id="page">

    <div id="header"><div class="section clearfix">

      
              <div id="logo-title">

                      <div id="logo"><a href="/" title="Home" rel="home"><img src="/sites/combine/files/COMBINE_website-banner.png" alt="Home" id="logo-image" /></a></div>
          
      
        </div> <!-- /#logo-title -->
      
      
    </div></div> <!-- /.section, /#header -->

    <div id="main-wrapper"><div id="main" class="clearfix  with-navigation">

      <div id="content" class="column"><div class="section">

        
        
        <div class="breadcrumb"><a href="/">Home</a> › COMBINE 2016 Abstracts</div>                  <h1 class="title">COMBINE 2016 Abstracts</h1>
                                
        
        <div id="content-area">
          <div id="node-218" class="node node-type-hpage build-mode-full clearfix">

   
  
  
  
      <div class="meta">
              <span class="submitted">
          Submitted by <a href="/user/1719" title="View user profile.">pzuliani</a> on Mon, 02/06/2017 - 14:21        </span>
      
          </div>
  
  <div class="content">
    <h2><span class="mw-headline" id="Invited_Talks">Invited Talks</span></h2> <p><i><b>Delivering cardiac computational biology to industry and the clinic: Advances towards reshaping future health and care</b></i> <br /> Alfonso Bueno-Orovio<br /> University of Oxford </p><p>Both biomedical research and clinical practice rely on complex datasets for the characterization of human health and tailoring and design of therapy. Given the complexity and variety of approaches and recordings, there is an increasing recognition of the need to embed computational methods in medicine and science for analysis, integration and prediction. This talk will explore the role and maturity of Computational Biology in facilitating a shift towards predictive human-based methodologies in cardiovascular medicine, spurred by advances in computational and experimental techniques, as well as by the growing acknowledgement of the limitations of animal models. Representative examples will illustrate how computational approaches complement, expand, bridge, and integrate experimental and clinical data and methods, and as such they are called to become an integral part of human-based methodologies in pharmacology and medicine. </p> <hr /> <p><i><b>FAIRDOM - FAIR asset management and sharing experiences in Systems and Synthetic Biology</b></i> <br /> Carole Goble<br /> University of Manchester </p><p>"Over the past 5 years we have seen a change in expectations for the management of all the outcomes of research – that is the “assets” of data, models, codes, SOPs and so forth. Don’t stop reading. Data management isn’t likely to win anyone a Nobel prize. But publications should be supported and accompanied by data, methods, procedures, etc. to assure reproducibility of results. Funding agencies expect data (and increasingly software) management retention and access plans as part of the proposal process for projects to be funded. Journals are raising their expectations of the availability of data and codes for pre- and post- publication. The multi-component, multi-disciplinary nature of Systems Biology demands the interlinking and exchange of assets and the systematic recording of metadata for their interpretation. The FAIR Guiding Principles for scientific data management and stewardship (<a rel="nofollow" class="external free" href="http://www.nature.com/articles/sdata201618">http://www.nature.com/articles/sdata201618</a>) has been an effective rallying-cry for EU and USA Research Infrastructures. FAIRDOM (Findable, Accessible, Interoperable, Reusable Data, Operations and Models) Initiative has 8 years of experience of asset sharing and data infrastructure ranging across European programmes (SysMO and EraSysAPP ERANets), national initiatives (de.NBI, German Virtual Liver Network, UK SynBio centres) and PI's labs. It aims to support Systems and Synthetic Biology researchers with data and model management, with an emphasis on standards smuggled in by stealth and sensitivity to asset sharing and credit anxiety. </p><p>This talk will use the FAIRDOM Initiative to discuss the FAIR management of data, SOPs, and models for Sys Bio, highlighting the challenges of and approaches to sharing, credit, citation and asset infrastructures in practice. I'll also highlight recent experiments in affecting sharing using behavioural interventions. </p><p><a rel="nofollow" class="external free" href="http://www.fair-dom.org">http://www.fair-dom.org</a> <a rel="nofollow" class="external free" href="http://www.fairdomhub.org">http://www.fairdomhub.org</a> <a rel="nofollow" class="external free" href="http://www.seek4science.org">http://www.seek4science.org</a>" </p> <hr /> <p><i><b>COMBINE and ISO/TC 276 Biotechnology: From grassroots community standards to official ISO standards</b></i><br /> Martin Golebiewski <br /> HITS gGmbH, Heidelberg, Germany </p><p>The whole data lifecycle in systems biology requires a high degree of standardization of formats for data, models and metadata: Acquisition and description of the data, processing and analysis, their efficient and secure exchange, data integration and incorporation into computational models, as well as the setup, handling and simulation of the models that all have to follow dedicated standards. To this end many grassroots community standards for exchange formats and metadata description have been defined by different scientific communities in the field. However, it often is confusing and cumbersome for the potential users (e.g. experimentalists or modelers) to find the appropriate standards for their tasks and apply them in their workflows. </p><p>In light of its rapid growth and market relevance, the International Organization for Standardization (ISO) has identified biotechnology as an area of major potential for standardization. In December 2013 the technical committee of ISO for biotechnology standards (ISO/TC 276) had its inauguration meeting. Where is the journey of ISO/TC 276 Biotechnology heading? How could existing scientific community standards be of use for the definition of official ISO standards? What are the benefits for the grassroots communities in doing so? In my talk I will outline options for classifying and integrating existing community standards into official norms and how that might help in defining a framework and guideline for community standards and their application in different use cases. </p> <hr /> <p><i><b>From Networks to Function Computational Models of Organogenesis</b></i><br /> Dagmar Iber<br /> ETH Zurich </p><p>One of the major challenges in biology concerns the integration of data across length and time scales into a consistent framework: how do macroscopic properties and functionalities arise from the molecular regulatory networks ­and how do they evolve? Morphogenesis provides an excellent model system to study how simple molecular networks robustly control complex pattern forming processes on the macroscopic scale in spite of molecular noise, and how important functional variants can evolve from small genetic changes. Recent advances in 3D imaging technologies, computer algorithms, and computer power now allow us to develop and analyse increasingly realistic models of biological control. In my talk, I will show how data-based modelling can be used to define mechanisms for fundamental developmental processes such as the control of branching processes and the scaling of developmental pattern on differently sized embryonic domains. </p> <hr /> <p><i><b>SBML in 2016</b></i><br /> Sarah Keating<br /> EMBL-EBI </p><p>SBML (the Systems Biology Markup Language) was created in 2000 and has since then become a lingua franca for encoding and exchanging computational models in biology. Since its introduction, SBML has gained support from a range of software packages for modeling, and numerous databases today make publicly available published models encoded in the format. These developments continue to promote and drive the evolution of SBML. This talk will briefly outline the format as it stands in 2016 (SBML Level 3) and give an overview of the capabilities that have been added to the format and those that are in the developmental stages. It will also provide an update on the software packages provided by the SBML Team to facilitate the infrastructure necessary to support adoption and use of SBML. </p> <hr /> <p><i><b>What infrastructure to cross the genotype-to-phenotype gap: foundations or flying buttresses?</b></i><br /> <i>Andrew J. Millar</i>1, Ricardo Honorato-Zimmer2, Ally Hume1,3, Robert Muetzelfeldt1,4, Gordon Plotkin2, Uriel Urquiza1, Argyris Zardilis1, Tomasz Zielinski1<br /> 1 SynthSys and School of Biological Sciences, University of Edinburgh, Edinburgh, Scotland. 2 SynthSys and School of Informatics, University of Edinburgh.3 EPCC, University of Edinburgh. 4 Simulistics Ltd., Edinburgh, Scotland. </p><p>Abstract<br /> My group and SynthSys’ Systems Biology research includes infrastructure support for data (e.g. www.biodare.ed.ac.uk, refs. 2, 3), models (e.g. www.plasmo.ed.ac.uk) and other intellectual assets (via FAIRdom). I will contrast ‘foundational’ infrastructure, underlying projects at a low level, to support by ‘flying buttresses’ where infrastructure is added to mature projects. A key use case involves the 24-hour circadian clock, which controls biological processes from the sleep-wake cycle to the cell cycle. We seek to understand how the dynamics of the intracellular clock gene circuit control growth, biomass and life history in whole plants, using the lab model plant species, Arabidopsis thaliana. In future, this should help to understand the ecological and breeding pressures on the clock genes. The rich data of the Arabidopsis community has allowed us and our partners in the BBSRC ROBuST and EU FP7 TiMet projects to model rhythmic processes between germination and flowering, including the crucial, nightly utilisation of starch carbon stores (4). We combined four existing models into the Arabidopsis Framework Model (FMv1), which predicts biomass quantitatively (1). We are now using FMv2 to understand quantitatively the pleiotropic phenotypes of a ‘slow’ clock mutant, bridging genotype to phenotype. The FM’s challenges for simulation tools include hourly data input during simulation, and dynamically creating compartments (leaves). Simulistics’ Simile environment supports declarative modelling from a ‘systems dynamics’ background, with object-oriented features, a graphical interface and model composition. Simile provides full control of the FMv1. I will discuss a different, online approach to support the developing FMv2, and the early development of Chromar, a formal modelling language to express models of this type. </p><p>Links to Data/Model/Software <br /> www.simulistics.com <br /> www.plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_76 </p><p>Selected references, with hyperlinks <br /> 1. Chew YH, Wenden B, Flis A, Mengin V, Taylor J, et al. 2014. Multiscale digital Arabidopsis predicts individual organ and whole-organism growth. Proc Natl Acad Sci U S A 111:E4127-36. <a rel="nofollow" class="external free" href="http://dx.doi.org/10.1073/pnas.1410238111">http://dx.doi.org/10.1073/pnas.1410238111</a> <br /> 2. Flis A, Fernandez AP, Zielinski T, Mengin V, Sulpice R, et al. 2015. Defining the robust behaviour of the plant clock gene circuit with absolute RNA timeseries and open infrastructure. Open Biol 5. <a rel="nofollow" class="external free" href="http://dx.doi.org/10.1098/rsob.150042">http://dx.doi.org/10.1098/rsob.150042</a> <br /> 3. Practical evaluation of SEEK and OpenBIS for biological data management in SynthSys. Troup, E., Clark, I., Swain, P., Millar, A.J. &amp; Zielinski, T. SynthSys Technical Report. <a rel="nofollow" class="external free" href="http://dx.doi.org/1842/12236">http://dx.doi.org/1842/12236</a> <br /> 4. Seaton DD, Smith RW, Song YH, MacGregor DR, Stewart K, et al. 2015. Linked circadian outputs control elongation growth and flowering in response to photoperiod and temperature. Mol Syst Biol 11:776. <a rel="nofollow" class="external free" href="http://dx.doi.org/10.15252/msb.20145766">http://dx.doi.org/10.15252/msb.20145766</a> </p> <hr /> <p><i><b>A Standard Enabled Workflow for Synthetic Biology</b></i><br /> Chris Myers<br /> University of Utah </p><p>A synthetic biology workflow is composed of data repositories that provide information about genetic parts, sequence-level design tools to compose these parts into circuits, and system-level design tools to construct and analyze models of complete designs. Data standards enable the ready exchange of information within such a workflow. They allow repositories and tools to be connected from a diversity of sources. This talk will describe one such workflow that utilizes the Synthetic Biology Open Language (SBOL) to describe genetic designs, the Systems Biology Markup Language (SBML) to model these designs, and SBOL Visual to visualize these designs. This workflow includes multiple repositories, such as the Newcastle SBOL Stack and the JBEI ICE Repository, and multiple software tools such as SBOLDesigner and iBioSim, among others. Recently, the ACS Synthetic Biology journal has recommended the use of SBOL in their publications. This talk will demonstrate how this workflow can be used to produce this type of design information. </p> <hr /> <p><i><b>CellML, SED-ML, and the Physiome Model Repository</b></i><br /> David Nickerson<br /> University of Auckland </p><p>CellML is an XML-based protocol for storing and exchanging computer-based mathematical models in an unambiguous, modular, and reusable manner. In addition to introducing CellML, in this presentation I will provide some of physiological examples that have help drive the development and adoption of CellML. I will also present the proposed changes being introduced in the next version of the specification, CellML 2.0. The Simulation Experiment Description Markup Language (SED-ML) is an XML-based format for encoding simulation setups to ensure exchangeability and reproducibility of simulation experiments. Here I will present an overview of the current version of SED-ML (Level 1 Version 2) and discuss the additional features introduced in the draft Level 1 Version 3 specification, as well as some of other upcoming features often requested. I will also demonstrate some of the functionality and content being added to the Physiome Model Repository which makes extensive use of both CellML and SED-ML to provide a rich repository of reproducible model descriptions. </p> <hr /> <p><i><b>Modelling ageing to enhance healthy lifespan</b></i><br /> Daryl Shanley<br /> Newcastle University </p><p>The ageing process is complex and multifactorial. The proximate cause of ageing is the accumulation of damage to component molecules, organelles, membranes and cells leading to loss of function in tissues and organs. How such damage arises, how multiple sources of damage affect each other, and how physiological function is affected are all highly complex1. Living systems are evolutionarily programmed for survival, even though such programming does not extend to living forever, and there is plasticity in the capacity to survive, depending on environment, lifestyle, etc. It has long been known that interventions such as caloric restriction and mutations in certain genes reveal plasticity in longevity in short-lived mammals (i.e. mice). To benefit human health and longevity we need to know how this plasticity can be targeted, but again we are faced with complexity. I will present an overview of our efforts to address this challenge using an integrated laboratory and computational modelling approach. The focus of our work has been incremental such as our work on disentangling the dynamics of the nutrient signalling network2-4, integrating mechanism of aging5, and in other work we have made ambitious attempts to model ageing in whole cells6. My presentation will focus mostly on deterministic ordinary differential equation modelling, but will include some work using rule-based approach, stochastic simulation and cover issues such as parameter identifiability and with spatial localisation. </p><p>References<br /> 1 Kirkwood, T. B. L. A systematic look at an old problem. Nature 451, 644-647 (2008).<br /> 2 Dalle Pezze, P. et al. A Dynamic Network Model of mTOR Signaling Reveals TSC-Independent mTORC2 Regulation. Science Signaling 5, ra25-ra25 (2012).<br /> 3 Sonntag, A. G., Dalle Pezze, P., Shanley, D. P. &amp; Thedieck, K. A modelling-experimental approach reveals insulin receptor substrate (IRS)-dependent regulation of adenosine monosphosphate-dependent kinase (AMPK) by insulin. The FEBS journal 279, 3314-3328 (2012).<br /> 4 Dalle Pezze, P. et al. A systems study on amino acid stimulation reveals concurrent activation of AMPK and mTOR converging on ULK1 and autophagy. Nature Communications, in press (2016).<br /> 5 Dolan, D. W. P. et al. Integrated Stochastic Model of DNA Damage Repair by Non-homologous End Joining and p53/p21- Mediated Early Senescence Signalling. PLoS Comput Biol 11, e1004246 (2015).<br /> 6 Dalle Pezze, P. et al. Dynamic Modelling of Pathways to Cellular Senescence Reveals Strategies for Targeted Interventions. PLoS Comput Biol 10, e1003728 (2014). </p> <hr /> <p><i><b>Control of epileptic seizures - a computational perspective</b></i><br /> Yujiang Wang<br /> Newcastle University </p><p>Epilepsy is a crippling disease that affects over 50 million people world-wide. Currently available clinical treatments are only effective in about 70% of people, and far less are treated without detriment to their quality of life. My work aims to understand the mechanisms of the disease better, and to suggest novel treatment to control epileptic seizures. To achieve this I use neuroimaging data and signal processing methods to extract relevant information, which inform dynamic computational models of epileptic seizures. Such models can be used to investigate mechanisms of the disease, to find new treatment strategies, or to tailor existing treatment to patient-specific needs. In my talk I will give a few examples of this approach, and also an outlook of how my proposed approaches can beneficial in the treatment of a wider range of neurological disorders. </p><p><br /> </p> <h2><span class="mw-headline" id="Contributed_Talks">Contributed Talks</span></h2> <p><i><b>Introducing FSK-ML - an SBML derivate for description and exchange of executable script based models</b></i><br /> <i>Miguel de Alba</i>, Guido Correia Carreira, Alexander Falenski, Annemarie Ksbohrer and Matthias Filter<br /> Federal Institute for Risk Assessment (BfR), Germany </p><p>SBML is the leading standard for models in Systems Biology. Unlike there, other scientific areas face the challenge that the majority of models have been developed in specific scripting languages like R or Matlab. In order to promote the standardized description and exchange of models in the domain of microbial risk assessment (MRA) we developed the Food Safety Knowledge Markup Language (FSK-ML) by extension and adaption of SBML . FSK-ML now allows to export and exchange executable script-based MRA models with well-structured meta-data. Built on top of the Combine Archive concept, the FSKX archive serves as a container to provide meta-data on MRA models, default values, simulation scenarios, experimental data or data generated by running a model simulation. Models programmed in scripting languages, such as R, Python or Matlab, are stored together with linked resources (e.g. R libraries) allowing to execute the original model scripts in a proper simulation environment. Finally a reference implementation using JSBML has been developed and will be showcased. A number of example applications have been deployed in the open-source data analytics platform Konstanz Information Miner (KNIME) making use of it. </p> <hr /> <p><i><b>Approaches for Developing Computational Models of Immune System Formation and Function</b></i><br /> Kieran Alden<br /> York University </p><p>The application of computational modelling is becoming prevalent in studies that aim to understand how complex interactions between diverse cell types and their environment influence our health. Work in the York Computational Immunology Lab focuses on both the development of computational models of immune system formation, such as the development of lymphoid organs, and function, particularly in Inflammatory Bowel Diseases and Leishmaniasis, in addition to the development of software tools and approaches that assist in the composition and analysis of these models. Although the majority of our models adopt the agent-based modelling paradigm, as the complexity of our models increases, we are beginning to adopt a hybrid approach, integrating other models (such as SBML models) within the agent-based structure. As such, and to better describe our in silico experimentation, we are beginning the process of making our tools (Spartan, ASPASIA, and others) compatible with appropriate COMBINE standards. In this talk an overview of the model design and analysis approaches we adopt will be provided, focusing on a short case study on how we have used ASPASIA, a tool-kit for evaluating the effects of biological interventions on SBML model behaviour, to gain insights into how Th17-cell plasticity is controlled, prior to integrating the SBML model into an individual focused one that permits an exploration of Inflammatory Bowel Diseases. </p> <hr /> <p>'<b>Version and Variant Control for Synthetic Biology'</b><br /> <i>Bryan Bartley</i> (University of Washington, Dept. of Bioengineering), Michal Galdzicki (Arzeda Corp., Seattle, WA), John Gennari (University of Washington, Biomedical &amp; Health Informatics), Herbert Sauro (University of Washington, Dept. of Bioengineering) </p><p>Version control systems have become invaluable for managing code complexity in software development projects. Version control for synthetic biology could prove similarly valuable, as synthetic biology projects grow in complexity. We discuss how versioning data for a genetic circuit may be captured using the Synthetic Biology Open Language and present a use case based on construction of a 4-module circuit. Versioning requires description of provenance information as well as details about the biological host of a circuit. Additionally we introduce variant control, an entirely new type of management system that addresses the needs of synthetic biologists to manage variant designs. Variant control leverages semantic annotations captured in SBOL to cluster similar designs together in design space. Variant design could help synthetic biologists explore design space and discover design rules. </p> <hr /> <p><i><b>Progress Report on Development of SBOL Visual 2.0</b></i><br /> Bryan Bartley (University of Washington), <i>Jacob Beal</i> (Raytheon BBN Technologies), Mike Bissell (Amyris), Kevin Clancy (Thermo Fisher Scientific), Robert Sidney Cox (Kobe University), Raik Grunberg (University of Montreal), Chris Myers (University of Utah), Nicolas Le Novere (Babraham Institute), Matthew Pocock (Turing Ate My Hamster, Ltd.), Chris Voigt (MIT), Zach Zundel (University of Utah) </p><p>People engineering biological organisms often need to communicate in diagrams, both about the structure of nucleic acid sequences and about functional relationships between components of such sequences with one another or with other biological substances. This talk provides an update on progress in developing SBOL Visual 2.0, which extends the prior structure-focused SBOL Visual 1.0 toward these goals, as well as drawing on SBGN in its approach to interactions. </p><p>Some typical practices and conventions have begun to emerge for such diagrams. SBOL Visual 2.0 aims to organize and systematize such conventions in order to produce a coherent language for expressing both structure and function of genetic designs. At the same time, the language's design aims to maximize the applicability of this language by allowing as much flexibility and freedom as possible in how such diagrams are organized, presented, and styled: in particular, it should be readily possible to create diagrams both by hand and with a variety of software programs. Finally, means will be provided for extending the language with new and custom diagram elements, and for adoption of useful new elements into the language. </p><p>Every element of a SBOL Visual 2.0 should correspond to an element of the SBOL 2 data model. As such, an SBOL Visual diagram will potentially be able to be parsed to create an SBOL data construct (though this may be impractical for many formats in practice). Not all SBOL data constructs will be able to be mapped into SBOL visual, however, and in fact attempting a complete mapping is generally undesirable, as effective communication will often involve selecting or emphasizing only certain aspects of a complete design, particularly when dealing with complex or multi-layered systems. </p> <hr /> <p><i><b>Tellurium: A Python Based Modeling and Reproducibility Platform for Systems Biology</b></i><br /> <i>Kiri Choi</i> 1, J. Kyle Medley 1, Caroline Cannistra 1, Matthias Konig 2, Lucian Smith 1, Kaylene Stocking 1, Herbert M. Sauro 1 <br /> 1 Department of Bioengineering, University of Washington<br /> 2 Institute for Theoretical Biology, Humboldt University of Berlin<br /> </p><p>Tellurium is a powerful Python-based integrated environment designed for model building, analysis, simulation and reproducibility in systems and synthetic biology. Tellurium is a modular, cross-platform, and open-source integrated development environment (IDE) composed of multiple libraries, plugins, and specialized modules and methods. Tellurium ensures exchangeability and reproducibility of computational models by supporting SBML (Systems Biology Markup Language), SED-ML (Simulation Experiment Description Markup Language), the COMBINE archive, and SBOL (Synthetic Biology Open Language). Tellurium is a self-contained modeling platform which comes with a fully configured Python distribution independent of other local Python installations on the target machine. Tellurium comes with various libraries, including but not limited to: libRoadRunner, Antimony, phraSED-ML, COBRApy, NumPy, SciPy, and etc. Additionally, we include several user-friendly plugins and advanced modules for a wide-variety of applications, ranging from visualization tools to complex algorithms for bifurcation analysis and multi-dimensional parameter scanning. By combining multiple libraries, plugins, and modules into a single platform, Tellurium provides a unified but extensible solution for biological modeling and simulation. </p> <hr /> <p><i><b>The Visual Protein Design Language</b></i><br /> <i>Robert Sidney Cox III</i>, Raik Gruenberg<br /> Caltech </p><p>We present a set of glyphs and graphical objects for representing the designs of engineered proteins, along with a set of drawing constraints. These objects allow the representation of globular protein domains, linkers and loop regions, as well as membrane spanning and anchor domains. Additional glyphs allow for the annotation of protein recognition sequences or tags including those on the N-terminus or C-terminus, enzymatic sites, points of noncovalent contact, covalent modifications by small molecules, cleavage sites, and protein stability elements. We provide a rich set of symbols which can be used with labels to describe many of the possible types of protein designs, while keeping the total number of glyphs and shapes small and easy to draw. </p><p>These glyphs are intended to be used in a prescriptive way, and have precise requirements on size scaling, alignment to the drawn line, and line width to distinguish protein design elements from other graphical features shown on the same diagram. We also prescribe a set of label characters for a limited number of variants of each glyph that the user can annotate. Variations are suggested for hand drawn versions of the glyphs and figures. While the drawing constraints are prescriptive, their use or absence is decided by the user or tool depending on the aims of communication. Thus a domain or region might be left unannotated, or several parts of a catalytic site could be annotated at relative locations along the protein line. Our primary aim is to allow direct communication of the most relavent features of a protein design, while providing a clear and rigorous way to draw protein designs which are not too ugly or cluttered. </p> <hr /> <p><i><b>The ZBIT Systems Biology Software and Web Service Collection</b></i><br /> Andreas Draeger<br /> Center for Bioinformatics Tuebingen, Applied Bioinformatics Group, University of Tuebingen, Germany </p><p>Background: Networks and models in systems biology today have been scaled up to the sizes of full genomes. They comprise thousands of reactions, metabolites, regulatory events, and many further biochemical components. In order to build, analyze, and explore networks and models on this scale, highly specific software solutions are required. </p><p>Results: We introduce the systems biology software collection that has been developed and continuously maintained at the Center for Bioinformatics Tuebingen (ZBIT) over more than one decade now. All tools have been created to solve research questions in ongoing large-scale systems biology projects with numerous national and international collaborators. These tools cover all aspects of the model life cycle. KEGGtranslator and BioPAX2SBML can be used to gather data for building draft networks based on the KEGG PATHWAY database and files in BioPAX format. SBMLsqueezer generates kinetic equations and derives the units for the parameters therein. SBMLsimulator unifies the Simulation Core Library and the heuristic optimization toolbox EvA2 for model simulation and calibration. ModelPolisher accesses the BiGG Models knowledge-base for model annotation. SBML2LaTeX documents models by creating human-readable reports. </p><p>Conclusion: All tools presented are freely available and can either be used as online programs at <a rel="nofollow" class="external free" href="http://webservices.cs.uni-tuebingen.de">http://webservices.cs.uni-tuebingen.de</a> using any common web browser, or for download as desktop programs from <a rel="nofollow" class="external free" href="http://www.cogsys.cs.uni-tuebingen.de/software/">http://www.cogsys.cs.uni-tuebingen.de/software/</a>. These tools have been used in several research projects, including community efforts, such as the path2models project. Their development has often been a driving force for further community efforts, such as the JSBML project. </p> <hr /> <p><i><b>MultiCellDS: a community-developed standard for curating microenvironment-dependent multicellular phenotype data as digital cell lines, snapshots, and collections</b></i><br /> <i>Samuel Friedman</i>, David Agus, Paul Macklin <br /> University of Southern California </p><p>Medicine and biology have become increasingly multi-institutional and collaborative. This accelerates scientific progress when research communities pool their resources where consortia can recruit more patients to clinical studies, create large datasets and assemble measurements from instruments that are prohibitive for individual institutions to build, and integrate more diverse sets of expertise. </p><p>Research communities gain further when adopting centralized data repositories and standardized data elements. Data and insight sharing are less efficient when communicated by direct member-to-member interactions with non-standard data, and scientific advancement often suffers. Shared repositories spread data archiving and maintenance costs, while data standards allow shared data analysis and visualization tools; together, they allow the broader community to bring fresh expertise and insights to the field. </p><p>The work has not yet reached its fullest potential. Lacking a standardized data representation and centralized repositories to systematically report these measurements, data becomes undiscoverable, difficult to adapt for computer simulations, and precludes further community-driven extensions of the original data. This reflects an unmet need in biology at any non-molecular scale: phenotypic measurements from multicellular experiments remain trapped in individual papers. </p><p>We present MultiCellDS (MultiCellular Data Standard): a community-developed standard that functionally describes cell phenotypes with contextual information from the microenvironment. The refined standard supports a library of over 200 digital cell lines: a hierarchical, extensible representation of a biological cell line's phenotype in one or more microenvironmental contexts; of digital snapshots: a hierarchal representation of multicellular data of various forms coming from clinical, experimental, and simulation domains; and of digital collections: a way to organize the wide variety of data from a multitude of sources. With customized levels of detail, MultiCellDS enables the pooling and curation of measurements and insights across many sources into a standardized, searchable repository. </p> <hr /> <p><i><b>AMICI: An ODE simulation framework for sensitivity analysis of large-scale models</b></i><br /> <i>Fabian Fröhlich</i>(1,2) and Jan Hasenauer(1,2)<br /> (1) Institute of Computational Biology, Helmholtz Zentrum Munchen, Ingolstatdter Landstrasse 1, 85764 Neuherberg, Germany, (2) Chair of Mathematical Modeling of Biological Systems, Center for Mathematics, Technische Universitat Munchen, Boltzmannstrasse 3, 24105 Garching, Germany </p><p>Ordinary differential equations (ODE) are commonly used to model the dynamic behavior of biochemical reaction networks. Forward sensitivities of ODE models can give additional insight into the model and are often used in parameter estimation and experimental design. For large scale networks with hundreds or thousands of parameters, forward sensitivities are typically computationally not tractable as the computation time scales linearly with respect to the number of parameters. For a set of benchmark problems we show that, in contrast to forward sensitivities, adjoint sensitivities scale almost constant with respect to the number of parameters and are tractable to compute for systems with thousands of parameters and state variables. The respective methods are implemented in the Advanced MATLAB Interface for CVODES and IDAS (AMICI). AMICI is a MATLAB toolbox, which implements both forward and adjoint sensitivities as well as second order sensitivities. AMICI can import SBML models and supports sensitivity computation for models with events and assignment rules. Optimized Native C code is generated for every model and efficient simulation routines are compiled as standalone .mex files and could be used by other toolboxes. These routines and enable model analysis such as parameter estimation or experimental design for pathway- to genome-scale models. </p> <hr /> <p><i><b>The NormSys registry for modeling standards in systems and synthetic biology</b></i><br /> Martin Golebiewski<br /> HITS gGmbH, Heidelberg, Germany </p><p>To survey standard formats for computational modeling in biology such as the COMBINE core standards and others we have developed a registry which not only lists the standards, but also compares their major features, their possible fields of biological application and use cases (including model examples), as well as their relationships, commonalities and differences. This NormSys registry for modeling standards (<a rel="nofollow" class="external free" href="http://normsys.h-its.org">http://normsys.h-its.org</a>) provides a common entry point for modelers and software developers who plan to apply the standards for their respective case of application, and serves them with detailed information and links to the standards, their specifications and APIs. </p> <hr /> <p><i><b>cy3sbml: a Cytoscape app for SBML</b></i><br /> Matthias Koenig<br /> Institute for Theoretical Biology, Humboldt University Berlin, Berlin, Germany </p><p>cy3sbml is an app for the work with SBML in Cytoscape 3 providing the following functionality: SBML import; support of SBML Core and the Layout, Qualitative Model, Groups, Hierarchical Model Composition, and Flux Balance Constraints packages; reaction-species and extended SBML graphs; access to MIRIAM and SBO-based annotations; use of SBML attributes for visual mappings, network filtering and visualization; SBML validation. cy3sbml includes importer for BioModels and BiGG models to load SBML from standard repositories. In addition to porting all functionality to Cytoscape 3 many new features have been implemented since the initial Cytoscape 2 release<br /> - support of the additional SBML packages comp and groups as well as fbc version 2 <br /> - implementation of kinetic graphs in addition to species-reaction graphs, providing insights into the relationship between parameters, kinetic laws, reactions, etc.<br /> - resolving additional annotation information from primary annotations using the ontology lookup service (OLS) <br /> - Integration with additional apps and systems biology workflows via OSGI APIs and commands accessible via cyREST <br /> - redesign of GUI using web technology (JavaFx WebView) <br /> - integration with BiGG database <br /> - simple installation via app store </p><p>Availability and implementation: Freely available for non-commercial purposes from the Cytoscape App Store at <a rel="nofollow" class="external free" href="http://apps.cytoscape.org/apps/cy3sbml">http://apps.cytoscape.org/apps/cy3sbml</a> </p><p>Supplementary information: Source code and additional information at <a rel="nofollow" class="external free" href="https://github.com/matthiaskoenig/cy3sbml/">https://github.com/matthiaskoenig/cy3sbml/</a> </p> <hr /> <p><i><b>A new thermodynamics-based approach to aid the design of natural-product biosynthetic pathways</b></i><br /> <i>Hiroyuki Kuwahara</i>, Xuefeng Cui, Meshari Alazmi, and Xin Gao<br /> King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Thuwal, 23955, Saudi Arabia </p><p>Recent advances in genome editing and metabolic engineering enabled the engineering of de novo biosynthetic pathways for high-value natural products such as those for biofuels, cosmetics, perfumes, and drugs. Because of a large search space for potential metabolic routes to produce a given natural product, the development of a computational method to rank biosynthetic pathways for a given chassis is essential to the engineering of productive heterogeneous biosynthesis systems. One important design criterion for biosynthetic pathway designs is often concerned with the conversion rate of a given starting material to produce a target product in a host organism. Since quantitative information required to construct kinetic models of such metabolic systems is seldom available, commonly applied approaches to assess this productivity criterion have been based on the net thermodynamic favorability of pathways using the standard Gibbs energy of reactions. While ranking schemes based on the net favorability may be seen as good heuristics based on this criterion at first glance, they may not correspond well with the true picture of the titer of the target product, especially when a given pathway has strong competing reactions in the chassis organism. This is because these approaches do not take competing endogenous metabolic reactions into account and can only quantify the ratio of the target concentration to the source concentration at equilibrium. </p><p>In this talk, we introduce a novel thermodynamics-based approach that searches for potential biosynthetic pathways and ranks them by considering the endogenous metabolic infrastructure. The core of our new method is a new host-dependent reaction weighting scheme, which captures the competition for a metabolic precursor with endogenous reactions using a statistical mechanical model. Thus, our reaction weighting scheme quantifies, for each metabolic reaction, a fraction of a given precursor that is converted into next intermediate metabolites by considering all competing endogenous metabolic reactions. Our pathway score is based on the sum of all reaction weights in a given pathway. Thus, it can better characterize the effects of the endogenous metabolic reactions and capture the productivity of each pathway in the given host more appropriately. Our algorithm exhaustively searches for biosynthetic routes from the given starting metabolite to the given target metabolite and generates top-K biosynthetic paths, each of which has at most n reaction steps. Unlike Gibbs energy-based reaction weighting schemes, our scheme guarantees that each reaction weight has the same sign. This allows for an efficient way to search for top-K routes by detecting and avoiding many unproductive routes quickly. </p> <hr /> <p><i><b>Biological Pathway Exchange (BioPAX) Format and Pathway Commons Update</b></i><br /> Augustin Luna<br /> The BioPAX Consortium </p><p>BioPAX is a standard language that aims to enable integration, exchange, visualization and analysis of biological pathway data. Specifically, BioPAX supports data exchange between pathway data groups and thus reduces the complexity of interchange between data formats by providing an accepted standard format for pathway data. It is an open and collaborative effort by the community of researchers, software developers, and institutions. We will present recent work and future prospects related to BioPAX and its usage in the Pathway Commons effort, which aggregates BioPAX-formatted data, and advances in usages with the Systems Biology Graphical Notation (SBGN). </p> <hr /> <p><i><b>Nona Research Foundation - Creating a SynBio Software Commons</b></i><br /> Andrea Desrosiers (Nona Research Foundation), <i>Curtis Madsen</i>, <i>Nicholas Roehner</i>, Douglas Densmore (Department of Electrical &amp; Computer Engineering, Boston University) </p><p>New methods in DNA fabrication and the focus on a design/fabricate/characterize cycle have dramatically increased the amount of data in existing experimental laboratories. However, the software used to manage this information has not scaled appropriately. Additionally, more advanced software tools in automated design synthesis and validation, literature mining, robotic automation and analysis are beginning to emerge in the academic wetlab. Unfortunately, these software tools currently cannot interoperate. They are created in isolation without future integration mechanisms. Interoperability is required in order to create a seamless automated workflow. Therefore, software platform is needed upon which modular tools can be created, shared, and developed, and the relevant data can be managed and distributed. </p><p>The Nona Research Foundation (aka Nona www.nonasoftware.org) focuses on addressing these needs and making sure they are met to elevate the entire field. Nona has begun to set up an online hub, organize related national events and programs, and hold local meetings where software developers, researchers, students, and interested third parties come together to share needs and experiences. Through these activities, Nona will serve as a clearinghouse for knowledge and emerging research helping to accelerate the creation of powerful new open-source tools for synthetic biology and bio-design automation. The network aims to bring together academic and research groups adept at designing software and carrying out research-based activities with others who can ensure broad distribution to make these advances widely available and useful to participants around the world. </p> <hr /> <p><i><b>Comprehensive representation of disease mechanisms on multiple layers of granularity in SBGN PD and AF</b></i><br /> <i>Alexander Mazein</i>1, Marek Ostaszewski2, Inna Kuperstein3, Mansoor Saqi1, Bertrand De Meulder1, Feng He4, Irina Balaur1, Diane Lefaudeux1, Johann Pellet1, Piotr Gawron2, Stephan Gebel2, Andrew Parton5, Steven Watterson5, NathanaÃ«l Lemonnier6, Pierre Hainaut6, Markus Ollert4, Emmanuel Barillot3, Andrei Zinovyev3, Rudi Balling2, Reinhard Schneider2 and Charles Auffray1 <br /> 1 European Institute for Systems Biology and Medicine, CNRS-ENS-UCBL, Universite de Lyon, 50 Avenue Tony Garnier, 69007 Lyon, France 2 Luxembourg Centre for Systems Biomedicine (LCSB), University of Luxembourg, Campus Belval, 7 Avenue des Hauts-Fourneaux, L-4362 Esch-sur-Alzette, Luxembourg 3 [1] Institut Curie, Paris, France; [2] INSERM, U900, Paris, France; [3] Mines ParisTech, Fontainebleau, France 4 Department of Infection and Immunity, Luxembourg Institute of Health (LIH), House of BioHealth, 29 Rue Henri Koch L-4354 Esch-Sur-Alzette, Luxembourg 5 Northern Ireland Centre for Stratified Medicine, Ulster University, C-Tric, Altnagelvin Hospital Campus, Derry, Co Londonderry, Northern Ireland, BT47 6SB, UK 6 Institut Albert Bonniot - INSERM U823, University Grenoble-Alpes, Grenoble, France </p><p>Motivation<br /> Large amount of high-throughput data become available in the effort to better understand diseases. Despite the availability of various network-based approaches, tools for disease-specific functional analysis are greatly underdeveloped and it becomes increasingly important to advance in systematic data interpretation and hypothesis generation. The direct approach to solve this problem is developing highly accurate comprehensive computerized representations of disease mechanisms on the level of cellular and molecular interactions (Fujita et al., 2013, <a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/23832570?dopt=Abstract">PMID 23832570</a>; Kuperstein et al., 2015, <a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/26192618?dopt=Abstract">PMID 26192618</a>; Mizuno et al., 2016, <a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/26849355?dopt=Abstract">PMID 26849355</a>). </p><p>Methods<br /> Recent advances in systems biology made it possible to unambiguously represent biological processes in a consistent way (Le Novère, 2015, <a class="external mw-magiclink-pmid" rel="nofollow" href="//www.ncbi.nlm.nih.gov/pubmed/25645874?dopt=Abstract">PMID 25645874</a>), made this information human- and machine-readable so it can be efficiently explored by computational methods. Disease-specific representations are developed in CellDesigner (www.celldesigner.org) following the Systems Biology Graphical Notation standard (www.sbgn.org). The involvement of domain experts from different groups ensures that different points of view are considered and all the disease hallmarks are covered and adequately represented. </p><p>Results<br /> We present a concept of the DISEASE MAPS as a community effort and as a collection of reference resources for making sense of omics data in studies focused on a particular disease. Essentially a disease map provides a consensus review on the known disease mechanisms in the format of interconnected metabolic, signalling and gene regulatory pathways, and can be used as the basis for hypothesis generation. We describe our experience on developing disease maps for Parkinson's disease, asthma and cancer, and demonstrate how these resources can be used for data visualisation and interpretation. Because these maps are developed using a strict computational format, they can be used for developing dynamic predictive mathematical models. While being complementary to generic pathway enrichment tools (such as freely available g-Profiler and DAVID, and commercial Ingenuity Pathway Analysis and MetaCore), the disease maps focus on the integration of information into a single hierarchically-organised network, thus enabling analysis using the full power of advanced systems biology approaches in the area of systems medicine. </p><p>Conclusion<br /> To progress with this approach we propose building on the best practices and lessons learned from previous projects and applying shared standards, tools and protocols for generating high-quality representations and enabling the exchange of reusable pathway modules (e.g. inflammation, central metabolism, etc.). We envision this strategy will facilitate powerful advances in systems medicine for understanding disease mechanisms, cross-disease comparison, finding disease comorbidities, suggesting drug repositioning, generating new hypotheses, and after careful validation, redefining disease ontologies based on their endotypes - confirmed molecular mechanisms. </p><p>Acknowledgements: Funded by IMI (U-BIOPRED n°115010, eTRIKS n°115446). </p> <hr /> <p><i><b>High-performance Model Simulation with libRoadRunner</b></i><br /> <i>J Kyle Medley</i> and Herbert M Sauro<br /> University of Washington </p><p>Timecourse simulation of kinetic models is an extremely important tool in biology, and has found applications in pharmacology (PK/PD models), rational design of synthetic systems and whole-cell model simulation. Pushing the envelope of high-speed model simulation is key to exploring more complex and diverse biological modeling approaches. libRoadRunner is a high-performance simulator and analysis library for SBML-encoded models. To facilitate maximum performance, libRoadRunner makes use of the LLVM library to compile models to native machine code which can be executed directly on the target CPU. By using an open and transparent development process, making our source code freely available, and working extensively with our collaborators through week-long workshops, we have enabled widespread access to this powerful modeling tool. </p> <hr /> <p><i><b>Data Integration and Mining for Synthetic Biology Design</b></i><br /> Goksel Misirli<br /> Newcastle University </p><p>As the complexity of engineered biological systems increases, computational approaches become ever more important to identify biologically feasible solutions. Computational modelling and simulation is crucial to design and verify genetic circuits in an efficient manner. The availability of modular and reusable models is desirable to automate this process. Here, we provide a modelling approach to automate the design of genetic circuits using rule-based models, extending previously defined Standard Virtual Parts and Kappa BioBrick Framework. We have defined abstract templates that can be instantiated, and are annotated, in order to automate composition for rule-based models and produce executable composite rule-based models. We also outline protocols to incorporate model repositories into the design process. </p> <hr /> <p><i><b>Interconversion Between BioPAX and SBML</b></i><br /> Tramy Nguyen<br /> University of Utah </p><p>Standards are an important feature of the systems and synthetic biology fields because they enable exchange and reproducibility of biological models. Interoperability of tools is important because each tool is tailored for a different application. Since there are different tools that utilize different standards, enabling the interconversion among the standards would be a useful feature since more tools would be able to exchange models. The focus of this project entails the interconversion between BioPAX and SBML. This project aims to have a lossless conversion. A key feature of this project is to have consistent terms that are used between the conversions going from both directions. Additionally, this converter is targeting SBML package extensions to support both SBML qualitative and hierarchal models among other things. </p> <hr /> <p><i><b>OpenCMISS-Osmium: helping PMR support the VPH requirements for identifiable and discoverable computational models</b></i><br /> Tommy Yu, Peter Hunter, <i>David Nickerson</i> <br /> Auckland Bioengineering Institute, University of Auckland, New Zealand </p><p>As community demand for reproducible and reusable science has pushed the evolution of model and simulation encoding standards, so has it driven the desire for the Physiome Model Repository (PMR) to support collaborative, identifiable, and discoverable archiving of computational models. The OpenCMISS-Osmium project will move the software stack behind PMR to a more modern and forward looking platform as well as addressing user feedback regarding the currently deployed generation of PMR instances. Osmium is the open-source software platform being developed to expand the capabilities of the PMR and support the upcoming PHYSIOME portal. </p><p>OpenCMISS-Osmium originated as a refactoring of PMR2, the software stack behind the current generation of PMR, and the additional new features to meet the growing demands for collaborative, reproducible, and reusable modelling in the Physiome/VPH communities. Supporting the PHYSIOME portal curation and publication workflows will, furthermore, require new features be implemented in Osmium beyond the current capabilities of PMR. Enabling the integration of published models in the PHYSIOME portal, and other collaborating community repositories, will similarly require extensions to existing features in PMR and the addition of new features. </p><p>As of May 2016, there were 940 workspaces in PMR, of which 273 are private. In the last year (since 2015-06), 101 workspaces have been created by 20 active users. We currently average 1600 visitors per month of which approximately 33% are new visitors. </p> <hr /> <p><i><b>Exploiting Electronic Health Record Standard openEHR to Manage Experimental Data in Computational Physiology</b></i><br /> Koray Atalag(1), Geoff Williams(2), Gary R. Mirams(2), <i>David Nickerson</i>(1), Jonathan Cooper(2)<br /> (1) Auckland Bioengineering Institute, University of Auckland, New Zealand<br /> (2) Department of Computer Science, University of Oxford, United Kingdom </p><p>In computational physiology, standards to express mathematical and anatomically based geometric models rely heavily on the XML suite of standards and the Semantic Web. The Physiome Model Repository (PMR) provides an infrastructure to manage models and personal workspaces using distributed version control as well as providing ontology-based semantic annotation and advanced semantic searching mechanisms (including a SPARQL endpoint). We have previously described how to link clinical data to this semantic pipeline using the open access electronic health record (EHR) standard, openEHR, such that both models and related clinical data could be discovered and retrieved. This is an important step forward for enabling translational research and creating personalised decision support tools for the computational physiology community. However there is no agreed formalism to manage the structure and semantics of experimental data (e.g. from a wet lab), nor one which supports the semantic linkage of such data to model resources. Yet a set of experimental data is the basis for model development and validation. Linking models and data is therefore vital, but is currently done manually or in an ad hoc process. We have explored the utilisation of the openEHR standard to manage experimental data. openEHR provides a generic model-based approach to data modelling, and a very flexible means to express, persist and query structured data. The main premise of openEHR is to be able to manage heterogeneous data without the need to build custom data models - reusable and modular models of information can instead be represented using high level tools (known as an Archetype) and can be persisted in this form and queried using an openEHR compliant backend system easily. This can simplify data management tasks for the computational physiology community and also enable semantic interoperability. </p> <hr /> <p><i><b>SigNetSim&#160;: A web-based framework for designing kinetic models of molecular signaling networks</b></i><br /> <i>Vincent Noel</i>¹, Marcelo S. Reis¹, Matheus H.S. Dias¹, Lulu Wu ¹ ², Amanda S. Guimares ¹ ², Daniel F. Reverbel², Junior Barrera¹ ², and Hugo A. Armelin¹ ³<br /> ¹ LETA-CeTICS, Instituto Butantan, Brazil; ² Instituto de Matematica e Estata­stica, Universidade de Sao Paulo, Brazil; ³ Instituto de Quemica, Universidade de Sao Paulo, Brazil. </p><p>Molecular biology is experiencing a revolution, thanks to data acquisition becoming increasingly cheaper, and to the development of Systems Biology, an emergent research field that shows new ways to study such high-throughput data. However, biologists need a new generation of tools capable of handling large quantitative data sets, and also performing rigorous mathematical analysis of the kinetic models that may be derived from them. </p><p> SigNetSim is a versatile framework coded in Python, whose main component is a library designed for building, adjusting and analyzing quantitative biological models, compatible with the latest standard for biological models (SBML 3.1 core). Once loaded, the model is converted into a formal mathematical representation, using the computer algebra system Sympy, thus allowing usage of formal methods on the model. When numerical methods are necessary, SigNetSim can generate C code, which can be executed in parallel. While designed to be run on computational servers and used remotely, for instance, via Jupyter notebook, SigNetSim also comes with an intuitive Django web interface made for non-specialists in programming languages. Through this interface, a user can input experimental data into a simple database, design models and optimize them, and perform simulation of multiple conditions. </p><p> SigNetSim was successfully used to model both the Ras GTPase switch and the MAPK cascade in the mouse Y1 adrenal tumor cell line, and the in vitro competition between telomeric sequences and NAD+ for binding to GAPDH protein of Trypanosoma cruzi. </p><p> Results showed that the SigNetSim framework is a very useful tool for designing kinetic models of molecular signaling networks. Through its user-friendly web interface, even researchers and students that are inexperienced in programming can build, adjust and simulate models for both scientific and didactic purposes. We are currently adding for hierarchical model composition into this framework, enabling us to easily build larger models. </p> <hr /> <p><i><b>SED-ML support in JWS Online</b></i><br /> <i>Martin Peters</i>1,2, Johann Eicher2, David van Niekerk2, Kathleen Green2, Stuart Owen3, Natalie Stanford3, Carole Goble3, Dagmar Waltemath1 and Jacky Snoep2,3<br /> 1 Rostock University, Germany <br /> 2 Stellenbosch University, South Africa <br /> 3 Manchester University, UK </p><p>JWS Online is a public resource of systems biology models (Olivier and Snoep, 2004). It offers a rich set of functionalities to explore, simulate, and modify simulation models. JWS Online is also an integral part of the SEEK data management system (Wolstencroft et al., 2015) that is the backbone of the FAIRDOMHub (<a rel="nofollow" class="external free" href="https://fairdomhub.org">https://fairdomhub.org</a>). We present recent progress in the support for standardised simulation descriptions and export of COMBINE archives (Bergmann et al., 2014) in the latest version of JWS Online. A SED-ML (Waltemath et al., 2011) database was built as part of JWS to store SED-ML scripts for the reproduction of model-based simulations. The scripts can be constructed from scratch, or generated automatically from a simulation in the JWS online simulator. Besides downloading the simulation scripts, JWS Online also offers export of complete simulation studies using the COMBINE archive format. Such an archive can be stored in a private area or, after curation, made public. It may contain the models used in a study, the simulation script, the result data sets, validation data, figures, etc. We make use of SED-ML and COMBINE archives to reproduce figures in scientific manuscripts. Model references can be made to JWS Online or Biomodels, and data can be linked from SEEK on the FAIRDOM data and model management platform, or other public web-sites. Furthermore, MATH-ML scripting gives a lot of versatility to precisely reproduce model simulation results. The COMBINE archives can be accessed via URLs which makes it possible to reproduce manuscript figures via live simulations simply by linking a figure to a URL. We envision to implement a system, in close collaborations of JWS with scientific journals, that ensures a rapid population of the SED-ML database. Most exciting illustrations of the power of the archives are examples that link experimental data and models, and examples that simulate several different model scenarios and plotting the results simultaneously. A set of reproducible studies are available for exploration at: <a rel="nofollow" class="external free" href="http://jjj.bio.vu.nl/models/experiments/">http://jjj.bio.vu.nl/models/experiments/</a> </p><p>References<br /> - Bergmann, F.T., Adams, R., Moodie, S., Cooper, J., Glont, M., Golebiewski, M., Hucka, M., Laibe, C., Miller, A.K., Nickerson, D.P., Olivier, B.G., Rodriguez, N., Sauro, H.M., Scharm, M., Soiland-Reyes, S., Waltemath, D., Yvon, F., Le Novère, N. (2014) COMBINE archive and OMEX format: one file to share all information to reproduce a modeling project. BMC Bioinformatics, 15:369.<br /> - Olivier, B.G. and Snoep, J.L. (2004). Web-based kinetic modelling using JWS Online. Bioinformatics 20:2143-2144. <br /> - Waltemath, D., Adams, R., Bergmann, F.T., Hucka, M. Kolpakov, F., Miller, A.K., Moraru, I.I., Nickerson, D., Sahle, S., Snoep, J.L. and Le Novère, N. (2011) Reproducible computational biology experiments with SED-ML - The Simulation Experiment Description Markup Language. BMC Systems Biology 5:198.<br /> - Wolstencroft, K., Owen, S., Krebs, O., Nguyen, Q., Stanford, N.J., Golebiewski, M., Weidemann, A., Bittkowski, M., An, L., Shockley, D., Snoep, J.L., Mueller, W., Goble, C. SEEK: a systems biology data and model management platform (2015) BMC systems biology, 9,33. <br /> </p> <hr /> <p><i><b>How to Remember and Revisit Many Genetic Design Variants Automatically</b></i><br /> <i>Nicholas Roehner</i> and Douglas Densmore <br /> Boston University </p><p>As genetic design scales from gene clusters to genomes, it becomes challenging to specify the space of possible designs and track how this space changes as new design rules are learned. Storing individual designs is not a practical solution to this problem, as even relatively small libraries of under 20 parts can be used to construct hundreds of thousands of design variants. While previous rule-based and grammar-based approaches to genetic design have been successful in specifying the composition constraints and patterns of DNA components making up design variants, they have done so without a firm theoretical basis for comparing these specifications. To meet this need, we have developed an approach to genetic version control and implemented it in a software tool called Knox. Knox efficiently stores genetic design spaces as graphs that implicitly encode rules for composing sets of DNA components. Knox also provides methods for merging, querying, and reverting changes to these graphs, thereby making it possible for machines to track different versions of genetic design spaces throughout the history of a project and identify common design motifs among them. As genetic design scales, we anticipate that genetic version control and tools like Knox will play a critical role in supporting applications ranging from statistical design to machine learning. </p> <hr /> <p><i><b>pathwayDesigner: A SBML compliant network simulation and design tool</b></i><br /> <i>Herbert M Sauro</i> and Kyle Medley<br /> University of Washington </p><p>PathwayDesigner is an SBML compliant cellular network design and simulation tool. PathwayDesigner supports SBML level 2 and 3 as well as a native text based format. PathwayDesigner allows users to draw cellular network diagrams using a simple interface. Users can add species, reactions (and associated rate laws) and compartments to a diagram. Users can also specify additional algebraic equations to supplement the model as well as discrete events and user defined functions (as described in the SBML spec document). Simulations are carried out using the libRoadRunner backend. Events can be attached to species and reactions with visual cues indicating their presence. pathwayDesigner includes a plugin API that allows extensions to be easily written. Included with the distribution are eight plugins that include extensions for interactive simulation, metabolic control analysis, antimony support, parameter scanning, arrow head editor, random network generator and a force directed network layout tool. SBML annotations are used to store all visual aspects related to a model. PathweyDesigner is also SBW compatible. Availability: pathwayDesigner is currently a Windows only tool but a prototype Mac OS version is available (pathwaydesigner.org). </p> <hr /> <p><i><b>PharmML - exchange format for models used in quantitative system pharmacology and pharmacometrics</b></i><br /> Maciej J Swat<br /> EMBL-EBI, Cambridge, UK </p><p>New format enabling the efficient exchange and integration of pharmacometric (PMX) and quantitative system pharmacology (QSP) models across software tools has been defined and implemented as key element of the DDMoRe interoperability platform [1,2]. PharmML has been designed as the storage and exchange medium for mathematical and statistical models [3]. PharmML is a declarative language which development has been based on requirements provided by the DDMoRe community, popPK/PD and QSP partners, and on specific use cases from the main target tools. DDMoRe developed number of translators allowing to run PharmML coded model in softwares and programming languages, such as NONMEM, Monolix, PFIM, BUGS, R, Matlab, SBML, Python etc. PharmML provides a structure for encoding continuous and discrete data models equipped with complex variability structure, covariate, structural and observation models. Definition of complex clinical trial designs and modeling steps is possible as well. As a comprehensive self-contained format, it allows to encode models in tool agnostic manner. To facilitate the encoding of statistical models a database of probability distributions, ProbOnto, has been developed [4]. Moreover, to store the typical results produced in a QSP/PMX workflow a complementary format, Standard Output (SO), has been created [5]. </p><p>The new exchange formats facilitate <br /> (i) smooth and error-free transmission of models between tools, <br /> (ii) use of complex workflows via standardised model and output definitions, <br /> (iii) reproducibility of research, <br /> (iv) bug tracking,<br /> (v) improved interaction with regulatory agencies regarding modeling and simulation,<br /> (vi) development of new tools and methods and<br /> (vii) reuse of existing model resources, e.g., BioModels database. </p><p><br /> References:<br /> [1] DDMoRe project, URL: www.ddmore.eu <br /> [2] Swat et al. (2015). Pharmacometrics Markup Language (PharmML): Opening New Perspectives for Model Exchange in Drug Development. CPT Pharmacometrics Syst Pharmacol, 4(6):316-9. URL: pharmml.org<br /> [3] Lavielle, M. (2014). Mixed Effects Models for the Population Approach: Models, Tasks, Methods and Tools. Chapman &amp; Hall/CRC Biostatistics Series.<br /> [4] Swat MJ, Grenon P, Wimalaratne S. ProbOnto - ontology and knowledge base of probability distributions, Bioinformatics 2016; doi: 10.1093/bioinformatics/btw170. URL: www.probonto.org <br /> [5] URL: ddmore.eu/projects/so-standard-output </p> <hr /> <p><i><b>Visualising differences in SBML models using SBGN and BiVeS</b></i><br /> Tom Gebhardt, Martin Scharm, <i>Vasundra Toure</i>, Dagmar Waltemath, Olaf Wolkenhauer<br /> University of Rostock </p><p>Simulation models change over time. Reasons may include typo corrections, parameter updates, specification updates or improved semantic annotations. The BiVeS tool is a Java library to detect differences between simulation models endoded in SBML or CellML. It returns a list of changes between two versions of a model. BiVeS provides both human-readable and computer-readable output formats. One output type is JSON, a data-interchange format that is based on JacaScript and easy to parse. SBGN is a visual standard to display biological networks. It defines sets of glyphs to use for the representation of biological entities (macromolecules, simple chemicals, complexed etc) and relations between them (catalysis, inhibition, association, dissociation etc). In our talk, we will introduce the BiVeS tool and its output types. We will focus on the novel JSON format. Contrary to the previous formats, BiVeS now exports all information necessary to generate the difference map using SBGN-compliant glyphs. The differences are furthermore highlighted in the SBGN map, using a specific color encoding. The support of SBGN for the visualisation of difference further improves the readability and interpretation of changes in simulation models, and it aids users in understanding the evolution of a model. </p> <hr /> <p><i><b>Modeling and Simulating Hybrid SBML Models</b></i><br /> <i>Leandro Watanabe</i> and Chris Myers<br /> University of Utah </p><p>The leading standard for the representation of biological models for simulation is the Systems Biology Markup Language (SBML). Traditionally, one would simulate such models using a single formalism such as ordinary differential equations or stochastic methods. However, more complex models like the whole-cell model requires the coupling of different formalisms since different biological process have been shown to be better represented using different formalism. For example, metabolism is often modeled using flux-balance analysis and some reactions as stochastic processes. Recently efforts have been put forth to encode the whole-cell model in SBML. In this process, a scheme to represent hybrid models have been proposed. As part of our work, a simulation method has been implemented within the tool iBioSim to support such models. </p> <hr /> <p><i><b>SBML Multi - From Specification to Application</b></i><br /> <i>Fengkai Zhang</i> and Martin Meier-Schellersheim<br /> Laboratory of Systems Biology, National Institute of Allergy and Infectious Diseases, NIH, USA </p><p>Domain-detailed reaction rule (or rule-based) modelling is an approach that describes interactions between pairs of molecular components, specifying how the interactions depend on particular states of the molecules and their location in specific compartments. To promote model sharing among the software tools supporting it the SBML Multi Package (Multistate, Multicomponent and Multicompartment Species Package for SBML Level 3) was developed as a standard specifically to satisfy the requirements and reflect the nature of rule-based models. In our presentation, we will briefly review rule-based modelling, provide an update on the development of the SBML Multi package and introduce the main functions of libsbml-multi library with examples. We will also illustrate how to export Simmune models to the SBML Multi format and, conversely, import SBML Multi models into Simmune for further analysis and simulation. This work is supported by the intramural program of the NIAID, NIH. </p> <hr /> <p><br /> </p><p><br /> </p> <!-- NewPP limit report Cached time: 20220802130008 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.102 seconds Real time usage: 0.106 seconds Preprocessor visited node count: 125/1000000 Preprocessor generated node count: 250/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 2/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  <div class="node-links">
      </div>

  
</div> <!-- /.node -->




          </div>

        
        
      </div></div> <!-- /.section, /#content -->

      <div id="navigation"><div class="section clearfix">

	           <div id="suckerfishmenu">
            <ul class="menu"><li class="leaf first"><a href="https://combine-org.github.io/events/" title="The annual COMBINE meeting will be held Oct 6-7, 2022, at the Humboldt University of Berlin, Germany.">COMBINE 2022</a></li>
<li class="leaf"><a href="/standards" title="Standards covered by or related to COMBINE activities">Standards</a></li>
<li class="leaf"><a href="/events" title="Events">Events</a></li>
<li class="collapsed"><a href="/documents" title="Documents">Documents</a></li>
<li class="leaf"><a href="/comm" title="Mailing lists and forums of discussion">Forums</a></li>
<li class="leaf last"><a href="/about" title="About">About</a></li>
</ul>          </div>
        
                  <div id="search-box">
            <form action="/events/COMBINE_2016/abstracts"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-B46X_Vq5cgGrhbOlUNKdTPuVXKV1PpEjGvypUC5Ltm4" value="form-B46X_Vq5cgGrhbOlUNKdTPuVXKV1PpEjGvypUC5Ltm4"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
          </div> <!-- /#search-box -->
        
        
      </div></div> <!-- /.section, /#navigation -->

      <div class="region region-sidebar-first column sidebar"><div class="section">
  
<div id="block-menu-menu-combine-left-menu" class="block block-menu first last region-odd odd region-count-1 count-1  block-1"><div class="block-inner">

      <h2 class="title">COMBINE</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first"><a href="/home" title="The home page of COMBINE">Home</a></li>
<li class="leaf"><a href="/help" title="">Help</a></li>
<li class="leaf last"><a href="/login" title="">Sign-in</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
</div></div><!-- /.section, /.region -->

      
    </div></div> <!-- /#main, /#main-wrapper -->

          <div id="footer"><div class="section">

        
        <div class="region region-footer">
  
<div id="block-block-1" class="block block-block first last region-odd even region-count-1 count-2  block-1"><div class="block-inner">

      <h2 class="title">COmputational Modeling in BIology NEtwork</h2>
  
  <div class="content">
    <p>COmputational Modeling in BIology NEtwork </p> <!-- NewPP limit report Cached time: 20220802130129 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.005 seconds Real time usage: 0.006 seconds Preprocessor visited node count: 1/1000000 Preprocessor generated node count: 4/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  
</div></div> <!-- /block-inner, /block -->
</div><!-- /.region -->
<div class="element-hidden" id="visible">
<a href="http://www.merrellpublishers.com/buy_meds.htm">cheap medicaments usa</a> 
<a href="http://www.drleanawen.com/styled-2/styled-9/commentary.html">generic sildenafil</a>
</div>
      </div></div> <!-- /.section, /#footer -->
    
  </div></div> <!-- /#page, /#page-wrapper -->

  
  
</body>
</html>
