
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>HARMONY 2015 - List of Posters | COMBINE</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/sites/combine/files/Symbol-COMBINE_logo.ico" type="image/x-icon" />
  <link type="text/css" rel="stylesheet" media="all" href="/modules/node/node.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/poll/poll.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/defaults.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system-menus.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/user/user.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/ctools/css/ctools.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/panels/css/panels.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/views/css/views.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/html-reset.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/wireframes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/tabs.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/messages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/pages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/block-editing.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/blocks.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/navigation.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/nodes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/comments.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/forms.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/cti-flex2.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/combine-custom.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/layout-fixed.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/colors/combine.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/local.css?H" />
<link type="text/css" rel="stylesheet" media="print" href="/sites/all/themes/cti_flex2/css/print.css?H" />
<!--[if IE]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie.css?H" />
<![endif]-->
<!--[if lte IE 6]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie6.css?H" />
<![endif]-->
  <script type="text/javascript" src="/misc/jquery.js?H"></script>
<script type="text/javascript" src="/misc/drupal.js?H"></script>
<script type="text/javascript" src="/sites/all/modules/panels/js/panels.js?H"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/" });
//--><!]]>
</script>
  <style type="text/css">
body {font-family: Tahoma, Verdana, Arial, Helvetica, "Bitstream Vera Sans", sans-serif;}
</style>  <style type="text/css">
</style>
  <!--[if lte IE 6]>
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/suckerfish.js"></script>
  <![endif]-->
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/combine.js"></script>
</head>

<body class="page not-front not-logged-in node-type-page page-events-harmony-2015-posters section-events one-sidebar sidebar-first page-panels ">

  <div id="page-wrapper"><div id="page">

    <div id="header"><div class="section clearfix">

      
              <div id="logo-title">

                      <div id="logo"><a href="/" title="Home" rel="home"><img src="/sites/combine/files/COMBINE_website-banner.png" alt="Home" id="logo-image" /></a></div>
          
      
        </div> <!-- /#logo-title -->
      
      
    </div></div> <!-- /.section, /#header -->

    <div id="main-wrapper"><div id="main" class="clearfix  with-navigation">

      <div id="content" class="column"><div class="section">

        
        
        <div class="breadcrumb"><a href="/">Home</a> › HARMONY 2015 - List of Posters</div>                  <h1 class="title">HARMONY 2015 - List of Posters</h1>
                                
        
        <div id="content-area">
          <div id="node-187" class="node node-type-page build-mode-full clearfix">

   
  
  
  
  
  <div class="content">
    <table class="borderless-table sm-padding sm-font" width="95%" align="center"> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Tellurium&#160;: A Python Based Integrated Environment for Systems Biology</b> </td></tr> <tr> <td> Kiri Choi<br /><br /> </td></tr> <tr> <td> Tellurium is a python based integrated environment for modeling and simulating biological systems. Tellurium is fully open, flexible, and easily expendable thanks to its python based nature, exemplified by several unique features. Particularly, we discuss the integration of several python libraries and user-friendly plug-ins, and creation of entirely self-contained environment. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Visual analysis of biological networks using VANTED and SBGN-ED</b> </td></tr> <tr> <td> Tobias Czauderna<br /><br /> </td></tr> <tr> <td> The Systems Biology Graphical Notation (SBGN) is a standard for the visual representation of biochemical networks and cellular processes studied in systems biology. Three orthogonal languages cover several aspects of biological systems in different levels of detail. SBGN helps to communicate knowledge more efficiently and accurately between different research communities. To support SBGN, methods and tools for editing, validating, and translating of SBGN maps are necessary.<br />VANTED is an integrative framework for systems biology applications. It aims at the integration, analysis and visual exploration of experimental data in the context of biological networks. The framework comprises the VANTED core as well as various extensions. These extensions are being developed for the needs of life scientists and extend the functionality of the VANTED core towards various tasks and topics in systems biology.<br />One extension is SBGN-ED. It allows creating and editing all types of SBGN maps. Furthermore the syntactical and semantical correctness of created or edited maps can be validated. Existing pathways maps from the KEGG database can be translated into SBGN maps including automatic layout. A visualisation of SBML models in SBGN is also provided, although this feature currently requires manual post-editing of the map layout. Additionally the tool allows exporting of SBGN maps into several file formats including SBGN-ML. Together with the VANTED framework SBGN-ED supports the visual analysis of SBGN maps. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Using Time Course Metabolomics to Elucidate Genome-Scale Pathway Utilization for CHO-S Cell Lines in Batch Culture</b> </td></tr> <tr> <td> Andreas Dräger<br /><br /> </td></tr> <tr> <td> CHO cells are the primary hosts for production of many human recombinant proteins. However, our incomplete understanding of CHO cell metabolism has limited our ability to rationally enhance transgene expression and limit the secretion of toxic byproducts.<br />Therefore, there is a need for rigorous assessment of the molecular mechanisms that affect production objectives and eventually introduces enough metabolic inefficiencies to decrease bioproduct yield.<br />We present an analysis pipeline that characterizes this diversity by using time course exo-metabolomics and genome-scale, mass-balance models of CHO cell metabolism to visualize shifts in reaction pathways, such as glycolytic and oxidative metabolism versus anabolic pathways of growth and protein production, in order to comprehensively characterize CHO metabolism during batch culture for the first time. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>NormSys - Harmonizing Standardization Processes for Model and Data Exchange in Systems Biology</b> </td></tr> <tr> <td> Martin Golebiewski<br /><br /> </td></tr> <tr> <td> Standards for data, resulting computer models and applied workflows have become a critical issue specifically in distributed and interdisciplinary approaches like systems biology. Different stakeholder groups need to be engaged in the standardization activities to allow an efficient and fast process of standardization and adoption of the developed standards: Researchers from both, academia and industries, with their existing grass-root standardization communities, as well as the official standardization bodies (e.g. DIN in Germany, CEN/CENELEC at European level, or ISO at international level) with their long-standing professional experience in the formal standardization process or representatives of scientific journals and research funding agencies.<br />"NORM-SYS - Normalization and standardization for the exchange of models and data in systems biology research" is a new project (funded by the German Federal Ministry for Economic Affairs and Energy) that aims at enhancing and promoting the formal normalization of existing community standards for computational modelling in systems biology in close collaboration with relevant stakeholder groups and grass-root standardization initiatives. One major goal of NormSys is to develop a concept for the transformation of existing standards into certified specifications or norms to achieve a more effective transfer of systems biology research results into applications. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Implementation of Spatial SBML Modeling Software based on Microscopic Image</b> </td></tr> <tr> <td> Kaito Ii<br /><br /> </td></tr> <tr> <td> In this research, we implemented an ImageJ plugin which creates SBML models, using cellular microscopic images, reflecting the three dimensional cellular geometry. With this implemented plugin, users are able to perform spatial model simulations based on realistic cellular geometry with SBML supported simulators such as VCell and Spatial Simulator[1].<br />By giving segmented images of organelles, the plugin processes the images which then creates an SBML model. First, the plugin assigns a pixel value to each organelles, composing a single gray scale image. While processing these input images, it also interpolates the image to adjust the voxel ratio of each axes. The gray scale image is then used to sort the adjacent relations between the organelles. This adjacent relation is determined by labeling the gray scale image. Lastly, informations of organelles and adjacent relations are converted to SBML form with spatial extension and outputted as an SBML file.<br />Image-based model was created using the implemented plugin. Segmented images of each organelles were developed in advance, inputting into the plugin. With the model, membrane diffusion simulation is performed in three dimension with Spatial Simulator. As a result, spatial model simulation based on cellular geometry was achieved.<br />In conclusion, this implemented ImageJ plugin allows for users to create SBML models with cellular geometry extracted from the microscopic images. By distributing the plugin through the world wide web, simulations with environments will get much closer to those of actual cells becomes widely possible.<br /><br />[1] Tatsuhiro Matsui, et al. Implementation of spatial model simulator and its SBML support. HARMONY 2012, May, 2012. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>SBOL Stack: The One-stop-shop to Storing and Publishing SBOL Data</b> </td></tr> <tr> <td> Curtis Madsen<br /><br /> </td></tr> <tr> <td> Recently, synthetic biologists have developed the Synthetic Biology Open Language (SBOL), a data exchange standard for descriptions of genetic parts, devices, modules, and systems. The goals of this standard are to allow researchers to exchange designs of biological parts and systems, to send and receive genetic designs to and from biofabrication centers, to facilitate storage of genetic designs in repositories, and to embed genetic designs in publications. In order to achieve these goals, the development of an infrastructure to store, retrieve, and exchange SBOL data is necessary.<br />To address this problem, we have developed the SBOL Stack, a Sesame Resource Description Framework (RDF) database specifically designed for storing and publishing of SBOL data. The SBOL Stack can be used to publish a library of synthetic parts and designs as a service, to share SBOL with collaborators, and to store designs of biological systems locally. It includes a web client that allows users to upload new biological data to the database and to perform SPARQL queries to access desired SBOL parts.<br />To facilitate exchange, instances of the SBOL Stack can be installed by researchers at various organizations. Users can then register these different instances of the SBOL Stack with their own instance and perform federated queries over all registered databases. These queries allow users to retrieve and compile more complete data from multiple databases without the need to manually query each repository individually. In fact, the SBOL Stack can register any Sesame RDF database, so other repositories that contain information about biological parts can be included in the federated queries. It is this automatic retrieval and integration that makes the SBOL Stack a must-have tool for researchers working on the design of systems in synthetic biology. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Accelerating SBML Spatial Model Simulator using GPGPU</b> </td></tr> <tr> <td> Kota Mashimo<br /><br /> </td></tr> <tr> <td> On a spatial model simulation based on PDEs, the simulation space is discretized by grid and simulators will compute on each grid sequentially which will increase the simulation time enormously depending on the number of grids. In order to solve this problem, we have applied parallelization to this sequential numerical integration using GPGPU (General Purpose computing on GPU).<br />GPGPU is a parallelization technique that applies GPU(Graphics Processing Unit) to general purpose computation. This technique has benefits compared with other parallelization techniques such as HPCs and PC clusters on the following points: cost performance, power performance, space saving, and allows to construct high-speed simulation environment in individual or laboratory level.<br />In this research, we parallelized a CPU-based spatial model simulator[1] by GPGPU. We implemented numerical integration of advection, reaction and diffusion equation with NVIDIA CUDA. For the evaluation of the CPU application, we used Intel Xeon X5687 (3.60 GHz, 57.6 GFLOPS) and for the evaluation of GPU application, we used Tesla M2090 (Fermi architecture, 1.33 TFLOPS).<br />GPUs have many cores and can generate multiple threads. Thus, we assigned a thread to each grid, and executed the integration on all grids simultaneously. In addition, we optimized the GPU memory access for further efficiency. As a result, we achieved 25x performance improvement in advection equation, 52x in reaction equation, 46x in diffusion equation for 512x512 grids.<br />In conclusion, we succeeded in accelerating a CPU-based SBML spatial model simulator by GPGPU. Our implementation allows to provide high-performance simulation environment for SBML model with spatial extension.<br /><br />[1] Tatsuhiro Matsui, et al. Implementation of spatial model simulator and its SBML support. HARMONY 2012, May, 2012. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Biological Interaction Types for WikiPathways</b> </td></tr> <tr> <td> Ryan Miller<br /><br /> </td></tr> <tr> <td> WikiPathways (www.wikipathways.org) is a community curation pathway databases and allows researchers to draw, curate and publish their annotated pathway diagrams in this wiki-based database.<br />The current content is saved in the GPML format and we recently also converted the content into RDF format for the semantic web. In RDF, we use two different vocabularies to represent WikiPathways data: the GPML vocabulary to store the graphical and GPML specific data and the WP vocabulary to retrieve the semantic knowledge from the pathways. Currently, the interaction types in the GPML are identified as strings that indicate what type of interaction it is. For example, "mim-inhibition" indicates the inhibition interaction type of the MIM standard. Besides MIM interactions, we also have a SBGN plugin for our pathway editor, PathVisio. Much like the MIM notation used earlier, the SBGN plugin indicates the types of interactions in the GPML, for example, "SBGN-inhibition" for an inhibition interaction.<br />In the future, both these types (and maybe other drawing standards) will coexist in RDF generated for WikiPathways. Therefore semantic mappings between the two notations in the WikiPathways vocabulary are needed, so a SPARQL query recognizes the generic type ‘Inhibition’ independent of the notation that was used when drawing the pathway. Furthermore, mappings to other semantic formats like BioPAX or SBO need to be developed.<br />Here, we would like to present our ideas about mappings between the different notations and formats and identify possible existing mappings between them. Additionally, we would like to discuss a possible basic set of interaction types that should be used in the default drawing panel for WikiPathways. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>A standard model repository for genetic design automation</b> </td></tr> <tr> <td> Goksel Misirli<br /><br /> </td></tr> <tr> <td> Goksel Misirli<sup>1</sup>, Jennifer Hallinan<sup>1</sup>, Owen Gilfellon<sup>1</sup> and Anil Wipat<sup>1</sup><br /><sup>1</sup>Newcastle University, UK<br /><br />One ambition of synthetic biology is the large-scale engineering of biological systems. However, as the complexity and size of designs increases, the manual design of genetic circuits becomes more challenging. Computational tools often use libraries of mathematical models of biological parts in order to aid the user in building complex and predictable designs. To support automated, model-driven design it is desirable that in silico models are modular, composable and in standard formats. Here, we present an approach for composable and modular models for synthetic biology, termed standard virtual parts (SVPs), to support software tools in model-driven process. A repository of SVPs has been established to facilitate computational genetic circuit design. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Computational Tools for the Creation, Simulation, and Dissemination of Epithelial Cell Models</b> </td></tr> <tr> <td> David Nickerson<br /><br /> </td></tr> <tr> <td> David P Nickerson<sup>1</sup>, Kirk L Hamilton<sup>2</sup>, Daniel A Beard<sup>3</sup> and Peter J Hunter<sup>1</sup><br /><sup>1</sup>Auckland Bioengineering Institute, University of Auckland, Auckland, New Zealand; <sup>2</sup>Department of Physiology, University of Otago, Dunedin, New Zealand; <sup>3</sup>Medical School, University of Michigan, USA<br /><br />We have developed a software suite for the creation, simulation, and dissemination of epithelial cell models. The suite, known as GET (Generalized Epithelial Transport), is freely available under an open-source license at: <a rel="nofollow" class="external free" href="https://bitbucket.org/get">https://bitbucket.org/get</a>. GET consists of a model creation tool, a simulation tool, and a server that provides access to some functionality from these tools via web services. The server also provides access to data from the Physiome Model Repository. The GET framework makes use of standard community file formats to store mathematical model and numerical simulation information in order to ensure users are able to reuse, reproduce, and exchange their work with other software tools, repositories, and scientists. The mathematical models are encoded in the CellML format (<a rel="nofollow" class="external free" href="http://cellml.org">http://cellml.org</a>) and the SED-ML format (<a rel="nofollow" class="external free" href="http://sed-ml.org">http://sed-ml.org</a>) is used to encode descriptions of the computational simulation experiments. The development of the GET suite has led to proposals for extensions to both the CellML and SED-ML specifications in order to meet the requirements of the epithelial cell models.<br /><br />Supported by the Virtual Physiological Rat Project, NIH grant [P50-GM094503] and Maurice Wilkins Centre for Molecular Biodiscovery. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Synthetic Biology Open Language (SBOL) 2.0</b> </td></tr> <tr> <td> Tramy Nguyen<br /><br /> </td></tr> <tr> <td> Tramy Nguyen<sup>1</sup>, Zhen Zhang<sup>1</sup> and Chris Myers<sup>1</sup><br /><sup>1</sup>Electrical and Computer Engineering, University of Utah, USA<br /><br />Synthetic Biology Open Language (SBOL) is an exchangeable data representation for synthetic biology models. The purpose of SBOL is to provide a data standard for the exchange for the designs of software tools, research groups, and commercial service providers. A Java based SBOL library has been implemented as an xml/rdf serialization. Namely, SBOL v1.1 has been previously released but it was constrained to a limited number of biological representations. In order to overcome this limitation, SBOL v2.0 was proposed to enable the representation of further information needed to exchange and reproduce genetic designs. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>The CombineArchiveWeb – A web based tool to handle files associated with a virtual experiment</b> </td></tr> <tr> <td> Martin Peters<br /><br /> </td></tr> <tr> <td> Sharing in silico experiments is essential for the advance of research in computational biology. Consequently, the COMBINE archive was designed as a digital container format. It eases the management of numerous files, fosters collaboration, and ultimately enables the exchange of reproducible research results. However, manual handling of COMBINE archives is tedious and error prone. We therefore developed the CombineArchiveWeb to support scientists in promoting and publishing their work by means of creating, exploring, modifying, and sharing archives. All files are equipped with meta data and can be distributed over the Web through shareable workspaces. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Identifying, Interpreting, and Communicating Changes in XML-encoded Models of Biological Systems</b> </td></tr> <tr> <td> Martin Scharm<br /><br /> </td></tr> <tr> <td> Martin Scharm<sup>1</sup>, Olaf Wolkenhauer<sup>1,2</sup> and Dagmar Waltemath<sup>1</sup><br /><sup>1</sup>Department of Systems Biology and Bioinformatics, University of Rostock, Rostock, Germany; <sup>2</sup>Stellenbosch Institute for Advanced Study (STIAS), Wallenberg Research Centre at Stellenbosch University, Stellenbosch 7600, South Africa<br /><br />Research in systems biology enhanced our knowledge of biological environments. Many discoveries are recorded in computational models which encode the structure of biological networks, and describe their temporal and spatial behavior. Due to tremendous efforts by the research community, the number of openly available models is numerous and still continually increasing. To support the sharing of models and, thus, the reuse of research results, repositories such as the BioModels Database and the CellML Model Repository collect and store models in exchangeable formats such as the Systems Biology Markup Language (SBML) or CellML.<br />Since only accessible models can be reused, such repositories are essential to guarantee transparent research. However, model repositories to date lack sufficient mechanisms to track the updates of models in their databases. Model versions often cannot be addressed unambiguously and changes occurring between versions of a model are not communicated transparently. Therefore, a framework to identify the differences between models and their versions is a fundamental requirement to compare and combine models. Only with difference detection at hand users are able to grasp a model’s history and to identify errors and inconsistencies.<br />Our current research concentrates on developing efficient and reliable difference detection for versions of models. Specifically, our algorithm for difference detection, BiVeS, is applicable to models encoded in SBML or CellML. As standard representation formats for computational models in biology use XML, BiVeS bases on an XML-diff algorithm, namely the XyDiff algorithm. The algorithm is format-specific in the sense that it respects the structure of the representation formats. The final set of of differences can be exported in both machine and human readable formats: BiVeS produces an XML-encoded patch containing all modifications which occurred between the two versions of a document. Changes between model versions are also summarized in a report and highlighted in a graph, which comprehensively displays the updates affecting the reaction network. The algorithm is implemented in a Java library.<br />Gaining insights into the process of development of a particular model has the potential to increase the confidence in this model and supports the collaboration of distinct research projects dramatically.<br />Consequently, existing model repositories can benefit from extending their software and functionalities with version control. On our poster, we show how the BiVeS library can be integrated with existing software and discuss first statistics about the evolution of computational models in open repositories. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>Storage and visualisation of rule-based models in graph database</b> </td></tr> <tr> <td> Anatoly Sorokin<br /><br /> </td></tr> <tr> <td> Graph is the most widely used type of representation of biological information, either for analysis, or for visualisation. At the same time most of present-day databases are relational and store information in tabular form. Recent development of graph-oriented database engines makes it possible to remove this discrepancy. The information about E.coli genome, proteome, regulome etc. from RegulonDB, GeneBank, UniProt together with regulatory models in a form sutable for rule-based analysis are combined in graph-oriented database. We are exploring the benefits and drawbacks of graph-oriented storage for dealing with whole genome data and models. </td></tr> <tr> <td> <hr style="border-color:#eeeeee" /> </td></tr> <tr> <td> <b>SBML Models Using Arrays</b> </td></tr> <tr> <td> Leandro Watanabe<br /><br /> </td></tr> <tr> <td> The Systems Biology Markup Language (SBML) is the de facto standard representation of biological models. However, SBML core constructs are limited to operations on scalars values while regular structures (i.e. many constructs sharing the same attributes) can only be represented in an inefficient manner. In order to overcome this limitation, an extension to SBML constructs, called the Arrays package, has been developed and implemented within the Java-based library of SBML called JSBML. This extension allows the construction of SBML objects in vector form. The JSBML library has a function for flattening (or inlining) arrays constructs to ease the integration of analysis tools with the Arrays package. However, flattening a model before simulation restricts arrays objects to be statically computable (i.e. constant sizes). This presentation shows a way to analyze arrayed models where the sizes can be dynamically changing. That is, arrays are handled on the fly, which eliminates the need of flattening models before simulation. This approach has been implemented within iBioSim, a tool that can be used for the modeling and analysis of SBML models. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr> </table> <!-- NewPP limit report Cached time: 20220802200502 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.044 seconds Real time usage: 0.044 seconds Preprocessor visited node count: 17/1000000 Preprocessor generated node count: 40/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  <div class="node-links">
      </div>

  
</div> <!-- /.node -->




          </div>

        
        
      </div></div> <!-- /.section, /#content -->

      <div id="navigation"><div class="section clearfix">

	           <div id="suckerfishmenu">
            <ul class="menu"><li class="leaf first"><a href="https://combine-org.github.io/events/" title="The annual COMBINE meeting will be held Oct 6-7, 2022, at the Humboldt University of Berlin, Germany.">COMBINE 2022</a></li>
<li class="leaf"><a href="/standards" title="Standards covered by or related to COMBINE activities">Standards</a></li>
<li class="leaf"><a href="/events" title="Events">Events</a></li>
<li class="collapsed"><a href="/documents" title="Documents">Documents</a></li>
<li class="leaf"><a href="/comm" title="Mailing lists and forums of discussion">Forums</a></li>
<li class="leaf last"><a href="/about" title="About">About</a></li>
</ul>          </div>
        
                  <div id="search-box">
            <form action="/events/HARMONY_2015/Posters"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-ixk_uAX69hKdsf2CVK_LvaXPO4Td1g7Dm1Gv49QKiwg" value="form-ixk_uAX69hKdsf2CVK_LvaXPO4Td1g7Dm1Gv49QKiwg"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
          </div> <!-- /#search-box -->
        
        
      </div></div> <!-- /.section, /#navigation -->

      <div class="region region-sidebar-first column sidebar"><div class="section">
  
<div id="block-menu-menu-combine-left-menu" class="block block-menu first last region-odd odd region-count-1 count-1  block-1"><div class="block-inner">

      <h2 class="title">COMBINE</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first"><a href="/home" title="The home page of COMBINE">Home</a></li>
<li class="leaf"><a href="/help" title="">Help</a></li>
<li class="leaf last"><a href="/login" title="">Sign-in</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
</div></div><!-- /.section, /.region -->

      
    </div></div> <!-- /#main, /#main-wrapper -->

          <div id="footer"><div class="section">

        
        <div class="region region-footer">
  
<div id="block-block-1" class="block block-block first last region-odd even region-count-1 count-2  block-1"><div class="block-inner">

      <h2 class="title">COmputational Modeling in BIology NEtwork</h2>
  
  <div class="content">
    <p>COmputational Modeling in BIology NEtwork </p> <!-- NewPP limit report Cached time: 20220802130129 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.005 seconds Real time usage: 0.006 seconds Preprocessor visited node count: 1/1000000 Preprocessor generated node count: 4/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  
</div></div> <!-- /block-inner, /block -->
</div><!-- /.region -->
<div class="element-hidden" id="visible">
<a href="http://www.merrellpublishers.com/buy_meds.htm">cheap medicaments usa</a> 
<a href="http://www.drleanawen.com/styled-2/styled-9/commentary.html">generic sildenafil</a>
</div>
      </div></div> <!-- /.section, /#footer -->
    
  </div></div> <!-- /#page, /#page-wrapper -->

  
  
</body>
</html>
