
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>HARMONY 2021 Attendee list | COMBINE</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/sites/combine/files/Symbol-COMBINE_logo.ico" type="image/x-icon" />
  <link type="text/css" rel="stylesheet" media="all" href="/modules/node/node.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/poll/poll.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/defaults.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system-menus.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/user/user.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/ctools/css/ctools.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/panels/css/panels.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/views/css/views.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/html-reset.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/wireframes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/tabs.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/messages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/pages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/block-editing.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/blocks.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/navigation.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/nodes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/comments.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/forms.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/cti-flex2.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/combine-custom.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/layout-fixed.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/colors/combine.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/local.css?H" />
<link type="text/css" rel="stylesheet" media="print" href="/sites/all/themes/cti_flex2/css/print.css?H" />
<!--[if IE]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie.css?H" />
<![endif]-->
<!--[if lte IE 6]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie6.css?H" />
<![endif]-->
  <script type="text/javascript" src="/misc/jquery.js?H"></script>
<script type="text/javascript" src="/misc/drupal.js?H"></script>
<script type="text/javascript" src="/sites/all/modules/panels/js/panels.js?H"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/" });
//--><!]]>
</script>
  <style type="text/css">
body {font-family: Tahoma, Verdana, Arial, Helvetica, "Bitstream Vera Sans", sans-serif;}
</style>  <style type="text/css">
</style>
  <!--[if lte IE 6]>
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/suckerfish.js"></script>
  <![endif]-->
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/combine.js"></script>
</head>

<body class="page not-front not-logged-in node-type-hpage page-events-harmony-2021-attendees section-events one-sidebar sidebar-first page-panels ">

  <div id="page-wrapper"><div id="page">

    <div id="header"><div class="section clearfix">

      
              <div id="logo-title">

                      <div id="logo"><a href="/" title="Home" rel="home"><img src="/sites/combine/files/COMBINE_website-banner.png" alt="Home" id="logo-image" /></a></div>
          
      
        </div> <!-- /#logo-title -->
      
      
    </div></div> <!-- /.section, /#header -->

    <div id="main-wrapper"><div id="main" class="clearfix  with-navigation">

      <div id="content" class="column"><div class="section">

        
        
        <div class="breadcrumb"><a href="/">Home</a> › HARMONY 2021 Attendee list</div>                  <h1 class="title">HARMONY 2021 Attendee list</h1>
                                
        
        <div id="content-area">
          <div id="node-291" class="node node-type-hpage build-mode-full clearfix">

   
  
  
  
      <div class="meta">
              <span class="submitted">
          Submitted by <a href="/user/1954" title="View user profile.">pgleeson</a> on Fri, 03/19/2021 - 11:58        </span>
      
          </div>
  
  <div class="content">
    <style>
table {
  border-collapse: collapse;
  width: 100%;
}

th, td {
  text-align: left;
  padding: 8px;
}

tr:nth-child(even) {background-color: #f2f2f2;}
</style>
<p>The following is the list of known attendees as of 25/03/2021 09:34:42 UTC (177 total registered, 159 listed below)</p>
<table>
    <th>Name</th><th>Affiliation</th><th>Project(s)</th>
    <tr><td>Aj Abam</td><td>CU Boulder</td><td>SBOL and SBOL Visual <br/>SBOL Development  </br/></td></tr>
    <tr><td>Nora Abi akar</td><td>CSCS</td><td>NeuroML  </td></tr>
    <tr><td>Pooja Babu</td><td>Forschungszentrum Juelich</td><td>NeuroML  </td></tr>
    <tr><td>Hasan Baig</td><td>UConn Health</td><td>SBML, SBOL and SBOL Visual <br/><a href="https://www.hasanbaig.com/projectsportfolio/" title="https://www.hasanbaig.com/projectsportfolio/">https://www.hasanbaig.com/projectsportfolio/</a> </br/></td></tr>
    <tr><td>Irina Balaur</td><td>University of Luxembourg</td><td>BioPAX, SBGN, SBML, SED-ML  </td></tr>
    <tr><td>Geoff Baldwin</td><td>Imperial College London</td><td>SBOL and SBOL Visual <br/>DNABOT </br/></td></tr>
    <tr><td>Julien  Ballbé </td><td>CNRS - Centre Borelli</td><td>NeuroML  </td></tr>
    <tr><td>Bryan Bartley</td><td>Raytheon BBN Technologies</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Moritz Beber</td><td>Unseen Bio ApS</td><td>SBML, SED-ML, COMBINE Archive <br/>COBRApy (<a href="https://cobrapy.readthedocs.io/" title="https://cobrapy.readthedocs.io/">https://cobrapy.readthedocs.io/</a>), eQuilibrator (<a href="http://equilibrator.weizmann.ac.il/" title="http://equilibrator.weizmann.ac.il/">http://equilibrator.weizmann.ac.il/</a>), MEMOTE (<a href="https://memote.io/" title="https://memote.io/">https://memote.io/</a>) </br/></td></tr>
    <tr><td>Barry Bentley</td><td>The Open University</td><td>BioPAX, SBGN, SBML, SED-ML, CellML, SBOL and SBOL Visual, NeuroML <br/>Classifying pathology </br/></td></tr>
    <tr><td>Frank Bergmann</td><td>Heidelberg University</td><td>SBGN, SBML, SED-ML, COMBINE Archive  </td></tr>
    <tr><td>Kim Blackwell</td><td>George Mason Univeresity</td><td>SBML, NeuroML <br/>Stochastic reaction diffusion simulation using human/machine readable xml: <a href="https://github.com/neurord/stochdiff;" title="https://github.com/neurord/stochdiff;">https://github.com/neurord/stochdiff;</a> parameter optimization for neuron and signaling pathway models: <a href="https://github.com/neurord/ajustador;" title="https://github.com/neurord/ajustador;">https://github.com/neurord/ajustador;</a> </br/></td></tr>
    <tr><td>Michael Blinov</td><td>UConn Health</td><td>BioPAX, SBGN, SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/><a href="http://vcell.org" title="http://vcell.org">http://vcell.org</a> </br/></td></tr>
    <tr><td>Mehdi Borjkhani</td><td>OBi ICTER PAS</td><td>CellML, NeuroML  </td></tr>
    <tr><td>Lutz Brusch</td><td>Technische Universität Dresden, Germany</td><td>SBML, SED-ML, COMBINE Archive, Parameter estimation, Spatio-temporal multicellular models <br/>Spatio-temporal multicellular models of tissues, organs with multi-scale coupling of mechanics to and from molecular networks and cell behaviors, Morpheus <a href="https://morpheus.gitlab.io" title="https://morpheus.gitlab.io">https://morpheus.gitlab.io</a>, <a href="https://multicellml.org" title="https://multicellml.org">https://multicellml.org</a>, parameters: <a href="https://fitmulticell.gitlab.io" title="https://fitmulticell.gitlab.io">https://fitmulticell.gitlab.io</a> </br/></td></tr>
    <tr><td>Kai Budde</td><td>University of Rostock</td><td>OMEX Metadata, COMBINE Archive <br/>Provenance of simulation studies </br/></td></tr>
    <tr><td>Lukas Buecherl</td><td>CU Boulder</td><td>SBML, SED-ML, SBOL and SBOL Visual, COMBINE Archive  </td></tr>
    <tr><td>Margaux Calice</td><td>CNRS, Borelli Center</td><td>NeuroML  </td></tr>
    <tr><td>Orhan Celiker</td><td>Massachusetts Institute of Technology</td><td>NeuroML  </td></tr>
    <tr><td>Charlie Clark</td><td>University of Bristol</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Sharon Crook</td><td>Arizona State University</td><td>SED-ML, NeuroML  </td></tr>
    <tr><td>Christopher Currin</td><td>Institute of Science and Technology Austria</td><td>NeuroML <br/>Analysis and modelling of neurons derived from human induced-pluripotent stem cells </br/></td></tr>
    <tr><td>Tobias Czauderna</td><td>Monash University</td><td>SBGN <br/>www.vanted.org, <a href="http://www.sbgn-ed.org" title="www.sbgn-ed.org">www.sbgn-ed.org</a> </br/></td></tr>
    <tr><td>Andrew Davison</td><td>CNRS</td><td>NeuroML  </td></tr>
    <tr><td>Marc Dinh</td><td>INRAE</td><td>BioPAX, SBGN, SBML, COMBINE Archive  </td></tr>
    <tr><td>Ugur Dogrusoz</td><td>Bilkent University</td><td>BioPAX, SBGN <br/>Newt </br/></td></tr>
    <tr><td>Katherine Doxey</td><td>Arizona State University</td><td>NeuroML  </td></tr>
    <tr><td>Janine Egert</td><td>Medical Center Freiburg</td><td>SBML, ModeleXChange <br/>PEtab </br/></td></tr>
    <tr><td>Dany Entezari</td><td>University of Edinburgh</td><td>SBOL and SBOL Visual <br/><a href="https://www.bignumber.xyz/map" title="https://www.bignumber.xyz/map">https://www.bignumber.xyz/map</a> </br/></td></tr>
    <tr><td>Cemal Erdem</td><td>Clemson University</td><td>BioPAX, SBGN, SBML, CellML, COMBINE Archive  </td></tr>
    <tr><td>Alessandro Felder</td><td>University College London</td><td>SBML  </td></tr>
    <tr><td>Elias mateo Fernandez santoro</td><td>Erasmus MC</td><td>NeuroML  </td></tr>
    <tr><td>Pedro Fontanarrosa</td><td>University of Utah</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Andrew Freiburger</td><td>University of Victoria</td><td>SBML <br/>I am a contributing researcher in the BioSimulations project (<a href="https://run.biosimulations.org" title="https://run.biosimulations.org">https://run.biosimulations.org</a>). </br/></td></tr>
    <tr><td>Fabian Froehlich</td><td>Harvard Medical School</td><td>BioPAX, SBML, SED-ML, PEtab  </td></tr>
    <tr><td>Mauricio Garcia</td><td>University of Toronto</td><td>SBML, SBOL and SBOL Visual <br/>I'm involved in the generation and evaluation of a membrane constrained allocation model in E. coli. Also, I'm interested in developing a mechanistic model of the resource allocation in the membrane.   </br/></td></tr>
    <tr><td>Alan Garny</td><td>University of Auckland</td><td>SED-ML, CellML, OMEX Metadata, COMBINE Archive <br/>libCellML and OpenCOR </br/></td></tr>
    <tr><td>John Gennari</td><td>University of Washington</td><td>BioPAX, SBML, SED-ML, CellML, SBOL and SBOL Visual, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Padraig Gleeson</td><td>University College London</td><td>SBML, SED-ML, CellML, NeuroML <br/>Open Source Brain </br/></td></tr>
    <tr><td>Martin Golebiewski</td><td>HITS gGmbH</td><td>SBGN, SBML, SED-ML, CellML, OMEX Metadata, COMBINE Archive, Annotations; EU-STANDS4PM; MultiCellML; Medical Informatics; ISO/TC 276 Biotechnology; ISO/TC 215 Health Informatics; DIN; GA4GH <br/>NFDI4Health (<a href="http://www.nfdi4health.de" title="www.nfdi4health.de">www.nfdi4health.de</a>); Liver Systems Medicine LiSyM (<a href="http://www.lisym.org" title="www.lisym.org">www.lisym.org</a>); EU-STANDS4PM (<a href="http://www.eu-stands4pm.eu" title="www.eu-stands4pm.eu">www.eu-stands4pm.eu</a>); SABIO-RK (sabiork.h-its.org); de.NBI (<a href="http://www.denbi.de" title="www.denbi.de">www.denbi.de</a>); FAIRDOM (fair-dom.org); ELIXIR (elixir-europe.org); ISO/TC 276 </br/></td></tr>
    <tr><td>Lyle Graham</td><td>Centre Giovanni Borelli - CNRS UMR9010</td><td>NeuroML  </td></tr>
    <tr><td>Jan Grzegorzewski</td><td>Humboldt Universität</td><td>SBML <br/> PK-DB is a database and web interface for pharmacokinetics data and information from clinical trials as well as pre-clinical research. PK-DB allows to curate pharmacokinetics data integrated with the corresponding meta-information. </br/></td></tr>
    <tr><td>Lea Gütebier</td><td>University of Greifswald</td><td>SBGN, SBML, SED-ML, CellML, COMBINE Archive  </td></tr>
    <tr><td>Thomas Hamm</td><td>Eberhard Karls Universität Tübingen</td><td>SBML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Jan Hasenauer</td><td>University of Bonn</td><td>SBML, SED-ML <br/>PEtab, AMICI, pyPESTO, pyABC </br/></td></tr>
    <tr><td>Benjamin Hatch</td><td>University of Utah</td><td>SBOL and SBOL Visual, COMBINE Archive <br/>SynBioHub 3 </br/></td></tr>
    <tr><td>Thorsten Hater</td><td>JSC/FZJ</td><td>NeuroML <br/>HBP/eBrains </br/></td></tr>
    <tr><td>Tomas Helikar</td><td>University of Nebraska - Lincoln</td><td>SBML, SED-ML <br/>Multi-scale modeling of the immune system; software development for computational modeling; <a href="https://cellcollective.org;" title="https://cellcollective.org;">https://cellcollective.org;</a> <a href="http://helikarlab.org" title="http://helikarlab.org">http://helikarlab.org</a> </br/></td></tr>
    <tr><td>Joseph Hellerstein</td><td>University of Washington</td><td>SBGN, SBML, CellML, SBOL and SBOL Visual <br/>SBstoat - parameter fitting (<a href="https://github.com/sys-bio/SBstoat" title="https://github.com/sys-bio/SBstoat">https://github.com/sys-bio/SBstoat</a>); SBMLLint (<a href="https://github.com/ModelEngineering/SBMLLint" title="https://github.com/ModelEngineering/SBMLLint">https://github.com/ModelEngineering/SBMLLint</a>) </br/></td></tr>
    <tr><td>Ron Henkel</td><td>University Medicine Greifswald</td><td>SBML, SED-ML, CellML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Henning Hermjakob</td><td>EMBL-EBI</td><td>BioPAX, SBGN, OMEX Metadata, COMBINE Archive <br/><a href="https://www.ebi.ac.uk/biomodels/" title="https://www.ebi.ac.uk/biomodels/">https://www.ebi.ac.uk/biomodels/</a> </br/></td></tr>
    <tr><td>Adel Heydarabadipour</td><td>Sharif University of Technology</td><td>BioPAX, SBGN, SBML, SED-ML, CellML, SBOL and SBOL Visual, NeuroML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Sedigheh Hosseinipanah</td><td>Shiraz University of Medical Sciences</td><td>BioPAX, CellML, NeuroML  </td></tr>
    <tr><td>Brent Huisman</td><td>FZJ</td><td>NeuroML <br/><a href="https://arbor-sim.github.io/" title="https://arbor-sim.github.io/">https://arbor-sim.github.io/</a> </br/></td></tr>
    <tr><td>Maziya Ibrahim</td><td>Indian Institute of Technology Madras (IITM), India</td><td>SBML  </td></tr>
    <tr><td>Jacob Jett</td><td>University of Illinois at Urbana-Champaign</td><td>SBOL and SBOL Visual <br/>Synthetic Biology Knowledge Systems </br/></td></tr>
    <tr><td>Jonathan Karr</td><td>Icahn School of Medicine at Mount Sinai</td><td>SBML, SED-ML, OMEX Metadata, COMBINE Archive, modelXchange <br/>BioSimulators (<a href="https://biosimulators.org" title="https://biosimulators.org">https://biosimulators.org</a>), BioSimulations (<a href="https://biosimulations.org" title="https://biosimulations.org">https://biosimulations.org</a>), Datanator (<a href="https://datanator.info" title="https://datanator.info">https://datanator.info</a>) </br/></td></tr>
    <tr><td>Michiel Karrenbelt</td><td>ETH Zurich</td><td>SBML, SED-ML, CellML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Kevin Keating</td><td>Worcester Polytechnic Institute</td><td>SBOL and SBOL Visual <br/>SBKS (<a href="https://synbioks.github.io/" title="https://synbioks.github.io/">https://synbioks.github.io/</a>), SynBioHub (as a user) </br/></td></tr>
    <tr><td>Sarah Keating</td><td>UCL</td><td>SBML, SED-ML, OMEX Metadata <br/>SBML infrastructure </br/></td></tr>
    <tr><td>Svenja Kemmer</td><td>Institute of Physics, University of Freiburg</td><td>SBML, PEtab <br/>PEtab development (<a href="https://petab.readthedocs.io;" title="https://petab.readthedocs.io;">https://petab.readthedocs.io;</a> <a href="https://github.com/PEtab-dev/PEtab" title="https://github.com/PEtab-dev/PEtab">https://github.com/PEtab-dev/PEtab</a>) </br/></td></tr>
    <tr><td>Haseong Kim</td><td>Korea research institute of bioscience and biotechnology </td><td>SBML, CellML, SBOL and SBOL Visual  </td></tr>
    <tr><td>Mohit Kulkarni</td><td>IIT Kanpur</td><td>CellML, NeuroML  </td></tr>
    <tr><td>Matthias König</td><td>Humboldt University Berlin</td><td>SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/>roadrunner, tellurium, sbmlsim, sbmlutils, pkdb </br/></td></tr>
    <tr><td>Polina Lakrisenko</td><td>Helmholtz Zentrum München, ICB</td><td>SBML, COMBINE Archive  </td></tr>
    <tr><td>Rahmi Lale</td><td>Norwegian University of Science and Technology, Department of Biotechnology</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Lennart Landsmeer</td><td>Erasmus MC</td><td>SBML, NeuroML <br/>ArborIO </br/></td></tr>
    <tr><td>Nantia Leonidou</td><td>Institute for Bioinformatics and Medical Informatics (IBMI)</td><td>SBML, CellML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Charl Linssen</td><td>Forschungszentrum Jülich</td><td>SBML, CellML, NeuroML <br/>NESTML: computational neuroscience domain-specific language for spiking neuron and synaptic plasticity models </br/></td></tr>
    <tr><td>Augustin Luna</td><td>Dana-Farber Cancer Institute</td><td>BioPAX, SBGN <br/>Pathway Commons (pathwaycommons.org; data stored in BioPAX and visualized in SBGN) </br/></td></tr>
    <tr><td>Payam  M hussein</td><td>Tishk international university</td><td>SBML, SED-ML, CellML  </td></tr>
    <tr><td>Chitaranjan Mahapatra</td><td>University of California San Francisco</td><td>CellML, NeuroML <br/><a href="https://github.com/cmahapatra97/Chitaranjan-Mahapatra" title="https://github.com/cmahapatra97/Chitaranjan-Mahapatra">https://github.com/cmahapatra97/Chitaranjan-Mahapatra</a> </br/></td></tr>
    <tr><td>Jeanet Mante</td><td>CU Boulder</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Jeanet Mante</td><td>CU Boulder</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Boris Marin</td><td>Universidade Federal do ABC</td><td>NeuroML  </td></tr>
    <tr><td>James Mclaughlin</td><td>EMBL-EBI</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Pedro Mendes</td><td>University of Connecticut</td><td>SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/>COPASI </br/></td></tr>
    <tr><td>Ion Moraru</td><td>UConn Health</td><td>SBML, SED-ML, COMBINE Archive <br/>vcell.org, biosimulators.org, biosimulations.org </br/></td></tr>
    <tr><td>Sébastien Moretti</td><td>SIB Swiss Institute of Bioinformatics</td><td>SBML <br/>MetaNetX/MNXref </br/></td></tr>
    <tr><td>Ricardo Murphy</td><td>University of Oslo</td><td>NeuroML  </td></tr>
    <tr><td>Chris Myers</td><td>University of Colorado Boulder</td><td>SBML, SED-ML, SBOL and SBOL Visual, OMEX Metadata, COMBINE Archive <br/>iBioSim, SynBioHub, SBOLDesigner, SBOLCanvas, VisBol </br/></td></tr>
    <tr><td>Giulio Napolitano</td><td>Centogene</td><td>BioPAX, SBGN, SBML  </td></tr>
    <tr><td>Maxwell Neal</td><td>Seattle Children's Research Institute</td><td>BioPAX, SBML, SED-ML, CellML, OMEX Metadata, COMBINE Archive <br/>SemGen (<a href="https://github.com/SemBioProcess/SemGen" title="https://github.com/SemBioProcess/SemGen">https://github.com/SemBioProcess/SemGen</a>) </br/></td></tr>
    <tr><td>Tung Nguyen</td><td>European Bioinformatics Institute (EMBL-EBI)</td><td>BioPAX, SBGN, SBML, SED-ML, CellML, SBOL and SBOL Visual, NeuroML, OMEX Metadata, COMBINE Archive <br/>BioModels, OmicsDI, TransQST </br/></td></tr>
    <tr><td>Anna Niarakis</td><td>Université d'Evry Val de Essonne</td><td>SBGN, SBML, SED-ML <br/>The COVID19 Disease Map project  </br/></td></tr>
    <tr><td>David Nickerson</td><td>Auckland Bioengineering Institute, University of Auckland</td><td>SED-ML, CellML, OMEX Metadata, COMBINE Archive <br/>Center for Reproducible Biomedical Modeling; Physiome Model Repository </br/></td></tr>
    <tr><td>Vincent Noel</td><td>Institut Curie - PSL University</td><td>SBGN, SBML, SED-ML <br/>SBGNRenderAPI <a href="https://github.com/vincent-noel/SBGNRenderAPI" title="https://github.com/vincent-noel/SBGNRenderAPI">https://github.com/vincent-noel/SBGNRenderAPI</a> </br/></td></tr>
    <tr><td>Elijah Noisin</td><td>Georgia Gwinette College </td><td>NeuroML <br/>Simulating Spontaneous Rhythmic Currents (SRC's) using NEURON. The Drosophila melanogaster (fruit fly-_-) aCC motor neuron is the imitated neuron chosen.  </br/></td></tr>
    <tr><td>Elad Noor</td><td>Weizmann Institute of Science</td><td>SBML  </td></tr>
    <tr><td>Erick Olivares</td><td>UTSA</td><td>NeuroML <br/>Neural network models of the basal ganglia </br/></td></tr>
    <tr><td>Brett Olivier</td><td>Vrije Universiteit Amsterdam</td><td>SBML, COMBINE Archive <br/>SBML, FBC, FROG </br/></td></tr>
    <tr><td>Sudhir Pathak</td><td>University of Pittsburgh</td><td>NeuroML  </td></tr>
    <tr><td>Dilan Pathirana</td><td>University of Bonn</td><td>SBML, PEtab <br/>PEtab extensions (model selection; parameter timecourses) </br/></td></tr>
    <tr><td>Paul Pfeiffer</td><td>HU Berlin</td><td>NeuroML <br/>In my work, I use Brian2, AUTO and dynamic clamp software like RELACS or RTXI. </br/></td></tr>
    <tr><td>Veronica Porubsky</td><td>University of Washington</td><td>NeuroML <br/>Center for Reproducible Biomedical Modeling </br/></td></tr>
    <tr><td>Norma  Pérez </td><td>CINVESTAV</td><td>SBGN, SBML  </td></tr>
    <tr><td>Lavanya Raajaraam</td><td>IIT Madras</td><td>SBML, COMBINE Archive <br/>Reproducibility analysis of constraint-based models:FROG analysis  </br/></td></tr>
    <tr><td>Kausthubh Ramachandran</td><td>EMBL-EBI</td><td>SBML, SED-ML, CellML, COMBINE Archive  </td></tr>
    <tr><td>Karthik Raman</td><td>Indian Institute of Technology Madras</td><td>SBGN, SBML <br/>SBML FBC, FBC curation </br/></td></tr>
    <tr><td>Anand Rampadarath</td><td>Auckland Bioengineering Institute</td><td>SBML, SED-ML, CellML, NeuroML, OMEX Metadata, COMBINE Archive <br/>CellML, LibOmexMeta, COMBINE archive </br/></td></tr>
    <tr><td>Achilles Rasquinha</td><td>University of Nebraska-Lincoln</td><td>SBGN, SBML, SED-ML, CellML <br/>cellcollective.org </br/></td></tr>
    <tr><td>Nicholas Rodriguez</td><td>Virginia Commonwealth University</td><td>BioPAX, SBGN, SBML, SED-ML, CellML, SBOL and SBOL Visual, NeuroML, OMEX Metadata, COMBINE Archive <br/>SBKS </br/></td></tr>
    <tr><td>Nicholas Roehner</td><td>Raytheon BBN Technologies</td><td>SBGN, SBML, SBOL and SBOL Visual  </td></tr>
    <tr><td>Andres Romanowski</td><td>University of Edinburgh</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>David Ross</td><td>NIST</td><td>SBOL and SBOL Visual <br/>Software for analysis of massively multiplexed genotype-phenotype data: <a href="https://github.com/djross22/nist_lacI_landscape_analysis" title="https://github.com/djross22/nist_lacI_landscape_analysis">https://github.com/djross22/nist_lacI_landscape_analysis</a>. Automated flow cytometry gating software: <a href="https://github.com/djross22/flowgatenist" title="https://github.com/djross22/flowgatenist">https://github.com/djross22/flowgatenist</a>. </br/></td></tr>
    <tr><td>Adrien Rougny</td><td>AIST</td><td>SBGN <br/>SBGN Bricks Ontology; stonpy; Reusability and composability in molecular networks </br/></td></tr>
    <tr><td>Sven Sahle</td><td>Heidelberg University</td><td>SBGN, SBML, SED-ML, COMBINE Archive  </td></tr>
    <tr><td>Sai Samineni</td><td>CU Boulder</td><td>SBML, CellML, SBOL and SBOL Visual, NeuroML  </td></tr>
    <tr><td>Herbert Sauro</td><td>University of Washington</td><td>SBGN, SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/>SBML Layout and Render </br/></td></tr>
    <tr><td>Laura Schmid</td><td>University of Stuttgart</td><td>SED-ML, CellML, NeuroML  </td></tr>
    <tr><td>James Scholz</td><td>University of Utah (Genetic Logic Lab)</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Falk Schreiber</td><td>University of Konstanz</td><td>SBGN, SBML, SBOL and SBOL Visual <br/>SBGN-ED </br/></td></tr>
    <tr><td>Yannik Schälte</td><td>Helmholtz Center Munich</td><td>SBML, PEtab  </td></tr>
    <tr><td>James Scott-brown</td><td>University of Oxford</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Vinoo Selvarajah</td><td>iGEM Foundation</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Brandon Sepulvado</td><td>NORC at the University of Chicago</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Ghayoomie Seyed vahid</td><td>OpenWorm</td><td>BioPAX, SBGN, SBML, SED-ML, CellML, SBOL and SBOL Visual, NeuroML <br/>Ion Channel Models </br/></td></tr>
    <tr><td>Maryam Shah</td><td>Precision Medicine Lab, Rehman Medical Institute</td><td>BioPAX, SBGN, SBML, SBOL and SBOL Visual, NeuroML <br/>Protein-Protein interaction networks </br/></td></tr>
    <tr><td>Bilal Shaikh</td><td>Mount Sinai</td><td> <br/>Biosimulations </br/></td></tr>
    <tr><td>Rahuman Sheriff</td><td>EMBL-EBI</td><td>SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/>BioModels, Reproducibility </br/></td></tr>
    <tr><td>Prubhtej Singh</td><td>GGSIP University, New Delhi, India</td><td>SBOL and SBOL Visual, NeuroML <br/>SBOL, SynBioHub </br/></td></tr>
    <tr><td>Ankur Sinha</td><td>University College London</td><td>NeuroML <br/>Open Source Brain </br/></td></tr>
    <tr><td>Marina Slashcheva</td><td>Karolinska Institute</td><td>NeuroML  </td></tr>
    <tr><td>Elliot Smith</td><td>Arizona State University</td><td>NeuroML  </td></tr>
    <tr><td>Lucian Smith</td><td>University of Washington</td><td>SBML, SED-ML, COMBINE Archive <br/>Tellurium, Roadrunner, Antimony </br/></td></tr>
    <tr><td>Hugh Sorby</td><td>Auckland Bioengineering Institute</td><td>CellML, OMEX Metadata  </td></tr>
    <tr><td>Jörn Starruß</td><td>TU Dresden</td><td>SBML, SED-ML, CellML, PEtab <br/>Morpheus - Multicellular modelling framework (morpheus.gitlab.io)) </br/></td></tr>
    <tr><td>Jake Sumner</td><td>Keele University</td><td>SBOL and SBOL Visual <br/>An SBOL Visual Tool  for converting JSON formatted data into Glyphs: <a href="https://jakesumner11.github.io/sbolvisual/" title="https://jakesumner11.github.io/sbolvisual/">https://jakesumner11.github.io/sbolvisual/</a> . It is still very early in development and will require changes. </br/></td></tr>
    <tr><td>Diana Szeliova</td><td>University of Vienna, Vienna, Austria</td><td>SBML  </td></tr>
    <tr><td>Sára Sáray</td><td>Institute of Experimental Medicine</td><td>SBML, CellML, NeuroML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Luca Tar</td><td>Institute of Experimental Medicine</td><td>SBML, CellML, NeuroML  </td></tr>
    <tr><td>Ysis Tarter</td><td>amyris</td><td>SBOL and SBOL Visual, NeuroML  </td></tr>
    <tr><td>Logan Terry</td><td>University of Utah</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Payton Thomas</td><td>University of Utah</td><td>SBML, SED-ML, SBOL and SBOL Visual, COMBINE Archive <br/>Stochastic analysis of cellular dynamics </br/></td></tr>
    <tr><td>Krishna Tiwari</td><td>EMBL-EBI</td><td>SBGN, SBML, COMBINE Archive <br/>Models citation and reproducibility. </br/></td></tr>
    <tr><td>Niccolo Totis</td><td>ku leuven</td><td>SBML  </td></tr>
    <tr><td>David Tourigny</td><td>Columbia University</td><td>SBML, SED-ML  </td></tr>
    <tr><td>Vasundra Touré</td><td>SIB</td><td>SBGN  </td></tr>
    <tr><td>Haddock Traci</td><td>iGEM Foundation</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Uriel Urquiza-garcia</td><td>Dusseldorf University </td><td>SBML, SBOL and SBOL Visual, COMBINE Archive  </td></tr>
    <tr><td>Prashant Vaidyanathan</td><td>Microsoft</td><td>SBOL and SBOL Visual  </td></tr>
    <tr><td>Jakob Vanhoefer</td><td>Universität Bonn</td><td>SBML <br/>pyPESTO developer, yaml2sbml developer, interested in PEtab </br/></td></tr>
    <tr><td>Gonzalo Vidal</td><td>Pontificia Universidad Católica de Chile</td><td>SBML, SED-ML, SBOL and SBOL Visual <br/>I’m developing Flapjack a data management and analysis tool for genetic circuit characterization <a href="https://doi.org/10.1021/acssynbio.0c00554" title="https://doi.org/10.1021/acssynbio.0c00554">https://doi.org/10.1021/acssynbio.0c00554</a> and LOICA a Python package to model and design genetic circuits <a href="https://github.com/SynBioUC/LOICA" title="https://github.com/SynBioUC/LOICA">https://github.com/SynBioUC/LOICA</a>  </br/></td></tr>
    <tr><td>Aayushi Vishnoi</td><td>IISER Bhopal</td><td>BioPAX, SBML, NeuroML, COMBINE Archive <br/>I am involved in making machine learning models for C elegans olfaction. </br/></td></tr>
    <tr><td>Steffen Waldherr</td><td>KU Leuven</td><td>SBML  </td></tr>
    <tr><td>Dagmar Waltemath</td><td>University Medicine Greifswald</td><td>SED-ML, CellML, OMEX Metadata, COMBINE Archive, ModelXchange <br/>COVID Archives </br/></td></tr>
    <tr><td>Andreas Weidemann</td><td>HITS gGmbH</td><td>SBML, OMEX Metadata, COMBINE Archive  </td></tr>
    <tr><td>Daniel Weindl</td><td>Helmholtz Zentrum München</td><td>SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/><a href="https://github.com/PEtab-dev/PEtab" title="https://github.com/PEtab-dev/PEtab">https://github.com/PEtab-dev/PEtab</a>  Data format for SysBio parameter estimation problems <a href="https://github.com/ICB-DCM/AMICI" title="https://github.com/ICB-DCM/AMICI">https://github.com/ICB-DCM/AMICI</a> <a href="https://github.com/ICB-DCM/pyPESTO" title="https://github.com/ICB-DCM/pyPESTO">https://github.com/ICB-DCM/pyPESTO</a> Parameter estimation / simulation of ODE models </br/></td></tr>
    <tr><td>Franz-georg Wieland</td><td>Uni Freiburg, Institute of Physics</td><td>SBML, SED-ML <br/>PeTab </br/></td></tr>
    <tr><td>Judith Wodke</td><td>Theoretical Biophysics, Humboldt-Universität zu Berlin</td><td>SBML, SED-ML, SBOL and SBOL Visual  </td></tr>
    <tr><td>Pranay Yadav</td><td>National Institute of Advanced Studies</td><td>NeuroML  </td></tr>
    <tr><td>Eric Young</td><td>Worcester Polytechnic Institute</td><td>SBOL and SBOL Visual <br/>Synthetic Biology Knowledge System, Prymetime genome assembly software, combinatorial library design (DoubleDutch) </br/></td></tr>
    <tr><td>Eric Yu</td><td>University of Utah</td><td>SBOL and SBOL Visual <br/>SynBioHub, SBOLExplorer </br/></td></tr>
    <tr><td>Fengkai Zhang</td><td>NIH</td><td>SBGN, SBML, SED-ML, OMEX Metadata, COMBINE Archive <br/>Simmune, SBML </br/></td></tr>
    <tr><td>Tomasz Zielinski</td><td>University of Edinburgh</td><td>SBML, SBOL and SBOL Visual, COMBINE Archive <br/>Lead developer of BioDare2 (<a href="https://biodare2.ed.ac.uk" title="https://biodare2.ed.ac.uk">https://biodare2.ed.ac.uk</a>) a repository for circadian, biological data, providing a platform for data sharing and period analysis. </br/></td></tr>
    <tr><td>Zachary Zundel</td><td>Facebook, Inc. </td><td>SBOL and SBOL Visual, OMEX Metadata, COMBINE Archive  </td></tr>
  </table>  </div>

  <div class="node-links">
      </div>

  
</div> <!-- /.node -->




          </div>

        
        
      </div></div> <!-- /.section, /#content -->

      <div id="navigation"><div class="section clearfix">

	           <div id="suckerfishmenu">
            <ul class="menu"><li class="leaf first"><a href="https://combine-org.github.io/events/" title="The annual COMBINE meeting will be held Oct 6-7, 2022, at the Humboldt University of Berlin, Germany.">COMBINE 2022</a></li>
<li class="leaf"><a href="/standards" title="Standards covered by or related to COMBINE activities">Standards</a></li>
<li class="leaf"><a href="/events" title="Events">Events</a></li>
<li class="collapsed"><a href="/documents" title="Documents">Documents</a></li>
<li class="leaf"><a href="/comm" title="Mailing lists and forums of discussion">Forums</a></li>
<li class="leaf last"><a href="/about" title="About">About</a></li>
</ul>          </div>
        
                  <div id="search-box">
            <form action="/events/HARMONY_2021/Attendees"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-Que7xGL6jEvYRfRyId3qwF8J3873ULK0Wz0ghc6LpDA" value="form-Que7xGL6jEvYRfRyId3qwF8J3873ULK0Wz0ghc6LpDA"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
          </div> <!-- /#search-box -->
        
        
      </div></div> <!-- /.section, /#navigation -->

      <div class="region region-sidebar-first column sidebar"><div class="section">
  
<div id="block-menu-menu-combine-left-menu" class="block block-menu first last region-odd odd region-count-1 count-1  block-1"><div class="block-inner">

      <h2 class="title">COMBINE</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first"><a href="/home" title="The home page of COMBINE">Home</a></li>
<li class="leaf"><a href="/help" title="">Help</a></li>
<li class="leaf last"><a href="/login" title="">Sign-in</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
</div></div><!-- /.section, /.region -->

      
    </div></div> <!-- /#main, /#main-wrapper -->

          <div id="footer"><div class="section">

        
        <div class="region region-footer">
  
<div id="block-block-1" class="block block-block first last region-odd even region-count-1 count-2  block-1"><div class="block-inner">

      <h2 class="title">COmputational Modeling in BIology NEtwork</h2>
  
  <div class="content">
    <p>COmputational Modeling in BIology NEtwork </p> <!-- NewPP limit report Cached time: 20220802130129 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.005 seconds Real time usage: 0.006 seconds Preprocessor visited node count: 1/1000000 Preprocessor generated node count: 4/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  
</div></div> <!-- /block-inner, /block -->
</div><!-- /.region -->
<div class="element-hidden" id="visible">
<a href="http://www.merrellpublishers.com/buy_meds.htm">cheap medicaments usa</a> 
<a href="http://www.drleanawen.com/styled-2/styled-9/commentary.html">generic sildenafil</a>
</div>
      </div></div> <!-- /.section, /#footer -->
    
  </div></div> <!-- /#page, /#page-wrapper -->

  
  
</body>
</html>
