
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <title>COMBINE 2013 - Abstracts | COMBINE</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="/sites/combine/files/Symbol-COMBINE_logo.ico" type="image/x-icon" />
  <link type="text/css" rel="stylesheet" media="all" href="/modules/node/node.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/poll/poll.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/defaults.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/system/system-menus.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/modules/user/user.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/ctools/css/ctools.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/panels/css/panels.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/modules/views/css/views.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/html-reset.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/wireframes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/tabs.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/messages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/pages.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/block-editing.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/blocks.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/navigation.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/nodes.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/comments.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/forms.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/cti-flex2.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/combine-custom.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/layout-fixed.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/colors/combine.css?H" />
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/local.css?H" />
<link type="text/css" rel="stylesheet" media="print" href="/sites/all/themes/cti_flex2/css/print.css?H" />
<!--[if IE]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie.css?H" />
<![endif]-->
<!--[if lte IE 6]>
<link type="text/css" rel="stylesheet" media="all" href="/sites/all/themes/cti_flex2/css/ie6.css?H" />
<![endif]-->
  <script type="text/javascript" src="/misc/jquery.js?H"></script>
<script type="text/javascript" src="/misc/drupal.js?H"></script>
<script type="text/javascript" src="/sites/all/modules/panels/js/panels.js?H"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, { "basePath": "/" });
//--><!]]>
</script>
  <style type="text/css">
body {font-family: Tahoma, Verdana, Arial, Helvetica, "Bitstream Vera Sans", sans-serif;}
</style>  <style type="text/css">
</style>
  <!--[if lte IE 6]>
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/suckerfish.js"></script>
  <![endif]-->
  <script type="text/javascript" src="/sites/all/themes/cti_flex2/js/combine.js"></script>
</head>

<body class="page not-front not-logged-in node-type-page page-events-combine-2013-abstracts section-events one-sidebar sidebar-first page-panels ">

  <div id="page-wrapper"><div id="page">

    <div id="header"><div class="section clearfix">

      
              <div id="logo-title">

                      <div id="logo"><a href="/" title="Home" rel="home"><img src="/sites/combine/files/COMBINE_website-banner.png" alt="Home" id="logo-image" /></a></div>
          
      
        </div> <!-- /#logo-title -->
      
      
    </div></div> <!-- /.section, /#header -->

    <div id="main-wrapper"><div id="main" class="clearfix  with-navigation">

      <div id="content" class="column"><div class="section">

        
        
        <div class="breadcrumb"><a href="/">Home</a> › COMBINE 2013 - Abstracts</div>                  <h1 class="title">COMBINE 2013 - Abstracts</h1>
                                
        
        <div id="content-area">
          <div id="node-141" class="node node-type-page build-mode-full clearfix">

   
  
  
  
  
  <div class="content">
    <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>Modelling Colonic Crypts with VCell and SBML/Spatial</b></i> </td></tr> <tr> <td> Marco Antoniotti, Anuradha Lakshminarayana, Carlo Maj, Daniele Ramazzotti, Jim Schaff (DISCO UNIMIB and UCHC), Department of Informatics, Systems and Communications, University degli Studi Milano Bicocca <br /><br /> </td></tr> <tr> <td> The physiological working of colonic crypts is dependent on the specific dynamic spatial organization of various cell types. The colonic crypt is characterized by a high rate cell renewal, and the proper functioning of the structure is ensured by a complex cell differentiation process occurring over different tissue layers. Proper simulations of colonic crypt dynamic requires the modeling of both temporal and spatial dimensions. To this end, we performed a modeling exercise using the recently developed SBML3 Spatial Processes package, in conjunction with the NRCAM UConn VCell software in order to model colonic crypts. The SBML3 Spatial Processes package allows for the representation of geometric and topological features on top of the normal dynamic modeling SBML capabilities. We verified that with the proper use of the different SBML features it is possible to define an extensive model of colonic cryptts composed of the main cellular types (from stem cells to fully differentiated cells), alongside their dynamics. The different cell types have been arranged to form a hollow prism placed in a 3-dimensional cartesian reference frame mimicking the structure of the colonic crypt. Despite the elementary and approximated model structure (and therefore less relevant from the biological viewpoint, compared to several already developed and published models), we have already seen that our model is capable of capturing important cell crypt structure and behavior. As such, this provides one of the first validations of the SBML3 Spatial Processes package, applied to a non-trivial system. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>Sharing, Versioning and Annotating SBML Models using the e!DAL Data Repository API</b></i> </td></tr> <tr> <td> Daniel Arend 1 , Jinbo Chen 1 , Anja Hartmann 1 , Tobias Czauderna 1 , Uwe Scholz 1 , Falk Schreiber 1,2 and Matthias Lange 1 1 - Leibniz Institute of Plant Genetics and Crop Plant Research, Gatersleben, Germany 2 - Institute of Computer Science, Martin Luther University Halle-Wittenberg, Halle, Germany <br /><br /> </td></tr> <tr> <td> Scientific collaboration induces complex requirements for information man- agement. In order to enable community-based development and sharing of biological models, a sustainable data sharing infrastructure must be pro- vided. Several approaches exist: for example, BioModels reflect the need for an integrated repository of curated models. However, this approach does not support community-based editing and versioning. The COMBINE archive is a data format for storing models, associated data and procedures, but does not provide a storage infrastructure for collaborative data sharing or publica- tion. WikiPathways is a collaborative platform for the curation of biological pathways but focuses on graphical pathway representation and not on SBML models. To address those shortcomings, we propose the application of the e!DAL data repository (Arend et al., 2012) to systems biology data. e!DAL (electro- nical Data Archive Library) provides an enhanced storage back-end, similar to a file system, for model data. Main features are version history, metadata management, citable data publication and support for information retrieval. It can be used either as a single instance application embedded storage back- end or as a centrally hosted repository, which allows collaborative editing and access to models among different partners. The e!DAL-API has been designed and tested using experiences from cross domain research projects and combines features known from file systems, databases, and content man- agement systems. In order to evaluate its capability for systems biology tools, an e!DAL- Vanted plug-in has been implemented. Using the Vanted (Rohn et al., 2012) plug-in architecture, the ordinary SBML file storage dialog has been en- hanced towards an extended model sharing infrastructure. The user has the option to select a local or a shared e!DAL repository. Version control, tech- nical metadata, as well as search within the models are supported. Finally, models can be published as URLs using the embedded HTTP service or as DOIs using DataCite infrastructure. Arend, D. et al. (2012). The e!DAL JAVA-API: Store, Share and Cite Primary Data in Life Sciences. In: IEEE International Conference on Bioinformatics and Biomed Ed. by J. Gao et al. IEEE Catalog Number: CFP12BIB-USB, pp. 511-515. Rohn, Hendrik et al. (2012). VANTED v2: a framework for systems biology applications. In: BMC Systems Biology 6.1, p. 139. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> SBML importer plugin for PathVisio </b></i> </td></tr> <tr> <td> Anwesha Bohler 1,2, Sri Harsha P 3, Martijn van Iersel 4, Alexander R. Pico 5, Chris T Evelo 1 1 BiGCaT- Department of Bioinformatics, Maastricht University, The Netherlands 2 Netherlands Consortium of Systems Biology (NCSB), The Netherlands 3 Keshav Memorial Institute of Technology, Hyderabad, Andhra Pradesh, India 4 General Bioinformatics, Reading, UK 5 Gladstone Institutes, San Francisco, California, USA <br /><br /> </td></tr> <tr> <td>Biological models are abstract representations of biological processes. Models can be diagrammatic, e.g pathways, serving as frameworks to visualize and integrate experimental measurements of genes and gene products. Models can also be mathematical, describing the biological process by means of various equations. Computational biologists use mathematical models to predict how a process might be affected by environmental or internal stimuli. SBML is a standard format widely accepted for storing and exchanging such mathematical models. PathVisio (pathvisio.org) is an open source pathway creating, editing, visualization, and analysis tool. It uses GPML to store and exchange pathways. The new SBML importer plugin for PathVisio facilitates model curation and extension by providing modelers with an up-to-date graphical (SBGN-PD) representation of their model. This representation can then be compared to existing pathways for the same process, which will facilitate both pathway improvement and critical assessment of model implementation aspects like lumping of reaction steps and reduction of parallel routes. Existing PathVisio functionality enables model outcomes to be graphically visualized on the model representation to ease comprehension. The plugin also enables model validation and direct import of SBML models from Biomodels. Furthermore, imported models can then be shared with the community for distribution and further curation through WikiPathways (wikipathways.org), which also uses GPML as its native format. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>Mathematical model of synthetic dosage gene interactions leading to EMT-like phenotype in vivo </b></i> </td></tr> <tr> <td> David Cohen, Andrei Zinovyev and Laurence Calzone, Institut Curie, Paris. <br /><br /> </td></tr> <tr> <td>Cell lineage tracing has previously shown that a NICD gain of function combined with a p53 loss of function can lead to the appearance of metastases in the intestine of mice. In these mice, metastases follow up the Epithetial to Mesenchymal transition (EMT), which has been reported to be an important event in cancer: cells can detach from their neighbours and migrate throughout the body to form new tumours. In this study, we constructed a mathematical model that simulates the mutant behaviour using logic formalism. The model shows the antagonistic role of microRNA and p53 and the agonistic role of Notch in the occurrence of EMT. Furthermore, the model predicts putative rescue mutants to escape EMT. The reduction of the initial network into a minimal model that still mimics the mutant behaviour highlights some motif requirement to account for the role of p53 in the regulation of EMT. The same motifs can be found in the G1/S phase transition in the cell cycle. This might indicate that the tumour suppressor p53 acts as a putative checkpoint in EMT. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Visualising biological networks in SBGN using VANTED and its SBGN-ED add-on </b></i> </td></tr> <tr> <td> POSTER ONLY Tobias Czauderna(1), Michael Wybrow(2), Torsten Vogt(1), Kim Marriott(2), Falk Schreiber(1,3) 1 Leibniz Institute of Plant Genetics and Crop Plant Research (IPK) Gatersleben, Germany 2 Caulfield School of Information Technology, Monash University, Australia 3 Institute of Computer Science, Martin Luther University Halle-Wittenberg, Germany <br /><br /> </td></tr> <tr> <td>The Systems Biology Graphical Notation (SBGN, <a rel="nofollow" class="external free" href="http://www.sbgn.org">http://www.sbgn.org</a>) is a standard for the visual representation of biochemical and cellular processes and networks studied in systems biology. Three different languages (PD - Process Description, ER - Entity Relationship, and AF - Activity Flow) cover several aspects of biological systems in different levels of detail. SBGN helps to communicate knowledge more efficient and accurate between different research communities. To support SBGN, methods and tools for editing, validating, and translating of SBGN maps are necessary. VANTED (<a rel="nofollow" class="external free" href="http://www.vanted.org">http://www.vanted.org</a>) is an integrative framework for systems biology applications. It aims at the integration, analysis and visual exploration of experimental data in the context of biological networks. The framework comprises the VANTED core as well as the capability for various extensions called add-ons. The add-ons are being developed for the needs of life scientists and extend the functionality of the VANTED core towards various tasks and topics in systems biology. One of these add-ons is SBGN-ED (<a rel="nofollow" class="external free" href="http://www.sbgn-ed.org">http://www.sbgn-ed.org</a>). SBGN-ED allows creating and editing all types of SBGN maps. Furthermore the syntactical and semantical correctness of created or edited maps can be validated. Already existing non-SBGN maps from the KEGG database can be translated into SBGN PD maps including automatic layout. A visualisation of SBML models in SBGN PD is also provided. However, this feature currently requires manual post-editing of the map layout. Additionally the tool allows exporting of SBGN maps into several file and image formats including the SBGN-ML format (LibSBGN, <a rel="nofollow" class="external free" href="http://www.sbgn.org/LibSBGN">http://www.sbgn.org/LibSBGN</a>). </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Numerical Markup Language and LibNUML </b></i> </td></tr> <tr> <td> Joseph O. Dada, University of Manchester. <br /><br /> </td></tr> <tr> <td> Numerical Markup Language and LibNUML Numerical Markup Language (NuML) is a standardization effort to provide a universal language for encoding numerical data. NuML originated from the numerical aspects of Systems Biology Results Markup Language (SBRML) with the aim of re-using it in multiple other standardization efforts. It is envisioned that NuML will provide a standardized and universal way for exchanging and archiving numerical results. To facilitate a convenient and efficient processing of NuML, a library (libNUML) for reading, writing, manipulating and validating data encoded in NuML has been developed. LibNUML is written in C/C++ with bindings in other major languages such as Java and C#. It offers many benefits that are not readily available in the off-the-shelf XML parser libraries, such as consistency and validity checking of NuML. The talk will present an overview of NuML and libNuML with some examples. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> The Systems Biology Simulation Core Library </b></i> </td></tr> <tr> <td> Andreas Draeger, University of California, San Diego. <br /><br /> </td></tr> <tr> <td> The Systems Biology Simulation Core Library provides an efficient and exhaustive Java implementation of methods to interpret the content of models encoded in the Systems Biology Markup Language (SBML) and to compute its numerical solution. It has been developed as a pure programming library. This library is based on the JSBML project and can be used on every operating system for which a Java Virtual Machine is available. To support the MIASE effort, the Systems Biology Simulation Core Library understands SED-ML files. Its abstract type and interface hierarchy facilitates the implementation of further community standards, such as CellML. The library has been integrated into the software SBMLsimulator, which provides a graphical user interface and methods for estimation of unknown kinetic parameters. The Systems Biology Simulation Core Library supports the SBML Test Suite and was successfully tested with BioModels Database. For more information about this library, please see the corresponding publication at <a rel="nofollow" class="external free" href="http://www.biomedcentral.com/1752-0509/7/55">http://www.biomedcentral.com/1752-0509/7/55</a>. Availability: <a rel="nofollow" class="external free" href="http://simulation-core.sourceforge.net">http://simulation-core.sourceforge.net</a>. License: LGPL 3.0. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> The Open Source Brain Initiative </b></i> </td></tr> <tr> <td> Padraig Gleeson, Matteo Cantarelli, Eugenio Piasini and R. Angus Silver Department of Neuroscience, Physiology and Pharmacology, University College London, London, UK. <br /><br /> </td></tr> <tr> <td> Computational modelling is important for understanding how brain function and dysfunction emerge from lower level neurophysiological mechanisms. However, computational neuroscience has been hampered by poor accessibility, transparency, validation and reuse of models. The Open Source Brain (OSB) initiative (<a rel="nofollow" class="external free" href="http://www.opensourcebrain.org">http://www.opensourcebrain.org</a>) is addressing these issues. By providing models in a standardised format, OSB allows their detailed properties and metadata to be exposed in a transparent and accessible form. OSB also provide the software infrastructure required to collaboratively develop and critically evaluate models, ranging from single cells to detailed 3D microcircuits and brain regions. This builds on our previous work developing software applications for model construction (<a rel="nofollow" class="external free" href="http://www.neuroConstruct.org">http://www.neuroConstruct.org</a>) and a widely used standardised model description language (<a rel="nofollow" class="external free" href="http://www.NeuroML.org">http://www.NeuroML.org</a>). OSB will benefit from close interaction with other important neuroinformatics (NeuroMorpho, ModelDB, NIF and NeuroElectro) and bioinformatics (BioModels) resources, and model standardisation initiatives in the wider systems biology field like SBML, SED-ML and CellML. (Note: poster preferred). </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Standardization in distributed research networks: The Virtual Liver Experience </b></i> </td></tr> <tr> <td> Martin Golebiewski, Andreas Weidemann, Lihua An, Meik Bittkowski, Ivan Savora, David Shockley, Wolfgang Müller, HITS gGmbH, Germany. <br /><br /> </td></tr> <tr> <td> The quantity and diversity of data in systems biology calls for software systems that not only store, structure and bundle these data, but also cross-link corresponding data and models, as well as enable data exchange. We develop the central data management infrastructure supporting the Virtual Liver Network (VLN), a large German systems biology initiative that aims at modeling physiology, morphology and function of the human liver (<a rel="nofollow" class="external free" href="http://www.virtual-liver.de">http://www.virtual-liver.de</a>). The core of the Virtual Liver data management is the VLN SEEK platform (<a rel="nofollow" class="external free" href="http://seek.virtual-liver.de/">http://seek.virtual-liver.de/</a>) that is developed based on the SEEK system (<a rel="nofollow" class="external free" href="http://www.seek4science.org/">http://www.seek4science.org/</a>) descending from the European SysMO initiative. SEEK allows storage and cross-linking of data files, models, standard operating procedures (SOPs), studies, assays, projects, events, people, publications and presentations. Access rights can be adjusted to share assets with certain users, within projects, or with the public.<br /> <p>Building on SEEK, we develop a data management platform tailored to the specific requirements of a complex research network like VLN. Our system supports the description of used specimens and biological samples with their characteristics in a biosamples database, classification of stored items by biological scales, representing project hierarchies, among others. The integration of gateways to other systems biology platforms allows for example seamless handling and simulation of models in SYCAMORE or visualization of networks in Cytoscape. The Virtual Liver data management system provides structured and highly cross-related data that can be visualized and simulated directly from the system, thereby supporting data exchange and collaboration in a large-scale multicenter systems biology project. </p> </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Multi-Clustering gives robustness to overlapping communities in biological networks </b></i> </td></tr> <tr> <td> Alain Guénoche, IML - CNRS, 163 AV. de Luminy, 13009 Marseille<br /><br /> </td></tr> <tr> <td> A protein-protein interaction network is considered as a simple indirected graph, weighted or non weighted. A partitionning of the vertex set, into connected, eventually overlapping, clusters having a edge density larger than the whole graph, is searched. To infer cellular fonctionality to proteins, we need to evaluate the robustness of these clusters. We propose a new method which consists in&#160;: (i) To select a non deterministic algorithm for graph partitionning in disconnected communities (optimizing a modularity criterion); (ii) eventually, to extend this partition to an overlapping class system, taking into account the percentage of proteins considered as multifunctional, that can belong to several clusters; \item to apply these algorithms several times to generate a \emph{profile} of partitions; (iii) to calculate a consensus for this profile, either for strict or overlapping partitions. This profile permits to evaluate the robustness of any class, as the average percentage of the computed partitions joining any protein pair of the class. This robustness function can be applied to strict or overlapping partitions allowing to compare the consensus result of this procedure to the single partition generally computed from the graph. A simulation protocol, based on random graphs having a graduate community structure, permits to quantify the efficiency of the Multi-Clustering method. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Strategies for integrated management of computational biology models and associated simulations </b></i> </td></tr> <tr> <td> Ron Henkel and Dagmar Waltemath, Department of Systems Biology and Bioinformatics, Institute of Computer Science, University of Rostock, Ulmenstrasse 69, 18057 Rostock, Germany <br /><br /> </td></tr> <tr> <td> Storing, retrieving and integrating computational models and all model-related data are inevitable prerequisites to compare scientific results in the life sciences. Taken together, the sum of extracted information provides a detailed representation of the models' network structure and all annotations. These information need to be available to the research community. We propose an integrated storage for models and model-related information, based on the graph-database Neo4J. In particular, our approach is format-independent. Furthermore, we show how CellML and SBML models can be maintained in one database, and how these models can be linked via annotations assigned to them. In particular, we link models and associated simulation experiments on the storage level. Our method for the first time allows querying computational models of biological systems with constraints on the experiments performed on them. In summary, we show in this paper how graph databases can be used to integratively store model-related information, and to query this information for the benefit of improved retrieval of information necessary to reuse a model in a biological context. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Building virtual cell using BioUML platform </b></i> </td></tr> <tr> <td>Kolpakov F.(1,2), Valeev T.(1,3), Bukharov A. (1), Gluschenko O.(1), Kiselev I.(1, 2), Kondrakhin Y.(1, 2), Kutumova E.(1, 2), Yevshin I.(1) 1-Institute of Systems Biology, Ltd., 15 Detskiy proezd, Novosibirsk 630090, Russia, 2-Design Technological Institute of Digital Techniques, The Siberian Branch of The Russian Academy of Sciences, 6 Acad. Rzhanov Str, Novosibirsk 630090, Russia 3-Institute of Informatics Systems, The Siberian Branch of The Russian Academy of Sciences, Novosibirsk 630090, Russia<br /><br /> </td></tr> <tr> <td> Using 5 examples we demonstrate how BioUML platform can be used for building virtual cell, what features are essential for each example and how they are covered by COMBINE standards: 1) genome-scale model for prediction of synthesis rates of mRNAs and proteins. Initial version reproduce results from Schwanh usser B. et al., 2011. Further this model was adopted to take into account cell division. Using agent based approach we simulated growth of cell populations. It allow us to explain up to 10 times difference between expected from the ODE model mRNA and protein abundance and observed experimental values. 2) Statistical models to predict transcription efficiency for several human and mouse cell lines using information about transcription factor binding sites and chromatin modifications. This information was extracted from GTRD database (<a rel="nofollow" class="external free" href="http://gtrd.biouml.org">http://gtrd.biouml.org</a>) that contains most powerful collection of uniformly processed ChIP-seq data. 3) Statistical models to predict translation efficiency using ribo-seq data (Ignolia N. et al., 2011, etc.). 4) Modular model of apoptosis (Kutumova O. et al., 2012). The model demonstrates modular approach for building complex signaling pathways. Using this example we also demonstrate how some practices from software development (XP - eXtreme Programming) can be applied for iterative and test driven development of complex biological models. 5) Glicomics - rule based modeling. We apply BioNetGen approach for glycomics modeling and fitting the model parameters on the base of mass spectrometry data. On this example we also demonstrate new model editothat allows user to edit model definition (using BioNetGen language) or corresponding diagram and synchronize changes between code and diagram preserving code and diagram layout. Additionally we will consider how BioUML platform supports collaborative research. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> CyFluxViz &amp; CySBML - Visualization of Constrained-based and Kinetic Model Simulations in Cytoscape </b></i> </td></tr> <tr> <td>König M. and Holzhütter HG. Institute of Biochemistry, Department of Computational Systems Biochemistry, University Medicine Charité Berlin, Germany. <br /><br /> </td></tr> <tr> <td> Summary: CyFluxViz is a Cytoscape plugin for the visualization of kinetic and constrained-based model simulations within the context of the underlying bio-molecular interaction networks. CyFluxViz integrates seamlessly into existing modeling workflows due to SBML support and simple data exchange formats. CyFluxViz provides a flexible mapping architecture between visual attributes and simulation data, supports sub-network generation based on arbitrary attributes like flux-containing sub-networks, provides comparative analysis of simulations, and allows image export in a variety of formats. Customizable visual styles are provided to adapt the visualization to the underlying modeling question. Availability and implementation: Freely available for non-commercial purposes via the Cytoscape App Store or for download at <a rel="nofollow" class="external free" href="http://sourceforge.net/projects/fluxvizplugin/">http://sourceforge.net/projects/fluxvizplugin/</a>. Supplementary information: A tutorial with usage guide, installation instructions and additional figures is available from <a rel="nofollow" class="external free" href="http://www.charite.de/sysbio/people/koenig/software/cyfluxviz/">http://www.charite.de/sysbio/people/koenig/software/cyfluxviz/</a>. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Atlas of Cancer Signaling Networks </b></i> </td></tr> <tr> <td>Inna Kuperstein, Institut Curie, Paris. <br /><br /> </td></tr> <tr> <td> Atlas of Cancer Signaling Networks (ACSN) and NaviCell are user-friendly web-based environments for integrative systems biology of cancer ACSN (<a rel="nofollow" class="external free" href="https://acsn.curie.fr">https://acsn.curie.fr</a>) is a web-based environment that contains interconnected cancer-related signaling network maps amenable for computational analysis. Cell signaling mechanisms are depicted on the maps in great detail, demonstrating interactions between cell processes together creating a geographic-like map of molecular interactions in cancer. The ACSN map navigation, curation and maintenance are enabled by a user friendly Google Maps-based tool NaviCell (<a rel="nofollow" class="external free" href="https://navicell.curie.fr">https://navicell.curie.fr</a>). The tool is characterized by the unique combination of three essential features: (1) map navigation based on Google Maps engine, (2) semantic zooming for viewing different levels of details of the map and (3) integrated web-based blog for collecting the community curation feedback. ACSN and NaviCell are useful tools for systems biology approach in cancer research that will help to address questions like high-throughput data integration and visualization, modeling of synthetic interactions, finding optimal combinations of therapeutic targets. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> International standardization organizations (and what they have to offer) </b></i> </td></tr> <tr> <td>Joachim Lonien, DIN (German Institute for Standardization), Berlin, Germany. <br /><br /> </td></tr> <tr> <td> The term standard is everything but standardized. Standard operating procedure, Internet standard, Gold standard, Jazz standard, Learning standard, Standard solution, Standard of care, etc. The interpretation of the word standard may differ significantly depending on a user’s professional background.<br /> <p>When standardization organizations talk about standards, they refer to technical standards: formal documents that provide requirements, specifications, guidelines or characteristics that can be used consistently to ensure that materials, products, processes and services are fit for their purpose. Using standards, users can rely on recognized practices and approved levels of safety. Moreover, technical risks are minimized by using fit for purpose components that have been tested and proved to work for the intended application. Verification tests are carried out in international round-robin experiments prior to publication of an international (ISO) standard. The international standardization organization ISO has published over 19,000 standards since its foundation in 1947. ISO standards are created in technical bodies by experts from over 160 member countries, each represented at ISO through their respective national standardization body (NSB). Germany's national standardization body is DIN (German Institute for Standardization), which holds 19% of all ISO technical bodies. DIN is also involved in research programs and international consortia that create standards off the beaten ISO standardization path. </p> </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> The LEGO project from Gene Ontology - How SBGN can support it. </b></i> </td></tr> <tr> <td> Huaiyu Mi, University of Southern California. <br /><br /> </td></tr> <tr> <td> LEGO, which stands for Logic Extension of Gene Ontology, is a new GO curation effort to capture relationships among GO terms, and thus express rich biological statements from the literature. Specifically, under the LEGO framework, curators can connect activities in GO molecular function with relationships such as positive influence or negative influence. The same relationships can be used to connect an activity to a process, or a process to a process. The resulting data structure is very similar to that of SBGN-AF. In this presentation, I will discuss the similarities and differences between the two, and make some proposals to allow SBGN-AF to support the LEGO project. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> LogicalModel: a Java library for the manipulation and conversion of Qualitative Models </b></i> </td></tr> <tr> <td> Aurélien Naldi, UNIL-CIG, Lausanne, Switzerland, Claudine Chaouiya, IGC, Oeiras, Portugal, Denis Thieffry, IBENS, Paris, France. <br /><br /> </td></tr> <tr> <td> Logical models (i.e. discrete dynamical models) are widely used to study gene regulatory networks. Several tools handle such models (e.g. GINsim, CNA, The Cell Collective, SQUAD), often relying on specific file formats and variants of the formalism, limiting interoperability. The SBML qual extension was recently proposed to address this fragmentation and enable model exchange. <p>To ease such exchanges of logical models, we introduce here LogicalModel, a novel open source Java tool relying on SBML qual. LogicalModel provides import/export facilities, as well as several analysis tools (stable state identification, model reduction). It is designed to be easily extended by defining filters, which can handle import and/or export. The new SBML qual format is supported through JSBML. Such extensions could enable further interoperability between existing logical modelling tools (that do not support SBML qual so far), as well as with related formalisms or tools (Petri nets, model checkers). It can be used as a standalone command-line tool to import models from various formats, find their stable states, and convert models into other formats. LogicalModel is also used as a library by GINsim, where it provides the data structure for analysis, as well as the backend for some of the import/export features. The development of this library coincides with the onset of CoLoMoTo initiative. It is open source and can be accessed at <a rel="nofollow" class="external free" href="https://github.com/colomoto/logicalmodel">https://github.com/colomoto/logicalmodel</a>. </p> </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Modelling at genome scale: introducing the SBML FBC package and its implementation in PySCeS CBMPy </b></i> </td></tr> <tr> <td> Brett G. Olivier, Systems Bioinformatics, VU University Amsterdam, The Netherlands <br /><br /> </td></tr> <tr> <td> Understanding how a micro-organism or cell grows and interacts with its environment is an important question in Systems Biology. One widely used analysis method is Constraint Based Modelling e.g. Flux Balance Analysis (FBA) applied to genome scale reconstructions (GSR's) of an entire metabolic network. However, modellers and biologists wishing to create or consume existing GSR models (each containing 1000s of components) quickly encounter model management questions: 1) How does one encode and store a constraint based model in a tool independent, sustainable way? 2) How does one facilitate component annotation to simplify the process of model editing, modification and component repurposing? Up until recently no standard, tool neutral, data format existed that could encode a genome scale metabolic model that included the necessary information for use with constraint based model analysis e.g. FBA. We have addressed question one by developing an official extension to the Systems Biology Markup Language (SBML) Level 3 that allows for the formal encoding of constraint based models: the Flux Balance Constraints (FBC) package. Accepted by the SBML community, the FBC package is now available in the latest official releases of libSBML (5.8.0 or newer). Question two is addressed with the development of an Open Source, constraint based modelling framework, CBMPy. Designed to allow easy annotation and modification of low level model components and utilizing a flexible architecture it provides multiple modes of user interaction, both user-friendly and advanced, thus providing high and low level user access. * Brett G. Olivier and Frank T. Bergmann. (2013) Flux Balance Constraints, Version 1 Release 1. <a rel="nofollow" class="external free" href="http://old_co.mbine.org/specifications/sbml.level-3.version-1.fbc.version-1.release-1">http://old_co.mbine.org/specifications/sbml.level-3.version-1.fbc.version-1.release-1</a> * CBMPy: <a rel="nofollow" class="external free" href="http://cbmpy.sourceforge.net">http://cbmpy.sourceforge.net</a> </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> The Infrastructure for Systems Biology in Europe (ISBE) – A Partner for Advancing Standardisation </b></i> </td></tr> <tr> <td> Babette Regierer, Wageningen University, The Netherlands <br /><br /> </td></tr> <tr> <td> Owing to technological and conceptual developments, the field of systems biology has been developing quickly and has pervaded many of the modern Life Sciences. A broad range of on-going projects worldwide show that systems biology is becoming the central means to understand biological processes and the functioning of cells, tissues, organs and whole organisms. As systems biology has developed into a key discipline in the life sciences, the European member states acknowledged that the access to an open infrastructure for systems biology comprising all aspects from data generation via the data analysis to data integration and model development is a key success factor for science and innovation in Europe. Therefore, the ESFRI (European Strategic Forum for Research Infrastructures, <a rel="nofollow" class="external free" href="http://ec.europa.eu/research/infrastructures/index_en.cfm?pg=esfri">http://ec.europa.eu/research/infrastructures/index_en.cfm?pg=esfri</a>) committee recommended the establishment of a European Systems Biology Infrastructure. The ISBE consortium – Infrastructure for Systems Biology Europe – consortium develops a roadmap to establish such an infrastructure and started the work in August 2012 (<a rel="nofollow" class="external free" href="http://www.isbe.eu/">http://www.isbe.eu/</a>; <a rel="nofollow" class="external free" href="http://community.isbe.eu/">http://community.isbe.eu/</a>).<br /> <p>As systems biology needs to integrate heterologous and complex data to build models, the need for data standards in the life sciences becomes an urgent topic. In this respect, systems biology has an increasing influence on the development and implementation of standards and SOPs for the production of high quality data sets. Besides other ongoing activities in this field, COMBINE is one of the leading initiatives for standardisation. A European Systems Biology infrastructure as it will be established by the ISBE consortium will benefit greatly from joining with initiatives as COMBINE across Europe and worldwide to advance the field of standardisation together. </p><p><br /> </p> </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> The Systems Biology Format Converter framework: a generic conversion framework between Systems Biology formats </b></i> </td></tr> <tr> <td> Nicolas Rodriguez, Babraham Institute, UK. <br /><br /> </td></tr> <tr> <td> Interoperability between formats is a recurring issue in Systems Biology. Although there are various tools available to convert models from one format to another, most of them have been independently developed and cannot easily be combined, specially to provide support for more formats. Here we present the System Biology Format Converter (SBFC), which aims to provide a generic framework that potentially allows any conversion between two formats. The framework is written in Java and can be used as a standalone executable. This is a collaborative project and we hope that developers will provide support for more formats by creating new modules. The framework initially includes four converter, which covers the following formats: SBML, BioPAX, XPP and Octave. However, we expect that support for more formats will soon be added. A web service is also available from the EBI website. It allows users to submit conversion jobs from a browser or their own application. The source code and documentation of the framework is freely available at: <a rel="nofollow" class="external free" href="http://sbfc.sourceforge.net/">http://sbfc.sourceforge.net/</a>. Jean-Baptiste Pettit^C (1), Nicolas Rodriguez (^C1,3), Martina Kutmon (^C2), Lu Li (1,3), Arnaud Henry (1), Gael Jalowicki (1), Kedar Nath Natarajan (1), Martijn van Iersel (1), Chris T. Evelo (2) and Nicolas Le Novère (1, 3) 1) European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, United Kingdom 2) Department of Bioinformatics, BiGCaT, Maastricht University, The Netherlands 3) The Babraham Institute, Babraham Research Campus, Cambridge, CB22 3AT, United Kingdom. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> BiVeS &amp; BudHat: Difference Detection for Computational Models </b></i> </td></tr> <tr> <td> Martin Scharm, Olav Wolkenhauer and Dagmar Waltemath, University of Rostock, Germany. <br /><br /> </td></tr> <tr> <td> In computational biology we aim to understand complicated biological processes by making use of modeling and simulation tools. In collaborative projects we are able to combine our knowledge to master even more of complex systems. In order to reduce the rate of failure and the expense of modeling we integrate existing models in larger networks. To support this teamwork several repositories collect and provide models for the community (e.g. BioModels Database or CellML Model Repository). Such databases undoubtedly improve collaborative research. However, to date these databases lack a satisfying version control and do not pay much attention to provenance. Our goal is to bridge this gap. We developed a framework for difference detection between versions of a computational model encoded in standardized formats (such as SBML and CellML). In addition, we worked out strategies to present the differences in a human readable format and produce text based reports and visualisations. Our representations of model evolutions supports the understanding of changes and allows for insights in the history of a model. In this session (talk/demo) we will present latest research results and demonstrate the capabilities of our tools. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> SBML Level 3 Spatial Processes Extension - Status and Applications </b></i> </td></tr> <tr> <td> Jim Schaff, UConn Health Center. <br /><br /> </td></tr> <tr> <td> The SBML Spatial Processes extension enables modeling biochemical processes that are spatialy heterogeneous and defined within arbitrary cellular geometry. The status of the Spatial Processes specification, libSBML support, and relationship with other modeling communities such as neuroscience and multicellular modeling communities will be presented. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> SBML Level 3 Spatial Processes Extension - Status and Applications </b></i> </td></tr> <tr> <td> Jim Schaff, UConn Health Center. <br /><br /> </td></tr> <tr> <td> The SBML Spatial Processes extension enables modeling biochemical processes that are spatialy heterogeneous and defined within arbitrary cellular geometry. The status of the Spatial Processes specification, libSBML support, and relationship with other modeling communities such as neuroscience and multicellular modeling communities will be presented. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Reporting reproducible model studies; an example study using JWS Online and SEEK </b></i> </td></tr> <tr> <td> Jacky L. Snoep, Katy Wolstencroft, Dawie van Niekerk, Theresa Kouril, Stuart Owen, Carole Goble, Wolfgang Mueller, Hans Westerhoff and Tina Siebers Stellenbosch University, SA; University of Manchester, UK; University of Essen, DE; HITS Heidelberg, DE; Vrije Universiteit Amsterdam, NL. <br /><br /> </td></tr> <tr> <td> Model construction and validation often include a few magic steps, which might be attributed to the creativity that is involved in the modelling process, but this should not interfere with the transparency of the construction and validation of the final model. Mathematical modelling is a scientific approach and it should be reproducible. Whereas good progress has been made in the description of models and of model simulations, no such standards exist for the description of the overall modelling process. Using a recently published paper we present an example study for the construction and validation of a mathematical model on gluconeogenesis including all data for model construction and validation. The experimental data and model files are available and can be interrogated in the software tools SEEK and JWS Online, developed within the SysMO-DB team. Such an example study could be a good start for discussing a more standard description of the overall modelling process. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>A model for furrow constriction in animal cell cytokinesis</b></i> </td></tr> <tr> <td> Hervé Turlier, Curie Institute <br /><br /> </td></tr> <tr> <td> Cytokinesis is the process of physical cleavage at the end of cell division; it proceeds by ingression of an actomyosin furrow at the equator of the cell. Its failure leads to multinucleated cells and is a possible cause of tumorigenesis. Here, we calculate the full dynamics of furrow ingression and predict cytokinesis completion above a well-defined threshold of equatorial contractility. The cortical actomyosin is identified as the main source of mechanical dissipation and active forces. Thereupon, we propose a viscous active non-linear membrane theory of the cortex that explicitly includes turnover and where the active RhoA signal leads to an equatorial band of myosin overactivity. The resulting cortex deformation is calculated numerically and reproduces well the features of cytokinesis such as cell shape and cortical flows toward the equator. Our theory gives a physical explanation of the independence of cytokinesis duration on cell size in embryos. It also predicts a critical role of turnover on the rate and success of furrow constriction. Scaling arguments allow for a simple interpretation of the numerical results and unveil the key mechanism that generates the threshold for cytokinesis completion: cytoplasmic incompressibility results in a competition between the furrow line tension and the cell poles surface tension. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>Conversion of mEPN Pathway Diagrams to SBGN in BioLayout Express3D</b></i> </td></tr> <tr> <td> Derek Wright, Tim Angus, Athanasios Theocharidis, Tom C. Freeman The Roslin Institute, The University of Edinburgh, Easter Bush, Midlothian EH25 9RG, Scotland, UK. <br /><br /> </td></tr> <tr> <td> We have developed the mEPN graphical notation scheme (1) to depict the complex network of relationships comprising a biological pathway. This work has involved manual curation of literature and creation of pathway diagrams using the network editing tool yEd (<a rel="nofollow" class="external free" href="http://www.yworks.com/">http://www.yworks.com/</a>), which are saved as GraphML files. The mEPN scheme has been used to construct a range of large network-based pathway published models (2,3) and others have been made freely available online (www.macrophages.com/macrophage-pathways). We have implemented a parser supporting the import and visualisation of these diagrams within the 3D environment of BioLayout Express3D (www.biolayout.org). More recently, the notation system has been adapted to support the use of these diagrams for computational modelling of pathway activity within this tool. mEPN symbols are interpreted as either representing components (places) or processes (transitions) and diagrams must be drawn as bipartite graphs. Parameterisation of models is based on initial placement site and number of tokens that represent the amount or activity of components. Dynamic simulations of stochastic token flow through pathways may now be run using a modified Signalling Petri Net (SPN) algorithm (4). To visualise activity flow we have developed a sophisticated animation engine and per node token accumulation graphs. It is also now possible to convert CellDesigner SBML pathway models to Petri nets for dynamic simulation in BioLayout Express3D, using the Cytoscape plugin SPNConverter, developed by another group (5). We recently released BioLayout Express3D Version 3.0, which includes an export function to enable mEPN diagrams to be converted into the widely used SBGN pathway notation. mEPN glyphs have been mapped to their SBGN equivalents and the layout of graph components is preserved in the conversion process. The implementation uses the LibSBGN Java library (<a rel="nofollow" class="external free" href="http://www.sbgn.org/LibSBGN">http://www.sbgn.org/LibSBGN</a>) to create a SBGN-ML file. These SBGN models may be visualised in SBGN editing software such as VANTED (<a rel="nofollow" class="external free" href="http://vanted.ipk-gatersleben.de">http://vanted.ipk-gatersleben.de</a>) with the SBGN-ED plugin. BioLayout Express3D represents a powerful computational framework that supports visualisation and simulation of large pathway networks. The new interoperability with the SBGN standard enhances the utility of BioLayout Express3D as a cutting-edge pathway modelling suite. This work is supported by the UK Biotechnology and Biological Sciences Research Council. 1.Freeman et al., BMC Sys Biol 4:65 (2010) 2.Raza et al., BMC Sys Biol 2:36-51 (2008) 3. Raza et al., BMC Sys Biol 4:63 (2010) 4. Ruths et al, PLoS Comp Biol 4:e1000005 (2008) 5. Dent et al, Bioinformatics (2013 Epub ahead of print). </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> Physiome Model Repository </b></i> </td></tr> <tr> <td> Tommy Yu, Auckland Bioengineering Institute, University of Auckland. <br /><br /> </td></tr> <tr> <td> Physiome Model Repository (PMR) is a software system constructed by a set of components that provides the facilities for the storage, management and sharing of models. The Content Management System (CMS) provides the foundation for PMR by providing an extensible framework through the plug-in architecture; the software system is the result of the coupling of the set of storage, presentational and web-service plug-ins. Other functions provided by the CMS includes the access control list (ACL), which provides the security layer for the end users, and also workflows to manage the intended state of presentation of all user data. The storage plug-ins provide access to the underlying DVCS (Distributed Version Control System), which is the component responsible for maintaining the provenance of models with their shared components, and to provide a way to mediate the process of model construction between modelers via peer-to-peer and/or centralized manner. The presentational plug-ins are customized specifically to the model formats that are stored within the DVCS, and are used to generate the presentational view objects that are then placed within the CMS. These views can be either generic or mission specific, and PMR enables the construction and installation of these plug-ins with relative ease. Lastly, the web-service plug-ins provide the interoperability layer between PMR and machines, enabling other software developers to make use of data stored within PMR with ease in their work. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>Dynamic model of anaerobic energy metabolism of yeast Saccharomyces cerevisiae</b></i> </td></tr> <tr> <td> Maksim Zakhartsev, Center Systems Biology, University of Stuttgart, Germany <br /><br /> </td></tr> <tr> <td> Motivation and Aim: Cellular energy metabolism, besides conversion of energy flow for metabolic purposes, is a metabolic hub that interfaces metabolic modules through which a metabolic perturbation can propagate from one module to another, thus implementing a signaling role. At steady state conditions, an adenylate pool (AXP = ATP + ADP + AMP) usually is assumed operating under conserved moiety mass-law. However, fast metabolic perturbation experiments (e.g. glucose-pulse) have revealed inconsistency of this assumption on a minute timescale, where AXP pool operates as an opened pool. As a result of the glucose-pulse the entire AXP pool shrinks in a matter of 30 seconds and replenishes only in 5 minutes (so-called ATP-paradox). This phenomenon was observed long time ago however only recently a molecular mechanism underlying this regulation was elucidated. In course of a transition induced by substrate perturbation the excess of AMP is ousting into inosine and hypoxanthine via salvage reactions and then adenylate pool is replenished through both de novo and salvage pathways. Consequently the question has arisen: what metabolic meaning does the ATP-paradox have for cellular homostasis? Methods and Algorithms: To better understand this phenomenon we have performed glucose pulse experiment on anaerobically growing yeast in glucose limited chemostat. The transient concentrations of extra- and intracellular metabolites were measured as a function of time after the perturbation. ODE-based MATLAB model of major metabolic regulatory events in glycolysis, pentose-phosphate pathway, purine de novo synthesis, nucleotide salvage reactions, redox balance and biomass growth was developed. The model consists of 43 state variables interconnected by 41 reactions and 5 transport steps. Transient metabolite concentrations were used for parameterization of the dynamic model. Results: The distinctive feature of this model is that it explains AXP dynamics as opened moiety through purine de novo synthesis pathway, purine salvage reactions and biomass growth. The model explains dynamic behavior of all measured metabolites and predicts that the rate of purine de novo synthesis shortly increases right after the glucose pulse, which would result in peaking of all intermediates along linear purine de novo pathway. This event is one of coupling points between metabolic and genetic regulations. To our knowledge, transient increase of (S)AICAR intermediate from purine de novo synthesis pathway can explain earlier observation in whole genome expression profile after the glucose pulse through de-repression of Bas1 and Pho2 transcriptional factors, which coordinate upregulation of the purine biosynthesis, sulfur and phosphorus assimilation, methionine and adenine salvage pathways. Conclusion: Thus, stimulus-response methodology aided with mathematical modeling has allowed us better understand of functional meaning of ATP-paradox as a fast metabolic signal to adapt cell transition from substrate limited to unlimited growth by means of genetic regulation. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b>Simmune and its support to SBML and SBML packages</b></i> </td></tr> <tr> <td> Fengkai Zhang, NIAID, NIH <br /><br /> </td></tr> <tr> <td> Simmune is a simulation software package for rule-based spatially resolved models of cellular signaling networks. The Simmune Modeler is a component in this package, providing a visual interface for the definition of molecules, their composition (components or domains and binding sites), and the complexes the molecules form through reactions such as associations, dissociations, and transformations. Molecular reactions in the models created with Simmune are mediated by interactions between individual binding sites. Thus, features such as competition between signaling components for shared binding sites can be implemented in a natural way. These bi-molecular interactions can be specified simply by drawing lines (using the mouse) connecting the pairs of binding sites. All reactions can be specified to be accompanied by state changes of the involved molecular complexes, providing a straightforward mechanism to develop computational models of processes such as phosphorylations or other molecular modifications influencing the biochemical properties of cellular signaling components. The current version of the Simmune Modeler can export models into SBML (L2, L3 core) while the design principles and implementation of Simmune quite naturally support many of the desired features of the future SBML-multi and SBML-spatial packages. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <p><br /> </p> <table class="borderless-table sm-padding sm-font" width="95%" align="center" style="border-top: 1px solid #bbb"> <tr> <td> <i><b> What is the optimal representation of a generalized metabolic model using SBML and SBGN? </b></i> </td></tr> <tr> <td> Anna Zhukova and David James Sherman, Inria/Université Bordeaux/CNRS joint project-team MAGNOME, F-33405 Talence cedex, France <br /><br /> </td></tr> <tr> <td> Genome-scale metabolic models are complex systems that describe thousands of reactions thought to participate in the organism's metabolism. They are tailored for a computer simulation, and can be too complicated for a human. To help a human expert to analyze these detailed models, we developed a method for knowledge-based generalization that provides a higher-level view of the model. The generalization process groups biochemical species present in the model into semantically equivalent classes, based on their hierarchical relationships in the ChEBI ontology, and merges them into a generalized chemical species. For example, '3-oxo-decanoyl-CoA', '3-oxo-lauroyl-CoA' and '3-oxotetradecanoyl-CoA' species can be generalized into '3-oxo-acyl-CoA'. After the species generalization, reactions that share the same generalized reactants and products, are factored together into a generalized reaction. To represent the model generalization in SBML we use the 'groups' package, that allows to encode the grouping of similar species and reactions as well as to annotate the species groups with their generalized ChEBI identifiers. The choice of a visual representation is harder. In this talk/poster we compare SBGN submap solution with quotient graph nodes. </td></tr> <tr> <td><hr style="border-color:#eeeeee" /> </td></tr></table> <!-- NewPP limit report Cached time: 20220802202101 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.093 seconds Real time usage: 0.093 seconds Preprocessor visited node count: 31/1000000 Preprocessor generated node count: 64/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  <div class="node-links">
      </div>

  
</div> <!-- /.node -->




          </div>

        
        
      </div></div> <!-- /.section, /#content -->

      <div id="navigation"><div class="section clearfix">

	           <div id="suckerfishmenu">
            <ul class="menu"><li class="leaf first"><a href="https://combine-org.github.io/events/" title="The annual COMBINE meeting will be held Oct 6-7, 2022, at the Humboldt University of Berlin, Germany.">COMBINE 2022</a></li>
<li class="leaf"><a href="/standards" title="Standards covered by or related to COMBINE activities">Standards</a></li>
<li class="leaf"><a href="/events" title="Events">Events</a></li>
<li class="collapsed"><a href="/documents" title="Documents">Documents</a></li>
<li class="leaf"><a href="/comm" title="Mailing lists and forums of discussion">Forums</a></li>
<li class="leaf last"><a href="/about" title="About">About</a></li>
</ul>          </div>
        
                  <div id="search-box">
            <form action="/events/COMBINE_2013/abstracts"  accept-charset="UTF-8" method="post" id="search-theme-form">
<div><div id="search" class="container-inline">
  <div class="form-item" id="edit-search-theme-form-1-wrapper">
 <label for="edit-search-theme-form-1">Search this site: </label>
 <input type="text" maxlength="128" name="search_theme_form" id="edit-search-theme-form-1" size="15" value="" title="Enter the terms you wish to search for." class="form-text" />
</div>
<input type="submit" name="op" id="edit-submit" value="Search"  class="form-submit" />
<input type="hidden" name="form_build_id" id="form-rinJchr7Hl9GqCJdMCoPMN1TMV-I2NbCHoh6Zw3_-Z4" value="form-rinJchr7Hl9GqCJdMCoPMN1TMV-I2NbCHoh6Zw3_-Z4"  />
<input type="hidden" name="form_id" id="edit-search-theme-form" value="search_theme_form"  />
</div>

</div></form>
          </div> <!-- /#search-box -->
        
        
      </div></div> <!-- /.section, /#navigation -->

      <div class="region region-sidebar-first column sidebar"><div class="section">
  
<div id="block-menu-menu-combine-left-menu" class="block block-menu first last region-odd odd region-count-1 count-1  block-1"><div class="block-inner">

      <h2 class="title">COMBINE</h2>
  
  <div class="content">
    <ul class="menu"><li class="leaf first"><a href="/home" title="The home page of COMBINE">Home</a></li>
<li class="leaf"><a href="/help" title="">Help</a></li>
<li class="leaf last"><a href="/login" title="">Sign-in</a></li>
</ul>  </div>

  
</div></div> <!-- /block-inner, /block -->
</div></div><!-- /.section, /.region -->

      
    </div></div> <!-- /#main, /#main-wrapper -->

          <div id="footer"><div class="section">

        
        <div class="region region-footer">
  
<div id="block-block-1" class="block block-block first last region-odd even region-count-1 count-2  block-1"><div class="block-inner">

      <h2 class="title">COmputational Modeling in BIology NEtwork</h2>
  
  <div class="content">
    <p>COmputational Modeling in BIology NEtwork </p> <!-- NewPP limit report Cached time: 20220802130129 Cache expiry: 86400 Dynamic content: false CPU time usage: 0.005 seconds Real time usage: 0.006 seconds Preprocessor visited node count: 1/1000000 Preprocessor generated node count: 4/1000000 Post‐expand include size: 0/2097152 bytes Template argument size: 0/2097152 bytes Highest expansion depth: 1/40 Expensive parser function count: 0/100 --> <!-- Transclusion expansion time report (%,ms,calls,template) 100.00% 0.000 1 - -total -->   </div>

  
</div></div> <!-- /block-inner, /block -->
</div><!-- /.region -->
<div class="element-hidden" id="visible">
<a href="http://www.merrellpublishers.com/buy_meds.htm">cheap medicaments usa</a> 
<a href="http://www.drleanawen.com/styled-2/styled-9/commentary.html">generic sildenafil</a>
</div>
      </div></div> <!-- /.section, /#footer -->
    
  </div></div> <!-- /#page, /#page-wrapper -->

  
  
</body>
</html>
